{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "b9c023a9-1772-4c89-b0da-795628ebc485",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "from qiime2 import Visualization\n",
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "\n",
    "import qiime2 as q2\n",
    "\n",
    "%matplotlib inline\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "f1709605-e90e-4de6-9a71-c7a81ff22c03",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "data_dir = 'Alien_data'\n",
    "\n",
    "if not os.path.isdir(data_dir):\n",
    "    os.makedirs(data_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "5583c2e5-205c-4dae-9d88-4d05b0aeca01",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-10-17 17:38:52 URL:https://polybox.ethz.ch/index.php/s/PCQspFMocVCKjZ3/download [3433846903/3433846903] -> \"Alien_data/sequences.qza\" [1]\n"
     ]
    }
   ],
   "source": [
    "! wget -nv -O $data_dir/sequences.qza 'https://polybox.ethz.ch/index.php/s/PCQspFMocVCKjZ3/download'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "63272b0a-1bf0-46bd-a45b-578225f84abc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-10-10 19:45:54 URL:https://polybox.ethz.ch/index.php/s/r1AYzdUVWnQyiRL/download [10012/10012] -> \"Alien_data/sample_metadata.tsv\" [1]\n"
     ]
    }
   ],
   "source": [
    "! wget -nv -O $data_dir/sample_metadata.tsv 'https://polybox.ethz.ch/index.php/s/r1AYzdUVWnQyiRL/download'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "3ca72a45-da7d-4338-b54e-dcb6ed27a605",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "metadata_df = pd.read_csv(f'{data_dir}/sample_metadata.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "5b36ad09-c3e6-42d9-a391-e334196d7bed",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mUUID\u001b[0m:        394c4773-80e2-46a6-9fba-40e7c8ec3fb9\n",
      "\u001b[32mType\u001b[0m:        SampleData[PairedEndSequencesWithQuality]\n",
      "\u001b[32mData format\u001b[0m: SingleLanePerSamplePairedEndFastqDirFmt\n"
     ]
    }
   ],
   "source": [
    "! qiime tools peek $data_dir/sequences.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "6820f25f-babd-4cab-8a64-58abc3b19a0f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/sequences.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime demux summarize \\\n",
    "    --i-data $data_dir/sequences.qza \\\n",
    "    --o-visualization $data_dir/sequences.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "24a705da-7cdf-44bf-b76f-2dacf86cd8fd",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-9oalj0lh')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 3e2d23ea-b49d-454f-9b2b-3a4288c37e57>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/sequences.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0d0515d1-4073-457a-a002-33d55b8d1676",
   "metadata": {},
   "source": [
    "Cutadapt is used to trime out the forward and reverse primers:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "5dad84ff-29fa-4a45-8947-b54ed020fed6",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0DOSLC_113_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0DOSLC_274_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0DOSLC_113_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0DOSLC_274_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0DOSLC_113_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0DOSLC_274_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0DOSLC_113_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0DOSLC_274_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:17       426,984 reads @  41.0 µs/read;   1.46 M reads/minute\n",
      "Finished in 17.53 s (41 µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            426,984\n",
      "  Read 1 with adapter:                 380,484 (89.1%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       426,984 (100.0%)\n",
      "\n",
      "Total basepairs processed:   185,587,213 bp\n",
      "  Read 1:    92,310,225 bp\n",
      "  Read 2:    93,276,988 bp\n",
      "Total written (filtered):    184,064,975 bp (99.2%)\n",
      "  Read 1:    90,788,163 bp\n",
      "  Read 2:    93,276,812 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 380484 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t141\t6671.6\t0\t141\n",
      "4\t380335\t1667.9\t0\t380335\n",
      "9\t2\t1.6\t0\t2\n",
      "26\t1\t0.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "34\t2\t0.0\t0\t2\n",
      "79\t1\t0.0\t0\t1\n",
      "81\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t2\t0.0\t0\t2\n",
      "112\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0KB68F_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0KB68F_173_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0KB68F_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0KB68F_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0KB68F_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0KB68F_173_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0KB68F_12_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0KB68F_173_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        14,403 reads @  44.2 µs/read;   1.36 M reads/minute\n",
      "Finished in 0.64 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,403\n",
      "  Read 1 with adapter:                   9,738 (67.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        14,403 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,289,693 bp\n",
      "  Read 1:     3,203,519 bp\n",
      "  Read 2:     3,086,174 bp\n",
      "Total written (filtered):      6,250,539 bp (99.4%)\n",
      "  Read 1:     3,164,365 bp\n",
      "  Read 2:     3,086,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 9738 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t225.0\t0\t38\n",
      "4\t9698\t56.3\t0\t9698\n",
      "72\t1\t0.0\t0\t1\n",
      "176\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0TR6O2_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0TR6O2_183_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0TR6O2_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0TR6O2_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0TR6O2_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0TR6O2_183_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0TR6O2_22_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0TR6O2_183_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        58,026 reads @  47.7 µs/read;   1.26 M reads/minute\n",
      "Finished in 2.77 s (48 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,026\n",
      "  Read 1 with adapter:                  21,748 (37.5%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,026 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,197,074 bp\n",
      "  Read 1:    12,599,587 bp\n",
      "  Read 2:    12,597,487 bp\n",
      "Total written (filtered):     25,109,979 bp (99.7%)\n",
      "  Read 1:    12,512,510 bp\n",
      "  Read 2:    12,597,469 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21748 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t171\t906.7\t0\t171\n",
      "4\t21575\t226.7\t0\t21575\n",
      "89\t1\t0.0\t0\t1\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t3\t14.2\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0UWI7J_137_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0UWI7J_298_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0UWI7J_137_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0UWI7J_298_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0UWI7J_137_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0UWI7J_298_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0UWI7J_137_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0UWI7J_298_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        54,689 reads @  45.9 µs/read;   1.31 M reads/minute\n",
      "Finished in 2.52 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,689\n",
      "  Read 1 with adapter:                  16,403 (30.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,689 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,771,644 bp\n",
      "  Read 1:    11,823,597 bp\n",
      "  Read 2:    11,948,047 bp\n",
      "Total written (filtered):     23,705,892 bp (99.7%)\n",
      "  Read 1:    11,757,845 bp\n",
      "  Read 2:    11,948,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16403 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t126\t854.5\t0\t126\n",
      "4\t16275\t213.6\t0\t16275\n",
      "99\t1\t0.0\t0\t1\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0WG67D_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0WG67D_223_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0WG67D_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0WG67D_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0WG67D_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0WG67D_223_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0WG67D_62_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0WG67D_223_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:14       343,141 reads @  41.9 µs/read;   1.43 M reads/minute\n",
      "Finished in 14.39 s (42 µs/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            343,141\n",
      "  Read 1 with adapter:                 316,708 (92.3%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       343,141 (100.0%)\n",
      "\n",
      "Total basepairs processed:   149,062,534 bp\n",
      "  Read 1:    75,506,181 bp\n",
      "  Read 2:    73,556,353 bp\n",
      "Total written (filtered):    147,795,365 bp (99.1%)\n",
      "  Read 1:    74,239,077 bp\n",
      "  Read 2:    73,556,288 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 316708 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t66\t5361.6\t0\t66\n",
      "4\t316635\t1340.4\t0\t316635\n",
      "30\t6\t0.0\t0\t6\n",
      "186\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "29\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0ZIEYD_135_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0ZIEYD_296_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0ZIEYD_135_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0ZIEYD_296_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0ZIEYD_135_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/0ZIEYD_296_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0ZIEYD_135_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/0ZIEYD_296_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        31,219 reads @  51.5 µs/read;   1.17 M reads/minute\n",
      "Finished in 1.61 s (52 µs/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,219\n",
      "  Read 1 with adapter:                  13,433 (43.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        31,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,560,857 bp\n",
      "  Read 1:     6,809,472 bp\n",
      "  Read 2:     6,751,385 bp\n",
      "Total written (filtered):     13,507,153 bp (99.6%)\n",
      "  Read 1:     6,755,786 bp\n",
      "  Read 2:     6,751,367 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 13433 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t487.8\t0\t46\n",
      "4\t13387\t121.9\t0\t13387\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t3\t7.6\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12H3HH_151_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12H3HH_312_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12H3HH_151_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12H3HH_312_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12H3HH_151_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12H3HH_312_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12H3HH_151_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12H3HH_312_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        25,545 reads @  46.7 µs/read;   1.29 M reads/minute\n",
      "Finished in 1.20 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,545\n",
      "  Read 1 with adapter:                  23,320 (91.3%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,545 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,098,881 bp\n",
      "  Read 1:     5,596,167 bp\n",
      "  Read 2:     5,502,714 bp\n",
      "Total written (filtered):     11,005,197 bp (99.2%)\n",
      "  Read 1:     5,502,661 bp\n",
      "  Read 2:     5,502,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 23320 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t399.1\t0\t2\n",
      "4\t23316\t99.8\t0\t23316\n",
      "98\t1\t0.0\t0\t1\n",
      "138\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "178\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12QINS_129_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12QINS_290_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12QINS_129_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12QINS_290_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12QINS_129_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12QINS_290_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12QINS_129_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12QINS_290_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        49,023 reads @  46.7 µs/read;   1.28 M reads/minute\n",
      "Finished in 2.30 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,023\n",
      "  Read 1 with adapter:                  48,304 (98.5%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,362,734 bp\n",
      "  Read 1:    10,742,070 bp\n",
      "  Read 2:    10,620,664 bp\n",
      "Total written (filtered):     21,169,540 bp (99.1%)\n",
      "  Read 1:    10,548,876 bp\n",
      "  Read 2:    10,620,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 48304 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t766.0\t0\t22\n",
      "4\t48282\t191.5\t0\t48282\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12ZW3B_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12ZW3B_209_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12ZW3B_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12ZW3B_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12ZW3B_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/12ZW3B_209_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12ZW3B_48_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/12ZW3B_209_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        19,511 reads @  46.9 µs/read;   1.28 M reads/minute\n",
      "Finished in 0.92 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,511\n",
      "  Read 1 with adapter:                  17,963 (92.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        19,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,479,351 bp\n",
      "  Read 1:     4,312,688 bp\n",
      "  Read 2:     4,166,663 bp\n",
      "Total written (filtered):      8,407,659 bp (99.2%)\n",
      "  Read 1:     4,240,996 bp\n",
      "  Read 2:     4,166,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 17963 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t160\t304.9\t0\t160\n",
      "4\t17803\t76.2\t0\t17803\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1IMOSV_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1IMOSV_210_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1IMOSV_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1IMOSV_210_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1IMOSV_49_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1IMOSV_210_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1IMOSV_49_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1IMOSV_210_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         8,849 reads @  55.4 µs/read;   1.08 M reads/minute\n",
      "Finished in 0.50 s (56 µs/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,849\n",
      "  Read 1 with adapter:                   4,606 (52.1%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,849 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,858,998 bp\n",
      "  Read 1:     1,950,881 bp\n",
      "  Read 2:     1,908,117 bp\n",
      "Total written (filtered):      3,840,529 bp (99.5%)\n",
      "  Read 1:     1,932,418 bp\n",
      "  Read 2:     1,908,111 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 4606 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t138.3\t0\t22\n",
      "4\t4583\t34.6\t0\t4583\n",
      "65\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t2.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1Y2WWO_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1Y2WWO_261_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1Y2WWO_100_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1Y2WWO_261_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1Y2WWO_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/1Y2WWO_261_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1Y2WWO_100_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/1Y2WWO_261_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:04        71,633 reads @  56.2 µs/read;   1.07 M reads/minute\n",
      "Finished in 4.03 s (56 µs/read; 1.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,633\n",
      "  Read 1 with adapter:                  36,670 (51.2%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,633 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,189,795 bp\n",
      "  Read 1:    15,906,407 bp\n",
      "  Read 2:    15,283,388 bp\n",
      "Total written (filtered):     31,042,940 bp (99.5%)\n",
      "  Read 1:    15,759,821 bp\n",
      "  Read 2:    15,283,119 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 36670 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t170\t1119.3\t0\t170\n",
      "4\t36497\t279.8\t0\t36497\n",
      "27\t2\t0.0\t0\t2\n",
      "34\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "84\t1\t0.0\t0\t1\n",
      "185\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2FPVDT_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2FPVDT_305_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2FPVDT_144_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2FPVDT_305_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2FPVDT_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2FPVDT_305_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2FPVDT_144_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2FPVDT_305_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:04        79,901 reads @  52.1 µs/read;   1.15 M reads/minute\n",
      "Finished in 4.17 s (52 µs/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,901\n",
      "  Read 1 with adapter:                  43,420 (54.3%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        79,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,712,235 bp\n",
      "  Read 1:    17,659,777 bp\n",
      "  Read 2:    17,052,458 bp\n",
      "Total written (filtered):     34,538,616 bp (99.5%)\n",
      "  Read 1:    17,486,170 bp\n",
      "  Read 2:    17,052,446 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 43420 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t73\t1248.5\t0\t73\n",
      "4\t43347\t312.1\t0\t43347\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t19.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2I7SIQ_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2I7SIQ_164_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2I7SIQ_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2I7SIQ_164_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2I7SIQ_3_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2I7SIQ_164_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2I7SIQ_3_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2I7SIQ_164_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        36,270 reads @  47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 1.71 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,270\n",
      "  Read 1 with adapter:                  24,351 (67.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,270 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,803,617 bp\n",
      "  Read 1:     8,035,264 bp\n",
      "  Read 2:     7,768,353 bp\n",
      "Total written (filtered):     15,705,939 bp (99.4%)\n",
      "  Read 1:     7,937,586 bp\n",
      "  Read 2:     7,768,353 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 24351 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t24\t566.7\t0\t24\n",
      "4\t24322\t141.7\t0\t24322\n",
      "27\t2\t0.0\t0\t2\n",
      "62\t1\t0.0\t0\t1\n",
      "93\t1\t0.0\t0\t1\n",
      "109\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2M59XO_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2M59XO_187_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2M59XO_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2M59XO_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2M59XO_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2M59XO_187_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2M59XO_26_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2M59XO_187_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:19       383,470 reads @  49.8 µs/read;   1.20 M reads/minute\n",
      "Finished in 19.11 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            383,470\n",
      "  Read 1 with adapter:                 181,806 (47.4%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       383,470 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,508,616 bp\n",
      "  Read 1:    83,633,468 bp\n",
      "  Read 2:    82,875,148 bp\n",
      "Total written (filtered):    165,781,156 bp (99.6%)\n",
      "  Read 1:    82,906,057 bp\n",
      "  Read 2:    82,875,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 181806 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t285\t5991.7\t0\t285\n",
      "4\t181513\t1497.9\t0\t181513\n",
      "5\t2\t374.5\t0\t2\n",
      "18\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "175\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t93.6\t0\t1\n",
      "43\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2ZT5IF_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2ZT5IF_190_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2ZT5IF_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2ZT5IF_190_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2ZT5IF_29_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/2ZT5IF_190_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2ZT5IF_29_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/2ZT5IF_190_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        49,462 reads @  50.4 µs/read;   1.19 M reads/minute\n",
      "Finished in 2.50 s (51 µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,462\n",
      "  Read 1 with adapter:                  37,025 (74.9%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,462 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,534,626 bp\n",
      "  Read 1:    10,982,600 bp\n",
      "  Read 2:    10,552,026 bp\n",
      "Total written (filtered):     21,386,058 bp (99.3%)\n",
      "  Read 1:    10,834,342 bp\n",
      "  Read 2:    10,551,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 37025 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t772.8\t0\t51\n",
      "4\t36972\t193.2\t0\t36972\n",
      "28\t1\t0.0\t0\t1\n",
      "189\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t12.1\t0\t2\n",
      "110\t1\t0.0\t0\t1\n",
      "188\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ALGNE_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ALGNE_303_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ALGNE_142_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ALGNE_303_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ALGNE_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ALGNE_303_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ALGNE_142_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ALGNE_303_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:18       395,575 reads @  46.0 µs/read;   1.30 M reads/minute\n",
      "Finished in 18.20 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            395,575\n",
      "  Read 1 with adapter:                 337,536 (85.3%)\n",
      "  Read 2 with adapter:                       8 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       395,575 (100.0%)\n",
      "\n",
      "Total basepairs processed:   171,955,103 bp\n",
      "  Read 1:    85,534,626 bp\n",
      "  Read 2:    86,420,477 bp\n",
      "Total written (filtered):    170,605,915 bp (99.2%)\n",
      "  Read 1:    84,185,862 bp\n",
      "  Read 2:    86,420,053 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 337536 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2102\t6180.9\t0\t2102\n",
      "4\t335425\t1545.2\t0\t335425\n",
      "17\t1\t0.0\t0\t1\n",
      "34\t2\t0.0\t0\t2\n",
      "79\t2\t0.0\t0\t2\n",
      "87\t1\t0.0\t0\t1\n",
      "108\t1\t0.0\t0\t1\n",
      "119\t1\t0.0\t0\t1\n",
      "201\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 8 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t96.6\t0\t1\n",
      "8\t1\t6.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "28\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "74\t1\t0.0\t0\t1\n",
      "80\t1\t0.0\t0\t1\n",
      "172\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3CGEF6_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3CGEF6_265_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3CGEF6_104_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3CGEF6_265_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3CGEF6_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3CGEF6_265_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3CGEF6_104_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3CGEF6_265_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        67,627 reads @  49.8 µs/read;   1.21 M reads/minute\n",
      "Finished in 3.37 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,627\n",
      "  Read 1 with adapter:                   6,239 (9.2%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,627 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,375,776 bp\n",
      "  Read 1:    14,881,045 bp\n",
      "  Read 2:    14,494,731 bp\n",
      "Total written (filtered):     29,350,729 bp (99.9%)\n",
      "  Read 1:    14,856,207 bp\n",
      "  Read 2:    14,494,522 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 6239 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t1056.7\t0\t118\n",
      "4\t6121\t264.2\t0\t6121\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "103\t1\t0.0\t0\t1\n",
      "106\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3DFM85_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3DFM85_215_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3DFM85_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3DFM85_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3DFM85_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3DFM85_215_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3DFM85_54_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3DFM85_215_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        22,916 reads @  49.2 µs/read;   1.22 M reads/minute\n",
      "Finished in 1.13 s (49 µs/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,916\n",
      "  Read 1 with adapter:                  16,459 (71.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,916 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,954,042 bp\n",
      "  Read 1:     5,044,217 bp\n",
      "  Read 2:     4,909,825 bp\n",
      "Total written (filtered):      9,888,221 bp (99.3%)\n",
      "  Read 1:     4,978,396 bp\n",
      "  Read 2:     4,909,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16459 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t15\t358.1\t0\t15\n",
      "4\t16444\t89.5\t0\t16444\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3XK7UB_153_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3XK7UB_314_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3XK7UB_153_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3XK7UB_314_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3XK7UB_153_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3XK7UB_314_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3XK7UB_153_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3XK7UB_314_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:15       335,450 reads @  46.2 µs/read;   1.30 M reads/minute\n",
      "Finished in 15.52 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            335,450\n",
      "  Read 1 with adapter:                  72,609 (21.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       335,450 (100.0%)\n",
      "\n",
      "Total basepairs processed:   146,090,984 bp\n",
      "  Read 1:    73,503,994 bp\n",
      "  Read 2:    72,586,990 bp\n",
      "Total written (filtered):    145,800,923 bp (99.8%)\n",
      "  Read 1:    73,213,973 bp\n",
      "  Read 2:    72,586,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 72609 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t675\t5241.4\t0\t675\n",
      "4\t71930\t1310.4\t0\t71930\n",
      "11\t1\t0.1\t0\t1\n",
      "22\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "190\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t81.9\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ZQWO3_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ZQWO3_203_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ZQWO3_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ZQWO3_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ZQWO3_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/3ZQWO3_203_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ZQWO3_42_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/3ZQWO3_203_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        39,380 reads @  46.8 µs/read;   1.28 M reads/minute\n",
      "Finished in 1.85 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,380\n",
      "  Read 1 with adapter:                  33,348 (84.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,380 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,162,091 bp\n",
      "  Read 1:     8,630,145 bp\n",
      "  Read 2:     8,531,946 bp\n",
      "Total written (filtered):     17,028,516 bp (99.2%)\n",
      "  Read 1:     8,496,570 bp\n",
      "  Read 2:     8,531,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 33348 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t615.3\t0\t33\n",
      "4\t33314\t153.8\t0\t33314\n",
      "220\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4JX8MN_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4JX8MN_301_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4JX8MN_140_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4JX8MN_301_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4JX8MN_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4JX8MN_301_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4JX8MN_140_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4JX8MN_301_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        45,505 reads @  49.2 µs/read;   1.22 M reads/minute\n",
      "Finished in 2.24 s (49 µs/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,505\n",
      "  Read 1 with adapter:                  32,783 (72.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,505 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,781,662 bp\n",
      "  Read 1:     9,968,809 bp\n",
      "  Read 2:     9,812,853 bp\n",
      "Total written (filtered):     19,650,617 bp (99.3%)\n",
      "  Read 1:     9,837,764 bp\n",
      "  Read 2:     9,812,853 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 32783 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t87\t711.0\t0\t87\n",
      "4\t32696\t177.8\t0\t32696\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4L9I67_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4L9I67_311_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4L9I67_150_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4L9I67_311_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4L9I67_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4L9I67_311_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4L9I67_150_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4L9I67_311_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         9,700 reads @  45.0 µs/read;   1.33 M reads/minute\n",
      "Finished in 0.44 s (46 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,700\n",
      "  Read 1 with adapter:                   4,762 (49.1%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         9,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,212,260 bp\n",
      "  Read 1:     2,134,584 bp\n",
      "  Read 2:     2,077,676 bp\n",
      "Total written (filtered):      4,193,141 bp (99.5%)\n",
      "  Read 1:     2,115,565 bp\n",
      "  Read 2:     2,077,576 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 4762 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t29\t151.6\t0\t29\n",
      "4\t4733\t37.9\t0\t4733\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "100\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4NU0Q9_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4NU0Q9_227_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4NU0Q9_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4NU0Q9_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4NU0Q9_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4NU0Q9_227_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4NU0Q9_66_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4NU0Q9_227_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        27,401 reads @  46.2 µs/read;   1.30 M reads/minute\n",
      "Finished in 1.27 s (46 µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,401\n",
      "  Read 1 with adapter:                  19,125 (69.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        27,401 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,903,692 bp\n",
      "  Read 1:     6,003,174 bp\n",
      "  Read 2:     5,900,518 bp\n",
      "Total written (filtered):     11,827,296 bp (99.4%)\n",
      "  Read 1:     5,926,778 bp\n",
      "  Read 2:     5,900,518 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 19125 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t313\t428.1\t0\t313\n",
      "4\t18811\t107.0\t0\t18811\n",
      "213\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4O2DTB_131_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4O2DTB_292_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4O2DTB_131_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4O2DTB_292_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4O2DTB_131_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/4O2DTB_292_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4O2DTB_131_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/4O2DTB_292_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:09       226,321 reads @  44.0 µs/read;   1.36 M reads/minute\n",
      "Finished in 9.97 s (44 µs/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            226,321\n",
      "  Read 1 with adapter:                 224,107 (99.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       226,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    98,372,391 bp\n",
      "  Read 1:    49,805,867 bp\n",
      "  Read 2:    48,566,524 bp\n",
      "Total written (filtered):     97,475,732 bp (99.1%)\n",
      "  Read 1:    48,909,208 bp\n",
      "  Read 2:    48,566,524 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 224107 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t319\t3536.3\t0\t319\n",
      "4\t223782\t884.1\t0\t223782\n",
      "27\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "141\t1\t0.0\t0\t1\n",
      "154\t1\t0.0\t0\t1\n",
      "193\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/51LN9N_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/51LN9N_204_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/51LN9N_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/51LN9N_204_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/51LN9N_43_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/51LN9N_204_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/51LN9N_43_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/51LN9N_204_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:11       272,853 reads @  41.7 µs/read;   1.44 M reads/minute\n",
      "Finished in 11.40 s (42 µs/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            272,853\n",
      "  Read 1 with adapter:                 269,013 (98.6%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       272,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:   118,493,124 bp\n",
      "  Read 1:    58,974,300 bp\n",
      "  Read 2:    59,518,824 bp\n",
      "Total written (filtered):    117,416,571 bp (99.1%)\n",
      "  Read 1:    57,897,779 bp\n",
      "  Read 2:    59,518,792 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 269013 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t70\t4263.3\t0\t70\n",
      "4\t268936\t1065.8\t0\t268936\n",
      "34\t2\t0.0\t0\t2\n",
      "64\t1\t0.0\t0\t1\n",
      "77\t1\t0.0\t0\t1\n",
      "79\t2\t0.0\t0\t2\n",
      "200\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CBV0W_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CBV0W_269_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CBV0W_108_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CBV0W_269_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CBV0W_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CBV0W_269_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CBV0W_108_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CBV0W_269_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        50,725 reads @  47.9 µs/read;   1.25 M reads/minute\n",
      "Finished in 2.44 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,725\n",
      "  Read 1 with adapter:                  39,715 (78.3%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,725 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,052,110 bp\n",
      "  Read 1:    10,927,298 bp\n",
      "  Read 2:    11,124,812 bp\n",
      "Total written (filtered):     21,892,922 bp (99.3%)\n",
      "  Read 1:    10,768,286 bp\n",
      "  Read 2:    11,124,636 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 39715 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t20\t792.6\t0\t20\n",
      "4\t39694\t198.1\t0\t39694\n",
      "176\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "34\t2\t0.0\t0\t2\n",
      "108\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CPP5N_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CPP5N_191_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CPP5N_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CPP5N_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CPP5N_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5CPP5N_191_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CPP5N_30_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5CPP5N_191_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        37,534 reads @  46.5 µs/read;   1.29 M reads/minute\n",
      "Finished in 1.75 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,534\n",
      "  Read 1 with adapter:                  28,741 (76.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,534 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,321,227 bp\n",
      "  Read 1:     8,297,924 bp\n",
      "  Read 2:     8,023,303 bp\n",
      "Total written (filtered):     16,206,282 bp (99.3%)\n",
      "  Read 1:     8,182,979 bp\n",
      "  Read 2:     8,023,303 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 28741 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t118\t586.5\t0\t118\n",
      "4\t28622\t146.6\t0\t28622\n",
      "103\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5Y49IM_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5Y49IM_162_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5Y49IM_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5Y49IM_162_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5Y49IM_1_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5Y49IM_162_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5Y49IM_1_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5Y49IM_162_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        18,252 reads @  50.0 µs/read;   1.20 M reads/minute\n",
      "Finished in 0.92 s (51 µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,252\n",
      "  Read 1 with adapter:                  15,504 (84.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,934,316 bp\n",
      "  Read 1:     3,998,801 bp\n",
      "  Read 2:     3,935,515 bp\n",
      "Total written (filtered):      7,872,342 bp (99.2%)\n",
      "  Read 1:     3,936,827 bp\n",
      "  Read 2:     3,935,515 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15504 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t285.2\t0\t46\n",
      "4\t15454\t71.3\t0\t15454\n",
      "5\t4\t17.8\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5ZP6SF_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5ZP6SF_245_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5ZP6SF_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5ZP6SF_245_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5ZP6SF_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/5ZP6SF_245_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5ZP6SF_84_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/5ZP6SF_245_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        42,747 reads @  50.4 µs/read;   1.19 M reads/minute\n",
      "Finished in 2.16 s (51 µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,747\n",
      "  Read 1 with adapter:                  31,293 (73.2%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,594,270 bp\n",
      "  Read 1:     9,244,235 bp\n",
      "  Read 2:     9,350,035 bp\n",
      "Total written (filtered):     18,469,007 bp (99.3%)\n",
      "  Read 1:     9,119,038 bp\n",
      "  Read 2:     9,349,969 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 31293 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t89\t667.9\t0\t89\n",
      "4\t31203\t167.0\t0\t31203\n",
      "118\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "66\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62A5IJ_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62A5IJ_263_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62A5IJ_102_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62A5IJ_263_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62A5IJ_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62A5IJ_263_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62A5IJ_102_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62A5IJ_263_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        56,130 reads @  48.4 µs/read;   1.24 M reads/minute\n",
      "Finished in 2.72 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,130\n",
      "  Read 1 with adapter:                  25,499 (45.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,130 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,386,314 bp\n",
      "  Read 1:    12,189,143 bp\n",
      "  Read 2:    12,197,171 bp\n",
      "Total written (filtered):     24,284,316 bp (99.6%)\n",
      "  Read 1:    12,087,145 bp\n",
      "  Read 2:    12,197,171 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 25499 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t93\t877.0\t0\t93\n",
      "4\t25405\t219.3\t0\t25405\n",
      "99\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62ARKK_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62ARKK_235_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62ARKK_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62ARKK_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62ARKK_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/62ARKK_235_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62ARKK_74_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/62ARKK_235_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        40,879 reads @  45.4 µs/read;   1.32 M reads/minute\n",
      "Finished in 1.86 s (46 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,879\n",
      "  Read 1 with adapter:                  22,114 (54.1%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        40,879 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,771,428 bp\n",
      "  Read 1:     9,034,746 bp\n",
      "  Read 2:     8,736,682 bp\n",
      "Total written (filtered):     17,682,909 bp (99.5%)\n",
      "  Read 1:     8,946,258 bp\n",
      "  Read 2:     8,736,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 22114 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t638.7\t0\t40\n",
      "4\t22072\t159.7\t0\t22072\n",
      "35\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/63BB0Q_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/63BB0Q_267_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/63BB0Q_106_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/63BB0Q_267_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/63BB0Q_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/63BB0Q_267_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/63BB0Q_106_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/63BB0Q_267_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         3,603 reads @  48.9 µs/read;   1.23 M reads/minute\n",
      "Finished in 0.18 s (51 µs/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,603\n",
      "  Read 1 with adapter:                   1,323 (36.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         3,603 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,567,652 bp\n",
      "  Read 1:       784,747 bp\n",
      "  Read 2:       782,905 bp\n",
      "Total written (filtered):      1,562,361 bp (99.7%)\n",
      "  Read 1:       779,456 bp\n",
      "  Read 2:       782,905 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 1323 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t56.3\t0\t1\n",
      "4\t1322\t14.1\t0\t1322\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/66AYMN_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/66AYMN_178_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/66AYMN_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/66AYMN_178_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/66AYMN_17_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/66AYMN_178_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/66AYMN_17_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/66AYMN_178_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:19       448,483 reads @  43.9 µs/read;   1.37 M reads/minute\n",
      "Finished in 19.70 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            448,483\n",
      "  Read 1 with adapter:                 363,134 (81.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       448,483 (100.0%)\n",
      "\n",
      "Total basepairs processed:   194,938,370 bp\n",
      "  Read 1:    98,705,547 bp\n",
      "  Read 2:    96,232,823 bp\n",
      "Total written (filtered):    193,486,017 bp (99.3%)\n",
      "  Read 1:    97,253,211 bp\n",
      "  Read 2:    96,232,806 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 363134 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1077\t7007.5\t0\t1077\n",
      "4\t362006\t1751.9\t0\t362006\n",
      "7\t1\t27.4\t0\t1\n",
      "9\t45\t1.7\t0\t45\n",
      "59\t1\t0.0\t0\t1\n",
      "87\t1\t0.0\t0\t1\n",
      "145\t1\t0.0\t0\t1\n",
      "178\t1\t0.0\t0\t1\n",
      "200\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t109.5\t0\t1\n",
      "11\t1\t0.1\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6E5YJU_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6E5YJU_200_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6E5YJU_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6E5YJU_200_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6E5YJU_39_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6E5YJU_200_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6E5YJU_39_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6E5YJU_200_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        20,073 reads @  47.4 µs/read;   1.27 M reads/minute\n",
      "Finished in 0.96 s (48 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,073\n",
      "  Read 1 with adapter:                  10,706 (53.3%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        20,073 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,716,877 bp\n",
      "  Read 1:     4,358,658 bp\n",
      "  Read 2:     4,358,219 bp\n",
      "Total written (filtered):      8,673,884 bp (99.5%)\n",
      "  Read 1:     4,315,716 bp\n",
      "  Read 2:     4,358,168 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 10706 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t25\t313.6\t0\t25\n",
      "4\t10680\t78.4\t0\t10680\n",
      "147\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "51\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6JACFU_155_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6JACFU_316_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6JACFU_155_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6JACFU_316_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6JACFU_155_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6JACFU_316_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6JACFU_155_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6JACFU_316_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        33,890 reads @  49.3 µs/read;   1.22 M reads/minute\n",
      "Finished in 1.68 s (49 µs/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,890\n",
      "  Read 1 with adapter:                   6,939 (20.5%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,773,943 bp\n",
      "  Read 1:     7,545,829 bp\n",
      "  Read 2:     7,228,114 bp\n",
      "Total written (filtered):     14,746,035 bp (99.8%)\n",
      "  Read 1:     7,517,960 bp\n",
      "  Read 2:     7,228,075 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 6939 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t529.5\t0\t18\n",
      "4\t6920\t132.4\t0\t6920\n",
      "135\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6MPC62_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6MPC62_171_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6MPC62_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6MPC62_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6MPC62_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6MPC62_171_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6MPC62_10_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6MPC62_171_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        45,093 reads @  48.3 µs/read;   1.24 M reads/minute\n",
      "Finished in 2.19 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,093\n",
      "  Read 1 with adapter:                  21,538 (47.8%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,093 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,608,691 bp\n",
      "  Read 1:     9,924,726 bp\n",
      "  Read 2:     9,683,965 bp\n",
      "Total written (filtered):     19,522,574 bp (99.6%)\n",
      "  Read 1:     9,838,636 bp\n",
      "  Read 2:     9,683,938 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21538 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t704.6\t0\t62\n",
      "4\t21476\t176.1\t0\t21476\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "27\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6PIJWX_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6PIJWX_259_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6PIJWX_98_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6PIJWX_259_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6PIJWX_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/6PIJWX_259_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6PIJWX_98_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/6PIJWX_259_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        37,836 reads @  47.9 µs/read;   1.25 M reads/minute\n",
      "Finished in 1.82 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,836\n",
      "  Read 1 with adapter:                  22,276 (58.9%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,836 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,432,205 bp\n",
      "  Read 1:     8,181,171 bp\n",
      "  Read 2:     8,251,034 bp\n",
      "Total written (filtered):     16,343,007 bp (99.5%)\n",
      "  Read 1:     8,091,979 bp\n",
      "  Read 2:     8,251,028 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 22276 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t591.2\t0\t30\n",
      "4\t22245\t147.8\t0\t22245\n",
      "122\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t9.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7092S6_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7092S6_246_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7092S6_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7092S6_246_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7092S6_85_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7092S6_246_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7092S6_85_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7092S6_246_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        50,930 reads @  47.4 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.42 s (48 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,930\n",
      "  Read 1 with adapter:                  20,941 (41.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,930 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,161,472 bp\n",
      "  Read 1:    10,965,092 bp\n",
      "  Read 2:    11,196,380 bp\n",
      "Total written (filtered):     22,077,878 bp (99.6%)\n",
      "  Read 1:    10,881,498 bp\n",
      "  Read 2:    11,196,380 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 20941 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t170\t795.8\t0\t170\n",
      "4\t20771\t198.9\t0\t20771\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7EO273_97_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7EO273_258_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7EO273_97_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7EO273_258_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7EO273_97_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7EO273_258_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7EO273_97_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7EO273_258_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        48,700 reads @  45.4 µs/read;   1.32 M reads/minute\n",
      "Finished in 2.22 s (46 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,700\n",
      "  Read 1 with adapter:                  16,917 (34.7%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,700 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,205,501 bp\n",
      "  Read 1:    10,691,237 bp\n",
      "  Read 2:    10,514,264 bp\n",
      "Total written (filtered):     21,137,587 bp (99.7%)\n",
      "  Read 1:    10,623,384 bp\n",
      "  Read 2:    10,514,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16917 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t38\t760.9\t0\t38\n",
      "4\t16874\t190.2\t0\t16874\n",
      "41\t1\t0.0\t0\t1\n",
      "45\t2\t0.0\t0\t2\n",
      "46\t1\t0.0\t0\t1\n",
      "66\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "61\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7FXIF8_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7FXIF8_201_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7FXIF8_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7FXIF8_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7FXIF8_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7FXIF8_201_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7FXIF8_40_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7FXIF8_201_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        20,155 reads @  42.7 µs/read;   1.40 M reads/minute\n",
      "Finished in 0.87 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,155\n",
      "  Read 1 with adapter:                  18,799 (93.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        20,155 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,746,862 bp\n",
      "  Read 1:     4,354,779 bp\n",
      "  Read 2:     4,392,083 bp\n",
      "Total written (filtered):      8,671,674 bp (99.1%)\n",
      "  Read 1:     4,279,591 bp\n",
      "  Read 2:     4,392,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 18799 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t314.9\t0\t8\n",
      "4\t18791\t78.7\t0\t18791\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7R1GFQ_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7R1GFQ_279_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7R1GFQ_118_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7R1GFQ_279_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7R1GFQ_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/7R1GFQ_279_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7R1GFQ_118_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/7R1GFQ_279_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        55,203 reads @  46.5 µs/read;   1.29 M reads/minute\n",
      "Finished in 2.57 s (47 µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,203\n",
      "  Read 1 with adapter:                  33,421 (60.5%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,203 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,999,881 bp\n",
      "  Read 1:    11,986,999 bp\n",
      "  Read 2:    12,012,882 bp\n",
      "Total written (filtered):     23,866,441 bp (99.4%)\n",
      "  Read 1:    11,853,559 bp\n",
      "  Read 2:    12,012,882 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 33421 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t245\t862.5\t0\t245\n",
      "4\t33175\t215.6\t0\t33175\n",
      "5\t1\t53.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/82O5JA_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/82O5JA_277_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/82O5JA_116_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/82O5JA_277_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/82O5JA_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/82O5JA_277_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/82O5JA_116_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/82O5JA_277_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:12       293,023 reads @  43.4 µs/read;   1.38 M reads/minute\n",
      "Finished in 12.72 s (43 µs/read; 1.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            293,023\n",
      "  Read 1 with adapter:                 267,083 (91.1%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       293,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:   127,460,452 bp\n",
      "  Read 1:    63,140,988 bp\n",
      "  Read 2:    64,319,464 bp\n",
      "Total written (filtered):    126,391,567 bp (99.2%)\n",
      "  Read 1:    62,072,194 bp\n",
      "  Read 2:    64,319,373 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 267083 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t135\t4578.5\t0\t135\n",
      "4\t266936\t1144.6\t0\t266936\n",
      "9\t1\t1.1\t0\t1\n",
      "35\t9\t0.0\t0\t9\n",
      "142\t1\t0.0\t0\t1\n",
      "179\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t71.5\t0\t1\n",
      "23\t1\t0.0\t0\t1\n",
      "31\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8A0F9A_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8A0F9A_161_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8A0F9A_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8A0F9A_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8A0F9A_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8A0F9A_161_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8A0F9A_0_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8A0F9A_161_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         9,784 reads @  49.2 µs/read;   1.22 M reads/minute\n",
      "Finished in 0.49 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,784\n",
      "  Read 1 with adapter:                   5,059 (51.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         9,784 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,278,491 bp\n",
      "  Read 1:     2,158,665 bp\n",
      "  Read 2:     2,119,826 bp\n",
      "Total written (filtered):      4,258,096 bp (99.5%)\n",
      "  Read 1:     2,138,270 bp\n",
      "  Read 2:     2,119,826 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 5059 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t152.9\t0\t12\n",
      "4\t5046\t38.2\t0\t5046\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8BSFV2_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8BSFV2_216_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8BSFV2_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8BSFV2_216_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8BSFV2_55_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8BSFV2_216_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8BSFV2_55_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8BSFV2_216_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         5,584 reads @  45.9 µs/read;   1.31 M reads/minute\n",
      "Finished in 0.26 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              5,584\n",
      "  Read 1 with adapter:                   5,475 (98.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         5,584 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,439,710 bp\n",
      "  Read 1:     1,226,513 bp\n",
      "  Read 2:     1,213,197 bp\n",
      "Total written (filtered):      2,417,488 bp (99.1%)\n",
      "  Read 1:     1,204,466 bp\n",
      "  Read 2:     1,213,022 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 5475 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t87.2\t0\t1\n",
      "4\t5473\t21.8\t0\t5473\n",
      "152\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8ETYA3_119_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8ETYA3_280_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8ETYA3_119_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8ETYA3_280_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8ETYA3_119_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8ETYA3_280_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8ETYA3_119_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8ETYA3_280_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:16       381,962 reads @  42.9 µs/read;   1.40 M reads/minute\n",
      "Finished in 16.40 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            381,962\n",
      "  Read 1 with adapter:                 168,422 (44.1%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       381,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:   166,201,112 bp\n",
      "  Read 1:    84,804,395 bp\n",
      "  Read 2:    81,396,717 bp\n",
      "Total written (filtered):    165,527,343 bp (99.6%)\n",
      "  Read 1:    84,130,686 bp\n",
      "  Read 2:    81,396,657 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 168422 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t913\t5968.2\t0\t913\n",
      "4\t167478\t1492.0\t0\t167478\n",
      "5\t1\t373.0\t0\t1\n",
      "9\t19\t1.5\t0\t19\n",
      "23\t1\t0.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "54\t1\t0.0\t0\t1\n",
      "77\t1\t0.0\t0\t1\n",
      "88\t1\t0.0\t0\t1\n",
      "113\t1\t0.0\t0\t1\n",
      "116\t1\t0.0\t0\t1\n",
      "119\t1\t0.0\t0\t1\n",
      "194\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8HNX4O_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8HNX4O_238_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8HNX4O_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8HNX4O_238_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8HNX4O_77_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8HNX4O_238_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8HNX4O_77_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8HNX4O_238_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        55,735 reads @  46.9 µs/read;   1.28 M reads/minute\n",
      "Finished in 2.62 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,735\n",
      "  Read 1 with adapter:                  29,497 (52.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,735 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,237,756 bp\n",
      "  Read 1:    12,209,925 bp\n",
      "  Read 2:    12,027,831 bp\n",
      "Total written (filtered):     24,119,790 bp (99.5%)\n",
      "  Read 1:    12,091,959 bp\n",
      "  Read 2:    12,027,831 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 29497 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t98\t870.9\t0\t98\n",
      "4\t29397\t217.7\t0\t29397\n",
      "5\t1\t54.4\t0\t1\n",
      "79\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8MIL3L_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8MIL3L_170_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8MIL3L_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8MIL3L_170_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8MIL3L_9_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8MIL3L_170_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8MIL3L_9_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8MIL3L_170_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        53,388 reads @  45.0 µs/read;   1.33 M reads/minute\n",
      "Finished in 2.41 s (45 µs/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,388\n",
      "  Read 1 with adapter:                  24,289 (45.5%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,388 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,215,651 bp\n",
      "  Read 1:    11,507,068 bp\n",
      "  Read 2:    11,708,583 bp\n",
      "Total written (filtered):     23,118,473 bp (99.6%)\n",
      "  Read 1:    11,409,975 bp\n",
      "  Read 2:    11,708,498 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 24289 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t63\t834.2\t0\t63\n",
      "4\t24226\t208.5\t0\t24226\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t3\t13.0\t0\t3\n",
      "33\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8QR9US_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8QR9US_197_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8QR9US_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8QR9US_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8QR9US_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8QR9US_197_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8QR9US_36_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8QR9US_197_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        68,209 reads @  46.8 µs/read;   1.28 M reads/minute\n",
      "Finished in 3.19 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,209\n",
      "  Read 1 with adapter:                  34,602 (50.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        68,209 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,640,492 bp\n",
      "  Read 1:    15,144,469 bp\n",
      "  Read 2:    14,496,023 bp\n",
      "Total written (filtered):     29,501,992 bp (99.5%)\n",
      "  Read 1:    15,005,969 bp\n",
      "  Read 2:    14,496,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 34602 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t78\t1065.8\t0\t78\n",
      "4\t34523\t266.4\t0\t34523\n",
      "174\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8W6ADJ_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8W6ADJ_244_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8W6ADJ_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8W6ADJ_244_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8W6ADJ_83_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/8W6ADJ_244_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8W6ADJ_83_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/8W6ADJ_244_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        63,227 reads @  49.8 µs/read;   1.20 M reads/minute\n",
      "Finished in 3.16 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,227\n",
      "  Read 1 with adapter:                  42,036 (66.5%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        63,227 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,520,815 bp\n",
      "  Read 1:    13,731,835 bp\n",
      "  Read 2:    13,788,980 bp\n",
      "Total written (filtered):     27,352,454 bp (99.4%)\n",
      "  Read 1:    13,563,474 bp\n",
      "  Read 2:    13,788,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 42036 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t65\t987.9\t0\t65\n",
      "4\t41969\t247.0\t0\t41969\n",
      "115\t1\t0.0\t0\t1\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/94F4JW_103_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/94F4JW_264_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/94F4JW_103_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/94F4JW_264_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/94F4JW_103_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/94F4JW_264_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/94F4JW_103_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/94F4JW_264_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        29,782 reads @  47.3 µs/read;   1.27 M reads/minute\n",
      "Finished in 1.41 s (47 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,782\n",
      "  Read 1 with adapter:                  15,377 (51.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        29,782 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,981,571 bp\n",
      "  Read 1:     6,511,252 bp\n",
      "  Read 2:     6,470,319 bp\n",
      "Total written (filtered):     12,919,683 bp (99.5%)\n",
      "  Read 1:     6,449,364 bp\n",
      "  Read 2:     6,470,319 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15377 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t30\t465.3\t0\t30\n",
      "4\t15341\t116.3\t0\t15341\n",
      "55\t1\t0.0\t0\t1\n",
      "73\t3\t0.0\t0\t3\n",
      "74\t1\t0.0\t0\t1\n",
      "86\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/95NCMG_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/95NCMG_226_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/95NCMG_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/95NCMG_226_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/95NCMG_65_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/95NCMG_226_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/95NCMG_65_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/95NCMG_226_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        58,261 reads @  46.0 µs/read;   1.31 M reads/minute\n",
      "Finished in 2.69 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,261\n",
      "  Read 1 with adapter:                  45,389 (77.9%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        58,261 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,331,177 bp\n",
      "  Read 1:    12,822,834 bp\n",
      "  Read 2:    12,508,343 bp\n",
      "Total written (filtered):     25,149,695 bp (99.3%)\n",
      "  Read 1:    12,641,359 bp\n",
      "  Read 2:    12,508,336 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 45389 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t81\t910.3\t0\t81\n",
      "4\t45308\t227.6\t0\t45308\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t3.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/97HREF_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/97HREF_309_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/97HREF_148_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/97HREF_309_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/97HREF_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/97HREF_309_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/97HREF_148_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/97HREF_309_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:16       364,301 reads @  43.9 µs/read;   1.37 M reads/minute\n",
      "Finished in 16.01 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            364,301\n",
      "  Read 1 with adapter:                  77,959 (21.4%)\n",
      "  Read 2 with adapter:                      16 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       364,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:   158,534,450 bp\n",
      "  Read 1:    78,416,971 bp\n",
      "  Read 2:    80,117,479 bp\n",
      "Total written (filtered):    158,222,058 bp (99.8%)\n",
      "  Read 1:    78,104,675 bp\n",
      "  Read 2:    80,117,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 77959 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t157\t5692.2\t0\t157\n",
      "4\t77794\t1423.1\t0\t77794\n",
      "6\t1\t88.9\t0\t1\n",
      "9\t1\t1.4\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "65\t1\t0.0\t0\t1\n",
      "87\t1\t0.0\t0\t1\n",
      "112\t1\t0.0\t0\t1\n",
      "128\t1\t0.0\t0\t1\n",
      "207\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 16 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t16\t88.9\t0\t16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9FGVWQ_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9FGVWQ_307_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9FGVWQ_146_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9FGVWQ_307_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9FGVWQ_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9FGVWQ_307_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9FGVWQ_146_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9FGVWQ_307_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        39,094 reads @  48.7 µs/read;   1.23 M reads/minute\n",
      "Finished in 1.91 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,094\n",
      "  Read 1 with adapter:                  11,067 (28.3%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,094 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,986,321 bp\n",
      "  Read 1:     8,566,072 bp\n",
      "  Read 2:     8,420,249 bp\n",
      "Total written (filtered):     16,942,106 bp (99.7%)\n",
      "  Read 1:     8,521,881 bp\n",
      "  Read 2:     8,420,225 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 11067 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t610.8\t0\t77\n",
      "4\t10990\t152.7\t0\t10990\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t4\t9.5\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MD25S_111_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MD25S_272_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MD25S_111_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MD25S_272_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MD25S_111_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MD25S_272_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MD25S_111_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MD25S_272_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:04        89,000 reads @  46.0 µs/read;   1.30 M reads/minute\n",
      "Finished in 4.10 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,000\n",
      "  Read 1 with adapter:                  25,816 (29.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        89,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,679,616 bp\n",
      "  Read 1:    19,585,241 bp\n",
      "  Read 2:    19,094,375 bp\n",
      "Total written (filtered):     38,576,386 bp (99.7%)\n",
      "  Read 1:    19,482,107 bp\n",
      "  Read 2:    19,094,279 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 25816 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t255\t1390.6\t0\t255\n",
      "4\t25556\t347.7\t0\t25556\n",
      "9\t4\t0.3\t0\t4\n",
      "109\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "31\t1\t0.0\t0\t1\n",
      "65\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MQ67J_154_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MQ67J_315_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MQ67J_154_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MQ67J_315_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MQ67J_154_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9MQ67J_315_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MQ67J_154_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9MQ67J_315_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        56,569 reads @  45.8 µs/read;   1.31 M reads/minute\n",
      "Finished in 2.59 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,569\n",
      "  Read 1 with adapter:                  18,400 (32.5%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        56,569 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,620,515 bp\n",
      "  Read 1:    12,252,401 bp\n",
      "  Read 2:    12,368,114 bp\n",
      "Total written (filtered):     24,546,798 bp (99.7%)\n",
      "  Read 1:    12,178,696 bp\n",
      "  Read 2:    12,368,102 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 18400 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t883.9\t0\t59\n",
      "4\t18337\t221.0\t0\t18337\n",
      "23\t2\t0.0\t0\t2\n",
      "61\t1\t0.0\t0\t1\n",
      "73\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t13.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9V6FS5_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9V6FS5_243_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9V6FS5_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9V6FS5_243_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9V6FS5_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9V6FS5_243_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9V6FS5_82_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9V6FS5_243_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:14       332,448 reads @  43.6 µs/read;   1.38 M reads/minute\n",
      "Finished in 14.51 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            332,448\n",
      "  Read 1 with adapter:                 104,544 (31.4%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       332,448 (100.0%)\n",
      "\n",
      "Total basepairs processed:   144,899,205 bp\n",
      "  Read 1:    73,485,468 bp\n",
      "  Read 2:    71,413,737 bp\n",
      "Total written (filtered):    144,481,313 bp (99.7%)\n",
      "  Read 1:    73,067,625 bp\n",
      "  Read 2:    71,413,688 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 104544 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t402\t5194.5\t0\t402\n",
      "4\t104140\t1298.6\t0\t104140\n",
      "29\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t81.2\t0\t2\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9X2OPW_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9X2OPW_202_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9X2OPW_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9X2OPW_202_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9X2OPW_41_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/9X2OPW_202_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9X2OPW_41_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/9X2OPW_202_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        21,975 reads @  43.9 µs/read;   1.37 M reads/minute\n",
      "Finished in 0.97 s (44 µs/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,975\n",
      "  Read 1 with adapter:                  12,669 (57.7%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,975 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,573,346 bp\n",
      "  Read 1:     4,890,657 bp\n",
      "  Read 2:     4,682,689 bp\n",
      "Total written (filtered):      9,522,654 bp (99.5%)\n",
      "  Read 1:     4,840,003 bp\n",
      "  Read 2:     4,682,651 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 12669 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t22\t343.4\t0\t22\n",
      "4\t12647\t85.8\t0\t12647\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "38\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A107EY_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A107EY_241_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A107EY_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A107EY_241_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A107EY_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A107EY_241_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A107EY_80_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A107EY_241_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:10       258,460 reads @  41.3 µs/read;   1.45 M reads/minute\n",
      "Finished in 10.67 s (41 µs/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            258,460\n",
      "  Read 1 with adapter:                 256,960 (99.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       258,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:   112,236,948 bp\n",
      "  Read 1:    55,599,834 bp\n",
      "  Read 2:    56,637,114 bp\n",
      "Total written (filtered):    111,209,124 bp (99.1%)\n",
      "  Read 1:    54,572,010 bp\n",
      "  Read 2:    56,637,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 256960 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t121\t4038.4\t0\t121\n",
      "4\t256838\t1009.6\t0\t256838\n",
      "109\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A2FX4X_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A2FX4X_184_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A2FX4X_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A2FX4X_184_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A2FX4X_23_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A2FX4X_184_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A2FX4X_23_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A2FX4X_184_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        50,770 reads @  47.4 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.41 s (47 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,770\n",
      "  Read 1 with adapter:                  17,841 (35.1%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,770 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,081,066 bp\n",
      "  Read 1:    10,985,287 bp\n",
      "  Read 2:    11,095,779 bp\n",
      "Total written (filtered):     22,009,682 bp (99.7%)\n",
      "  Read 1:    10,913,920 bp\n",
      "  Read 2:    11,095,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 17841 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t168\t793.3\t0\t168\n",
      "4\t17672\t198.3\t0\t17672\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t49.6\t0\t1\n",
      "6\t2\t12.4\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A454WP_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A454WP_212_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A454WP_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A454WP_212_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A454WP_51_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A454WP_212_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A454WP_51_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A454WP_212_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        22,415 reads @  45.9 µs/read;   1.31 M reads/minute\n",
      "Finished in 1.03 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,415\n",
      "  Read 1 with adapter:                   9,412 (42.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,415 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,757,933 bp\n",
      "  Read 1:     4,921,963 bp\n",
      "  Read 2:     4,835,970 bp\n",
      "Total written (filtered):      9,720,305 bp (99.6%)\n",
      "  Read 1:     4,884,341 bp\n",
      "  Read 2:     4,835,964 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 9412 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t26\t350.2\t0\t26\n",
      "4\t9386\t87.6\t0\t9386\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t5.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A7DEJE_157_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A7DEJE_318_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A7DEJE_157_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A7DEJE_318_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A7DEJE_157_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/A7DEJE_318_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A7DEJE_157_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/A7DEJE_318_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        44,747 reads @  46.6 µs/read;   1.29 M reads/minute\n",
      "Finished in 2.09 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,747\n",
      "  Read 1 with adapter:                  21,817 (48.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,747 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,433,497 bp\n",
      "  Read 1:     9,630,447 bp\n",
      "  Read 2:     9,803,050 bp\n",
      "Total written (filtered):     19,346,259 bp (99.6%)\n",
      "  Read 1:     9,543,209 bp\n",
      "  Read 2:     9,803,050 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21817 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t35\t699.2\t0\t35\n",
      "4\t21781\t174.8\t0\t21781\n",
      "9\t1\t0.2\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AFG7YZ_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AFG7YZ_167_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AFG7YZ_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AFG7YZ_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AFG7YZ_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AFG7YZ_167_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AFG7YZ_6_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AFG7YZ_167_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         6,819 reads @  41.4 µs/read;   1.45 M reads/minute\n",
      "Finished in 0.29 s (42 µs/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,819\n",
      "  Read 1 with adapter:                     220 (3.2%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         6,819 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,962,847 bp\n",
      "  Read 1:     1,494,266 bp\n",
      "  Read 2:     1,468,581 bp\n",
      "Total written (filtered):      2,961,969 bp (100.0%)\n",
      "  Read 1:     1,493,388 bp\n",
      "  Read 2:     1,468,581 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 220 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t106.5\t0\t2\n",
      "4\t218\t26.6\t0\t218\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AL6GY9_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AL6GY9_194_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AL6GY9_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AL6GY9_194_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AL6GY9_33_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/AL6GY9_194_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AL6GY9_33_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/AL6GY9_194_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        50,037 reads @  47.4 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.38 s (48 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,037\n",
      "  Read 1 with adapter:                  36,669 (73.3%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        50,037 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,755,224 bp\n",
      "  Read 1:    10,874,523 bp\n",
      "  Read 2:    10,880,701 bp\n",
      "Total written (filtered):     21,608,515 bp (99.3%)\n",
      "  Read 1:    10,727,859 bp\n",
      "  Read 2:    10,880,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 36669 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t183\t781.8\t0\t183\n",
      "4\t36485\t195.5\t0\t36485\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t12.2\t0\t2\n",
      "33\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/B4P2N8_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/B4P2N8_251_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/B4P2N8_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/B4P2N8_251_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/B4P2N8_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/B4P2N8_251_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/B4P2N8_90_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/B4P2N8_251_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        60,962 reads @  47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.88 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,962\n",
      "  Read 1 with adapter:                  21,410 (35.1%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,530,478 bp\n",
      "  Read 1:    13,202,315 bp\n",
      "  Read 2:    13,328,163 bp\n",
      "Total written (filtered):     26,444,345 bp (99.7%)\n",
      "  Read 1:    13,116,188 bp\n",
      "  Read 2:    13,328,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t59\t952.5\t0\t59\n",
      "4\t21341\t238.1\t0\t21341\n",
      "35\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "42\t1\t0.0\t0\t1\n",
      "45\t3\t0.0\t0\t3\n",
      "50\t1\t0.0\t0\t1\n",
      "71\t1\t0.0\t0\t1\n",
      "77\t1\t0.0\t0\t1\n",
      "140\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t14.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BKS62K_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BKS62K_240_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BKS62K_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BKS62K_240_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BKS62K_79_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BKS62K_240_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BKS62K_79_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BKS62K_240_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        34,620 reads @  52.0 µs/read;   1.15 M reads/minute\n",
      "Finished in 1.80 s (52 µs/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,620\n",
      "  Read 1 with adapter:                  16,732 (48.3%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        34,620 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,063,953 bp\n",
      "  Read 1:     7,552,606 bp\n",
      "  Read 2:     7,511,347 bp\n",
      "Total written (filtered):     14,996,930 bp (99.6%)\n",
      "  Read 1:     7,485,598 bp\n",
      "  Read 2:     7,511,332 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16732 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t540.9\t0\t74\n",
      "4\t16657\t135.2\t0\t16657\n",
      "158\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t540.9\t0\t1\n",
      "6\t2\t8.5\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BL4AQ3_105_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BL4AQ3_266_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BL4AQ3_105_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BL4AQ3_266_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BL4AQ3_105_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BL4AQ3_266_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BL4AQ3_105_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BL4AQ3_266_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        13,486 reads @  48.9 µs/read;   1.23 M reads/minute\n",
      "Finished in 0.66 s (49 µs/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,486\n",
      "  Read 1 with adapter:                   5,313 (39.4%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,877,821 bp\n",
      "  Read 1:     2,921,084 bp\n",
      "  Read 2:     2,956,737 bp\n",
      "Total written (filtered):      5,856,461 bp (99.6%)\n",
      "  Read 1:     2,899,848 bp\n",
      "  Read 2:     2,956,613 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 5313 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t16\t210.7\t0\t16\n",
      "4\t5297\t52.7\t0\t5297\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "124\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BU4IJU_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BU4IJU_249_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BU4IJU_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BU4IJU_249_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BU4IJU_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/BU4IJU_249_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BU4IJU_88_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/BU4IJU_249_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        49,855 reads @  51.9 µs/read;   1.16 M reads/minute\n",
      "Finished in 2.60 s (52 µs/read; 1.15 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,855\n",
      "  Read 1 with adapter:                  40,031 (80.3%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,710,140 bp\n",
      "  Read 1:    11,019,701 bp\n",
      "  Read 2:    10,690,439 bp\n",
      "Total written (filtered):     21,549,982 bp (99.3%)\n",
      "  Read 1:    10,859,549 bp\n",
      "  Read 2:    10,690,433 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 40031 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t779.0\t0\t46\n",
      "4\t39984\t194.7\t0\t39984\n",
      "78\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t12.2\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C0PEHY_121_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C0PEHY_282_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C0PEHY_121_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C0PEHY_282_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C0PEHY_121_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C0PEHY_282_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C0PEHY_121_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C0PEHY_282_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        52,075 reads @  53.1 µs/read;   1.13 M reads/minute\n",
      "Finished in 2.77 s (53 µs/read; 1.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,075\n",
      "  Read 1 with adapter:                  40,074 (77.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        52,075 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,667,944 bp\n",
      "  Read 1:    11,410,733 bp\n",
      "  Read 2:    11,257,211 bp\n",
      "Total written (filtered):     22,507,437 bp (99.3%)\n",
      "  Read 1:    11,250,261 bp\n",
      "  Read 2:    11,257,176 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 40074 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t58\t813.7\t0\t58\n",
      "4\t40014\t203.4\t0\t40014\n",
      "27\t1\t0.0\t0\t1\n",
      "215\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C776ZA_109_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C776ZA_270_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C776ZA_109_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C776ZA_270_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C776ZA_109_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C776ZA_270_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C776ZA_109_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C776ZA_270_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        37,352 reads @  48.7 µs/read;   1.23 M reads/minute\n",
      "Finished in 1.83 s (49 µs/read; 1.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,352\n",
      "  Read 1 with adapter:                  36,490 (97.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        37,352 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,296,952 bp\n",
      "  Read 1:     8,191,938 bp\n",
      "  Read 2:     8,105,014 bp\n",
      "Total written (filtered):     16,150,838 bp (99.1%)\n",
      "  Read 1:     8,045,824 bp\n",
      "  Read 2:     8,105,014 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 36490 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t583.6\t0\t17\n",
      "4\t36472\t145.9\t0\t36472\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C7PEWB_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C7PEWB_213_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C7PEWB_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C7PEWB_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C7PEWB_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/C7PEWB_213_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C7PEWB_52_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/C7PEWB_213_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        44,149 reads @  50.0 µs/read;   1.20 M reads/minute\n",
      "Finished in 2.21 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,149\n",
      "  Read 1 with adapter:                  15,735 (35.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,149 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,214,859 bp\n",
      "  Read 1:     9,804,079 bp\n",
      "  Read 2:     9,410,780 bp\n",
      "Total written (filtered):     19,151,877 bp (99.7%)\n",
      "  Read 1:     9,741,184 bp\n",
      "  Read 2:     9,410,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15735 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t45\t689.8\t0\t45\n",
      "4\t15690\t172.5\t0\t15690\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t689.8\t0\t1\n",
      "84\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/CG40D4_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/CG40D4_181_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/CG40D4_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/CG40D4_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/CG40D4_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/CG40D4_181_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/CG40D4_20_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/CG40D4_181_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        54,897 reads @  44.8 µs/read;   1.34 M reads/minute\n",
      "Finished in 2.46 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,897\n",
      "  Read 1 with adapter:                  53,732 (97.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        54,897 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,910,742 bp\n",
      "  Read 1:    12,138,387 bp\n",
      "  Read 2:    11,772,355 bp\n",
      "Total written (filtered):     23,695,860 bp (99.1%)\n",
      "  Read 1:    11,923,505 bp\n",
      "  Read 2:    11,772,355 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 53732 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t857.8\t0\t46\n",
      "4\t53686\t214.4\t0\t53686\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/D31PBF_152_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/D31PBF_313_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/D31PBF_152_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/D31PBF_313_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/D31PBF_152_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/D31PBF_313_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/D31PBF_152_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/D31PBF_313_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:12       285,478 reads @  45.1 µs/read;   1.33 M reads/minute\n",
      "Finished in 12.90 s (45 µs/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            285,478\n",
      "  Read 1 with adapter:                 279,588 (97.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       285,478 (100.0%)\n",
      "\n",
      "Total basepairs processed:   124,075,531 bp\n",
      "  Read 1:    62,820,448 bp\n",
      "  Read 2:    61,255,083 bp\n",
      "Total written (filtered):    122,957,081 bp (99.1%)\n",
      "  Read 1:    61,701,998 bp\n",
      "  Read 2:    61,255,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 279588 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t254\t4460.6\t0\t254\n",
      "4\t279330\t1115.1\t0\t279330\n",
      "29\t1\t0.0\t0\t1\n",
      "81\t1\t0.0\t0\t1\n",
      "88\t1\t0.0\t0\t1\n",
      "170\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/DALXNM_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/DALXNM_176_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/DALXNM_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/DALXNM_176_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/DALXNM_15_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/DALXNM_176_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/DALXNM_15_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/DALXNM_176_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        18,928 reads @  45.8 µs/read;   1.31 M reads/minute\n",
      "Finished in 0.88 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,928\n",
      "  Read 1 with adapter:                   2,015 (10.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,928 (100.0%)\n",
      "\n",
      "Total basepairs processed:     8,251,197 bp\n",
      "  Read 1:     4,123,995 bp\n",
      "  Read 2:     4,127,202 bp\n",
      "Total written (filtered):      8,243,209 bp (99.9%)\n",
      "  Read 1:     4,116,007 bp\n",
      "  Read 2:     4,127,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 2015 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t295.8\t0\t72\n",
      "4\t1943\t73.9\t0\t1943\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/EAJ8D1_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/EAJ8D1_247_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/EAJ8D1_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/EAJ8D1_247_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/EAJ8D1_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/EAJ8D1_247_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/EAJ8D1_86_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/EAJ8D1_247_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:15       337,297 reads @  47.3 µs/read;   1.27 M reads/minute\n",
      "Finished in 15.95 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            337,297\n",
      "  Read 1 with adapter:                 151,355 (44.9%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       337,297 (100.0%)\n",
      "\n",
      "Total basepairs processed:   146,521,873 bp\n",
      "  Read 1:    74,885,902 bp\n",
      "  Read 2:    71,635,971 bp\n",
      "Total written (filtered):    145,916,073 bp (99.6%)\n",
      "  Read 1:    74,280,465 bp\n",
      "  Read 2:    71,635,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 151355 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t308\t5270.3\t0\t308\n",
      "4\t151045\t1317.6\t0\t151045\n",
      "143\t1\t0.0\t0\t1\n",
      "190\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "164\t1\t0.0\t0\t1\n",
      "199\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F7IP1O_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F7IP1O_221_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F7IP1O_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F7IP1O_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F7IP1O_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F7IP1O_221_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F7IP1O_60_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F7IP1O_221_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        35,329 reads @  47.8 µs/read;   1.26 M reads/minute\n",
      "Finished in 1.70 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,329\n",
      "  Read 1 with adapter:                  13,851 (39.2%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,329 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,336,308 bp\n",
      "  Read 1:     7,600,042 bp\n",
      "  Read 2:     7,736,266 bp\n",
      "Total written (filtered):     15,280,912 bp (99.6%)\n",
      "  Read 1:     7,544,652 bp\n",
      "  Read 2:     7,736,260 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 13851 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t552.0\t0\t14\n",
      "4\t13837\t138.0\t0\t13837\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t8.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F89RLS_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F89RLS_233_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F89RLS_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F89RLS_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F89RLS_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/F89RLS_233_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F89RLS_72_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/F89RLS_233_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        21,363 reads @  51.3 µs/read;   1.17 M reads/minute\n",
      "Finished in 1.11 s (52 µs/read; 1.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,363\n",
      "  Read 1 with adapter:                  14,914 (69.8%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        21,363 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,284,712 bp\n",
      "  Read 1:     4,742,490 bp\n",
      "  Read 2:     4,542,222 bp\n",
      "Total written (filtered):      9,224,925 bp (99.4%)\n",
      "  Read 1:     4,682,810 bp\n",
      "  Read 2:     4,542,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 14914 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t333.8\t0\t17\n",
      "4\t14896\t83.4\t0\t14896\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "107\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FA88B6_123_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FA88B6_284_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FA88B6_123_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FA88B6_284_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FA88B6_123_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FA88B6_284_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FA88B6_123_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FA88B6_284_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        62,958 reads @  48.4 µs/read;   1.24 M reads/minute\n",
      "Finished in 3.06 s (49 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,958\n",
      "  Read 1 with adapter:                  21,751 (34.5%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        62,958 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,375,615 bp\n",
      "  Read 1:    13,672,621 bp\n",
      "  Read 2:    13,702,994 bp\n",
      "Total written (filtered):     27,288,700 bp (99.7%)\n",
      "  Read 1:    13,585,751 bp\n",
      "  Read 2:    13,702,949 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21751 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t134\t983.7\t0\t134\n",
      "4\t21617\t245.9\t0\t21617\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t15.4\t0\t2\n",
      "33\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FSX8UT_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FSX8UT_208_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FSX8UT_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FSX8UT_208_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FSX8UT_47_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/FSX8UT_208_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FSX8UT_47_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/FSX8UT_208_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:14       358,992 reads @  41.3 µs/read;   1.45 M reads/minute\n",
      "Finished in 14.84 s (41 µs/read; 1.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            358,992\n",
      "  Read 1 with adapter:                 245,911 (68.5%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       358,992 (100.0%)\n",
      "\n",
      "Total basepairs processed:   155,834,650 bp\n",
      "  Read 1:    77,217,036 bp\n",
      "  Read 2:    78,617,614 bp\n",
      "Total written (filtered):    154,850,610 bp (99.4%)\n",
      "  Read 1:    76,233,193 bp\n",
      "  Read 2:    78,617,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 245911 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t57\t5609.2\t0\t57\n",
      "4\t245850\t1402.3\t0\t245850\n",
      "9\t1\t1.4\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "79\t1\t0.0\t0\t1\n",
      "157\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t21.9\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "159\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G3VBHP_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G3VBHP_281_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G3VBHP_120_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G3VBHP_281_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G3VBHP_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G3VBHP_281_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G3VBHP_120_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G3VBHP_281_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        46,988 reads @  49.3 µs/read;   1.22 M reads/minute\n",
      "Finished in 2.32 s (49 µs/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,988\n",
      "  Read 1 with adapter:                  30,586 (65.1%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        46,988 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,456,072 bp\n",
      "  Read 1:    10,341,982 bp\n",
      "  Read 2:    10,114,090 bp\n",
      "Total written (filtered):     20,333,547 bp (99.4%)\n",
      "  Read 1:    10,219,494 bp\n",
      "  Read 2:    10,114,053 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 30586 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t51\t734.2\t0\t51\n",
      "4\t30533\t183.5\t0\t30533\n",
      "27\t1\t0.0\t0\t1\n",
      "176\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t11.5\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G7D1L0_101_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G7D1L0_262_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G7D1L0_101_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G7D1L0_262_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G7D1L0_101_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/G7D1L0_262_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G7D1L0_101_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/G7D1L0_262_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        30,122 reads @  47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 1.42 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,122\n",
      "  Read 1 with adapter:                  18,551 (61.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,122 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,083,281 bp\n",
      "  Read 1:     6,570,015 bp\n",
      "  Read 2:     6,513,266 bp\n",
      "Total written (filtered):     13,009,094 bp (99.4%)\n",
      "  Read 1:     6,495,828 bp\n",
      "  Read 2:     6,513,266 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 18551 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t470.7\t0\t18\n",
      "4\t18532\t117.7\t0\t18532\n",
      "5\t1\t29.4\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GRMT7P_147_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GRMT7P_308_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GRMT7P_147_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GRMT7P_308_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GRMT7P_147_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GRMT7P_308_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GRMT7P_147_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GRMT7P_308_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        38,985 reads @  44.7 µs/read;   1.34 M reads/minute\n",
      "Finished in 1.75 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,985\n",
      "  Read 1 with adapter:                   6,462 (16.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,985 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,955,431 bp\n",
      "  Read 1:     8,504,574 bp\n",
      "  Read 2:     8,450,857 bp\n",
      "Total written (filtered):     16,929,484 bp (99.8%)\n",
      "  Read 1:     8,478,832 bp\n",
      "  Read 2:     8,450,652 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 6462 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t223\t609.1\t0\t223\n",
      "4\t6238\t152.3\t0\t6238\n",
      "121\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "98\t1\t0.0\t0\t1\n",
      "107\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GX7LG5_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GX7LG5_250_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GX7LG5_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GX7LG5_250_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GX7LG5_89_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GX7LG5_250_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GX7LG5_89_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GX7LG5_250_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        55,104 reads @  47.2 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.61 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,104\n",
      "  Read 1 with adapter:                  27,029 (49.1%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        55,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,940,854 bp\n",
      "  Read 1:    12,128,166 bp\n",
      "  Read 2:    11,812,688 bp\n",
      "Total written (filtered):     23,832,804 bp (99.5%)\n",
      "  Read 1:    12,020,122 bp\n",
      "  Read 2:    11,812,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 27029 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t105\t861.0\t0\t105\n",
      "4\t26923\t215.2\t0\t26923\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t13.5\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GZ5795_133_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GZ5795_294_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GZ5795_133_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GZ5795_294_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GZ5795_133_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/GZ5795_294_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GZ5795_133_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/GZ5795_294_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        51,716 reads @  47.0 µs/read;   1.28 M reads/minute\n",
      "Finished in 2.44 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,716\n",
      "  Read 1 with adapter:                  20,389 (39.4%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,716 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,490,842 bp\n",
      "  Read 1:    11,333,278 bp\n",
      "  Read 2:    11,157,564 bp\n",
      "Total written (filtered):     22,409,167 bp (99.6%)\n",
      "  Read 1:    11,251,694 bp\n",
      "  Read 2:    11,157,473 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 20389 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t91\t808.1\t0\t91\n",
      "4\t20295\t202.0\t0\t20295\n",
      "5\t1\t50.5\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "99\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "30\t2\t0.0\t0\t2\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HCMB0Y_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HCMB0Y_231_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HCMB0Y_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HCMB0Y_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HCMB0Y_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HCMB0Y_231_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HCMB0Y_70_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HCMB0Y_231_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        48,785 reads @  43.6 µs/read;   1.37 M reads/minute\n",
      "Finished in 2.13 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,785\n",
      "  Read 1 with adapter:                  20,453 (41.9%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,785 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,232,728 bp\n",
      "  Read 1:    10,785,619 bp\n",
      "  Read 2:    10,447,109 bp\n",
      "Total written (filtered):     21,150,623 bp (99.6%)\n",
      "  Read 1:    10,703,743 bp\n",
      "  Read 2:    10,446,880 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 20453 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t31\t762.3\t0\t31\n",
      "4\t20421\t190.6\t0\t20421\n",
      "99\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "91\t1\t0.0\t0\t1\n",
      "138\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HEKP4L_115_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HEKP4L_276_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HEKP4L_115_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HEKP4L_276_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HEKP4L_115_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HEKP4L_276_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HEKP4L_115_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HEKP4L_276_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        13,871 reads @  49.3 µs/read;   1.22 M reads/minute\n",
      "Finished in 0.69 s (50 µs/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,871\n",
      "  Read 1 with adapter:                  10,078 (72.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,871 (100.0%)\n",
      "\n",
      "Total basepairs processed:     6,050,995 bp\n",
      "  Read 1:     3,066,509 bp\n",
      "  Read 2:     2,984,486 bp\n",
      "Total written (filtered):      6,010,704 bp (99.3%)\n",
      "  Read 1:     3,026,218 bp\n",
      "  Read 2:     2,984,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 10078 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t216.7\t0\t21\n",
      "4\t10057\t54.2\t0\t10057\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HH1SXZ_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HH1SXZ_198_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HH1SXZ_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HH1SXZ_198_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HH1SXZ_37_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HH1SXZ_198_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HH1SXZ_37_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HH1SXZ_198_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:15       362,510 reads @  42.5 µs/read;   1.41 M reads/minute\n",
      "Finished in 15.42 s (43 µs/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            362,510\n",
      "  Read 1 with adapter:                 298,109 (82.2%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       362,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:   157,449,126 bp\n",
      "  Read 1:    80,486,764 bp\n",
      "  Read 2:    76,962,362 bp\n",
      "Total written (filtered):    156,257,040 bp (99.2%)\n",
      "  Read 1:    79,294,895 bp\n",
      "  Read 2:    76,962,145 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 298109 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t754\t5664.2\t0\t754\n",
      "4\t297351\t1416.1\t0\t297351\n",
      "5\t1\t354.0\t0\t1\n",
      "9\t2\t1.4\t0\t2\n",
      "180\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t88.5\t0\t2\n",
      "38\t1\t0.0\t0\t1\n",
      "167\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HMPJUC_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HMPJUC_256_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HMPJUC_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HMPJUC_256_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HMPJUC_95_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/HMPJUC_256_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HMPJUC_95_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/HMPJUC_256_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        42,202 reads @  48.1 µs/read;   1.25 M reads/minute\n",
      "Finished in 2.04 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,202\n",
      "  Read 1 with adapter:                  41,244 (97.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,344,400 bp\n",
      "  Read 1:     9,369,599 bp\n",
      "  Read 2:     8,974,801 bp\n",
      "Total written (filtered):     18,179,800 bp (99.1%)\n",
      "  Read 1:     9,204,999 bp\n",
      "  Read 2:     8,974,801 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 41244 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t376\t659.4\t0\t376\n",
      "4\t40868\t164.9\t0\t40868\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/IXJOV0_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/IXJOV0_273_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/IXJOV0_112_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/IXJOV0_273_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/IXJOV0_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/IXJOV0_273_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/IXJOV0_112_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/IXJOV0_273_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        33,648 reads @  47.0 µs/read;   1.28 M reads/minute\n",
      "Finished in 1.59 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,648\n",
      "  Read 1 with adapter:                  32,856 (97.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,634,310 bp\n",
      "  Read 1:     7,374,239 bp\n",
      "  Read 2:     7,260,071 bp\n",
      "Total written (filtered):     14,502,911 bp (99.1%)\n",
      "  Read 1:     7,242,840 bp\n",
      "  Read 2:     7,260,071 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 32856 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t76\t525.8\t0\t76\n",
      "4\t32778\t131.4\t0\t32778\n",
      "28\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J4NCRD_156_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J4NCRD_317_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J4NCRD_156_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J4NCRD_317_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J4NCRD_156_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J4NCRD_317_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J4NCRD_156_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J4NCRD_317_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:16       386,171 reads @  41.6 µs/read;   1.44 M reads/minute\n",
      "Finished in 16.05 s (42 µs/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            386,171\n",
      "  Read 1 with adapter:                  56,088 (14.5%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       386,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:   167,906,818 bp\n",
      "  Read 1:    83,868,896 bp\n",
      "  Read 2:    84,037,922 bp\n",
      "Total written (filtered):    167,682,272 bp (99.9%)\n",
      "  Read 1:    83,644,512 bp\n",
      "  Read 2:    84,037,760 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 56088 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t181\t6033.9\t0\t181\n",
      "4\t55900\t1508.5\t0\t55900\n",
      "31\t1\t0.0\t0\t1\n",
      "33\t5\t0.0\t0\t5\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "30\t1\t0.0\t0\t1\n",
      "33\t4\t0.0\t0\t4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J6IEC9_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J6IEC9_230_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J6IEC9_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J6IEC9_230_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J6IEC9_69_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/J6IEC9_230_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J6IEC9_69_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/J6IEC9_230_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         2,184 reads @  45.3 µs/read;   1.32 M reads/minute\n",
      "Finished in 0.10 s (47 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,184\n",
      "  Read 1 with adapter:                   1,785 (81.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         2,184 (100.0%)\n",
      "\n",
      "Total basepairs processed:       948,179 bp\n",
      "  Read 1:       474,221 bp\n",
      "  Read 2:       473,958 bp\n",
      "Total written (filtered):        941,040 bp (99.2%)\n",
      "  Read 1:       467,082 bp\n",
      "  Read 2:       473,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 1785 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t34.1\t0\t1\n",
      "4\t1784\t8.5\t0\t1784\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JEJ9C6_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JEJ9C6_188_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JEJ9C6_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JEJ9C6_188_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JEJ9C6_27_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JEJ9C6_188_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JEJ9C6_27_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JEJ9C6_188_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        36,366 reads @  42.5 µs/read;   1.41 M reads/minute\n",
      "Finished in 1.55 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,366\n",
      "  Read 1 with adapter:                  31,918 (87.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,366 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,802,292 bp\n",
      "  Read 1:     7,898,067 bp\n",
      "  Read 2:     7,904,225 bp\n",
      "Total written (filtered):     15,674,601 bp (99.2%)\n",
      "  Read 1:     7,770,376 bp\n",
      "  Read 2:     7,904,225 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 31918 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t568.2\t0\t8\n",
      "4\t31909\t142.1\t0\t31909\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JPW9HU_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JPW9HU_275_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JPW9HU_114_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JPW9HU_275_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JPW9HU_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JPW9HU_275_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JPW9HU_114_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JPW9HU_275_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        71,690 reads @  46.0 µs/read;   1.30 M reads/minute\n",
      "Finished in 3.31 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             71,690\n",
      "  Read 1 with adapter:                  48,198 (67.2%)\n",
      "  Read 2 with adapter:                      18 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        71,690 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,147,430 bp\n",
      "  Read 1:    15,916,937 bp\n",
      "  Read 2:    15,230,493 bp\n",
      "Total written (filtered):     30,954,569 bp (99.4%)\n",
      "  Read 1:    15,724,184 bp\n",
      "  Read 2:    15,230,385 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 48198 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t1120.2\t0\t62\n",
      "4\t48135\t280.0\t0\t48135\n",
      "27\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 18 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t18\t17.5\t0\t18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JRGI27_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JRGI27_207_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JRGI27_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JRGI27_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JRGI27_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JRGI27_207_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JRGI27_46_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JRGI27_207_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        26,550 reads @  42.7 µs/read;   1.40 M reads/minute\n",
      "Finished in 1.14 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,550\n",
      "  Read 1 with adapter:                  25,874 (97.5%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,528,998 bp\n",
      "  Read 1:     5,738,628 bp\n",
      "  Read 2:     5,790,370 bp\n",
      "Total written (filtered):     11,425,332 bp (99.1%)\n",
      "  Read 1:     5,635,107 bp\n",
      "  Read 2:     5,790,225 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 25874 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t414.8\t0\t5\n",
      "4\t25868\t103.7\t0\t25868\n",
      "34\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t0\t1\n",
      "113\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JW08QG_159_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JW08QG_320_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JW08QG_159_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JW08QG_320_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JW08QG_159_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/JW08QG_320_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JW08QG_159_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/JW08QG_320_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        38,783 reads @  47.8 µs/read;   1.25 M reads/minute\n",
      "Finished in 1.86 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,783\n",
      "  Read 1 with adapter:                  18,371 (47.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        38,783 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,904,858 bp\n",
      "  Read 1:     8,579,897 bp\n",
      "  Read 2:     8,324,961 bp\n",
      "Total written (filtered):     16,831,461 bp (99.6%)\n",
      "  Read 1:     8,506,500 bp\n",
      "  Read 2:     8,324,961 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 18371 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t88\t606.0\t0\t88\n",
      "4\t18282\t151.5\t0\t18282\n",
      "5\t1\t37.9\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/K613DJ_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/K613DJ_220_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/K613DJ_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/K613DJ_220_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/K613DJ_59_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/K613DJ_220_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/K613DJ_59_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/K613DJ_220_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:08       215,749 reads @  40.6 µs/read;   1.48 M reads/minute\n",
      "Finished in 8.77 s (41 µs/read; 1.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            215,749\n",
      "  Read 1 with adapter:                 199,710 (92.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       215,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:    93,752,636 bp\n",
      "  Read 1:    47,920,429 bp\n",
      "  Read 2:    45,832,207 bp\n",
      "Total written (filtered):     92,954,617 bp (99.1%)\n",
      "  Read 1:    47,122,570 bp\n",
      "  Read 2:    45,832,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 199710 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1285\t3371.1\t0\t1285\n",
      "4\t198419\t842.8\t0\t198419\n",
      "5\t4\t210.7\t0\t4\n",
      "100\t1\t0.0\t0\t1\n",
      "208\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "33\t1\t0.0\t0\t1\n",
      "127\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KR5DEB_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KR5DEB_168_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KR5DEB_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KR5DEB_168_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KR5DEB_7_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KR5DEB_168_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KR5DEB_7_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KR5DEB_168_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         8,587 reads @  44.7 µs/read;   1.34 M reads/minute\n",
      "Finished in 0.39 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,587\n",
      "  Read 1 with adapter:                   3,978 (46.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,587 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,754,462 bp\n",
      "  Read 1:     1,887,056 bp\n",
      "  Read 2:     1,867,406 bp\n",
      "Total written (filtered):      3,738,561 bp (99.6%)\n",
      "  Read 1:     1,871,155 bp\n",
      "  Read 2:     1,867,406 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 3978 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t11\t134.2\t0\t11\n",
      "4\t3967\t33.5\t0\t3967\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KTSCMP_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KTSCMP_285_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KTSCMP_124_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KTSCMP_285_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KTSCMP_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/KTSCMP_285_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KTSCMP_124_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/KTSCMP_285_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        67,929 reads @  48.1 µs/read;   1.25 M reads/minute\n",
      "Finished in 3.28 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,929\n",
      "  Read 1 with adapter:                  42,048 (61.9%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        67,929 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,557,332 bp\n",
      "  Read 1:    14,720,618 bp\n",
      "  Read 2:    14,836,714 bp\n",
      "Total written (filtered):     29,389,123 bp (99.4%)\n",
      "  Read 1:    14,552,534 bp\n",
      "  Read 2:    14,836,589 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 42048 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t127\t1061.4\t0\t127\n",
      "4\t41920\t265.3\t0\t41920\n",
      "23\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "125\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/L60NPJ_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/L60NPJ_297_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/L60NPJ_136_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/L60NPJ_297_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/L60NPJ_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/L60NPJ_297_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/L60NPJ_136_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/L60NPJ_297_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        48,207 reads @  48.5 µs/read;   1.24 M reads/minute\n",
      "Finished in 2.34 s (49 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,207\n",
      "  Read 1 with adapter:                  13,314 (27.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,944,793 bp\n",
      "  Read 1:    10,468,485 bp\n",
      "  Read 2:    10,476,308 bp\n",
      "Total written (filtered):     20,891,438 bp (99.7%)\n",
      "  Read 1:    10,415,142 bp\n",
      "  Read 2:    10,476,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 13314 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t753.2\t0\t32\n",
      "4\t13281\t188.3\t0\t13281\n",
      "123\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t11.8\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LCG2LJ_141_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LCG2LJ_302_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LCG2LJ_141_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LCG2LJ_302_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LCG2LJ_141_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LCG2LJ_302_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LCG2LJ_141_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LCG2LJ_302_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        46,814 reads @  45.3 µs/read;   1.33 M reads/minute\n",
      "Finished in 2.12 s (45 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,814\n",
      "  Read 1 with adapter:                  43,011 (91.9%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        46,814 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,346,770 bp\n",
      "  Read 1:    10,118,134 bp\n",
      "  Read 2:    10,228,636 bp\n",
      "Total written (filtered):     20,174,731 bp (99.2%)\n",
      "  Read 1:     9,946,095 bp\n",
      "  Read 2:    10,228,636 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 43011 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t5\t731.5\t0\t5\n",
      "4\t43006\t182.9\t0\t43006\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LIFJEK_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LIFJEK_253_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LIFJEK_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LIFJEK_253_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LIFJEK_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/LIFJEK_253_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LIFJEK_92_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/LIFJEK_253_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        25,379 reads @  50.3 µs/read;   1.19 M reads/minute\n",
      "Finished in 1.28 s (51 µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,379\n",
      "  Read 1 with adapter:                   7,309 (28.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        25,379 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,062,371 bp\n",
      "  Read 1:     5,561,164 bp\n",
      "  Read 2:     5,501,207 bp\n",
      "Total written (filtered):     11,033,154 bp (99.7%)\n",
      "  Read 1:     5,531,947 bp\n",
      "  Read 2:     5,501,207 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 7309 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t19\t396.5\t0\t19\n",
      "4\t7290\t99.1\t0\t7290\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/M12Q6E_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/M12Q6E_295_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/M12Q6E_134_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/M12Q6E_295_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/M12Q6E_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/M12Q6E_295_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/M12Q6E_134_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/M12Q6E_295_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:07       164,369 reads @  42.8 µs/read;   1.40 M reads/minute\n",
      "Finished in 7.04 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            164,369\n",
      "  Read 1 with adapter:                 155,070 (94.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       164,369 (100.0%)\n",
      "\n",
      "Total basepairs processed:    71,451,295 bp\n",
      "  Read 1:    35,696,958 bp\n",
      "  Read 2:    35,754,337 bp\n",
      "Total written (filtered):     70,831,177 bp (99.1%)\n",
      "  Read 1:    35,076,840 bp\n",
      "  Read 2:    35,754,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 155070 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t242\t2568.3\t0\t242\n",
      "4\t154827\t642.1\t0\t154827\n",
      "84\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/MN6YRR_160_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/MN6YRR_321_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/MN6YRR_160_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/MN6YRR_321_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/MN6YRR_160_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/MN6YRR_321_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/MN6YRR_160_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/MN6YRR_321_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:07       189,301 reads @  41.9 µs/read;   1.43 M reads/minute\n",
      "Finished in 7.93 s (42 µs/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            189,301\n",
      "  Read 1 with adapter:                   1,960 (1.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       189,301 (100.0%)\n",
      "\n",
      "Total basepairs processed:    82,315,655 bp\n",
      "  Read 1:    41,297,251 bp\n",
      "  Read 2:    41,018,404 bp\n",
      "Total written (filtered):     82,307,790 bp (100.0%)\n",
      "  Read 1:    41,289,494 bp\n",
      "  Read 2:    41,018,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 1960 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t83\t2957.8\t0\t83\n",
      "4\t1877\t739.5\t0\t1877\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "108\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/N4VICF_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/N4VICF_236_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/N4VICF_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/N4VICF_236_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/N4VICF_75_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/N4VICF_236_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/N4VICF_75_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/N4VICF_236_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        30,876 reads @  48.5 µs/read;   1.24 M reads/minute\n",
      "Finished in 1.50 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             30,876\n",
      "  Read 1 with adapter:                  10,120 (32.8%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        30,876 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,446,227 bp\n",
      "  Read 1:     6,778,903 bp\n",
      "  Read 2:     6,667,324 bp\n",
      "Total written (filtered):     13,405,599 bp (99.7%)\n",
      "  Read 1:     6,738,313 bp\n",
      "  Read 2:     6,667,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 10120 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t33\t482.4\t0\t33\n",
      "4\t10085\t120.6\t0\t10085\n",
      "45\t1\t0.0\t0\t1\n",
      "106\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t482.4\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NGEUPE_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NGEUPE_224_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NGEUPE_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NGEUPE_224_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NGEUPE_63_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NGEUPE_224_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NGEUPE_63_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NGEUPE_224_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        24,060 reads @  44.2 µs/read;   1.36 M reads/minute\n",
      "Finished in 1.07 s (44 µs/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,060\n",
      "  Read 1 with adapter:                  10,655 (44.3%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,467,793 bp\n",
      "  Read 1:     5,297,235 bp\n",
      "  Read 2:     5,170,558 bp\n",
      "Total written (filtered):     10,425,198 bp (99.6%)\n",
      "  Read 1:     5,254,643 bp\n",
      "  Read 2:     5,170,555 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 10655 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t28\t375.9\t0\t28\n",
      "4\t10627\t94.0\t0\t10627\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t375.9\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NTRPTL_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NTRPTL_237_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NTRPTL_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NTRPTL_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NTRPTL_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NTRPTL_237_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NTRPTL_76_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NTRPTL_237_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        48,146 reads @  44.6 µs/read;   1.34 M reads/minute\n",
      "Finished in 2.15 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,146\n",
      "  Read 1 with adapter:                  27,703 (57.5%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,945,148 bp\n",
      "  Read 1:    10,598,236 bp\n",
      "  Read 2:    10,346,912 bp\n",
      "Total written (filtered):     20,834,605 bp (99.5%)\n",
      "  Read 1:    10,487,693 bp\n",
      "  Read 2:    10,346,912 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 27703 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t372\t752.3\t0\t372\n",
      "4\t27330\t188.1\t0\t27330\n",
      "107\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NV2MHM_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NV2MHM_214_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NV2MHM_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NV2MHM_214_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NV2MHM_53_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/NV2MHM_214_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NV2MHM_53_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/NV2MHM_214_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        66,853 reads @  44.7 µs/read;   1.34 M reads/minute\n",
      "Finished in 2.99 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,853\n",
      "  Read 1 with adapter:                  39,890 (59.7%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        66,853 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,114,800 bp\n",
      "  Read 1:    14,781,031 bp\n",
      "  Read 2:    14,333,769 bp\n",
      "Total written (filtered):     28,955,189 bp (99.5%)\n",
      "  Read 1:    14,621,511 bp\n",
      "  Read 2:    14,333,678 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 39890 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t40\t1044.6\t0\t40\n",
      "4\t39850\t261.1\t0\t39850\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "91\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/O3OW1F_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/O3OW1F_196_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/O3OW1F_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/O3OW1F_196_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/O3OW1F_35_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/O3OW1F_196_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/O3OW1F_35_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/O3OW1F_196_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        32,680 reads @  44.8 µs/read;   1.34 M reads/minute\n",
      "Finished in 1.47 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,680\n",
      "  Read 1 with adapter:                  24,229 (74.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,226,279 bp\n",
      "  Read 1:     7,035,273 bp\n",
      "  Read 2:     7,191,006 bp\n",
      "Total written (filtered):     14,129,163 bp (99.3%)\n",
      "  Read 1:     6,938,157 bp\n",
      "  Read 2:     7,191,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 24229 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t36\t510.6\t0\t36\n",
      "4\t24191\t127.7\t0\t24191\n",
      "122\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/P0A2X1_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/P0A2X1_232_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/P0A2X1_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/P0A2X1_232_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/P0A2X1_71_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/P0A2X1_232_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/P0A2X1_71_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/P0A2X1_232_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        11,074 reads @  43.9 µs/read;   1.37 M reads/minute\n",
      "Finished in 0.49 s (44 µs/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,074\n",
      "  Read 1 with adapter:                   4,535 (41.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        11,074 (100.0%)\n",
      "\n",
      "Total basepairs processed:     4,810,637 bp\n",
      "  Read 1:     2,437,774 bp\n",
      "  Read 2:     2,372,863 bp\n",
      "Total written (filtered):      4,792,267 bp (99.6%)\n",
      "  Read 1:     2,419,614 bp\n",
      "  Read 2:     2,372,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 4535 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t173.0\t0\t8\n",
      "4\t4526\t43.3\t0\t4526\n",
      "32\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "78\t1\t0.0\t0\t1\n",
      "132\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PCUMU7_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PCUMU7_165_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PCUMU7_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PCUMU7_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PCUMU7_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PCUMU7_165_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PCUMU7_4_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PCUMU7_165_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        18,056 reads @  42.6 µs/read;   1.41 M reads/minute\n",
      "Finished in 0.77 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,056\n",
      "  Read 1 with adapter:                  15,668 (86.8%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,869,713 bp\n",
      "  Read 1:     3,980,105 bp\n",
      "  Read 2:     3,889,608 bp\n",
      "Total written (filtered):      7,806,717 bp (99.2%)\n",
      "  Read 1:     3,917,300 bp\n",
      "  Read 2:     3,889,417 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15668 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t282.1\t0\t13\n",
      "4\t15651\t70.5\t0\t15651\n",
      "30\t3\t0.0\t0\t3\n",
      "72\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t0\t1\n",
      "42\t1\t0.0\t0\t1\n",
      "113\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PG5LI3_99_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PG5LI3_260_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PG5LI3_99_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PG5LI3_260_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PG5LI3_99_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PG5LI3_260_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PG5LI3_99_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PG5LI3_260_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        73,539 reads @  46.3 µs/read;   1.29 M reads/minute\n",
      "Finished in 3.41 s (46 µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,539\n",
      "  Read 1 with adapter:                  23,098 (31.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        73,539 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,943,856 bp\n",
      "  Read 1:    15,824,696 bp\n",
      "  Read 2:    16,119,160 bp\n",
      "Total written (filtered):     31,851,736 bp (99.7%)\n",
      "  Read 1:    15,732,576 bp\n",
      "  Read 2:    16,119,160 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 23098 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t303\t1149.0\t0\t303\n",
      "4\t22794\t287.3\t0\t22794\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PGIR4X_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PGIR4X_271_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PGIR4X_110_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PGIR4X_271_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PGIR4X_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PGIR4X_271_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PGIR4X_110_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PGIR4X_271_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        28,941 reads @  44.7 µs/read;   1.34 M reads/minute\n",
      "Finished in 1.30 s (45 µs/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,941\n",
      "  Read 1 with adapter:                  25,181 (87.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        28,941 (100.0%)\n",
      "\n",
      "Total basepairs processed:    12,601,492 bp\n",
      "  Read 1:     6,296,569 bp\n",
      "  Read 2:     6,304,923 bp\n",
      "Total written (filtered):     12,500,700 bp (99.2%)\n",
      "  Read 1:     6,195,833 bp\n",
      "  Read 2:     6,304,867 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 25181 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t17\t452.2\t0\t17\n",
      "4\t25163\t113.1\t0\t25163\n",
      "33\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t452.2\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PP79I8_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PP79I8_193_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PP79I8_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PP79I8_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PP79I8_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PP79I8_193_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PP79I8_32_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PP79I8_193_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         8,680 reads @  48.1 µs/read;   1.25 M reads/minute\n",
      "Finished in 0.42 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,680\n",
      "  Read 1 with adapter:                   3,754 (43.2%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,680 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,789,986 bp\n",
      "  Read 1:     1,904,727 bp\n",
      "  Read 2:     1,885,259 bp\n",
      "Total written (filtered):      3,774,961 bp (99.6%)\n",
      "  Read 1:     1,889,702 bp\n",
      "  Read 2:     1,885,259 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 3754 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t18\t135.6\t0\t18\n",
      "4\t3735\t33.9\t0\t3735\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PQVFXQ_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PQVFXQ_257_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PQVFXQ_96_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PQVFXQ_257_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PQVFXQ_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PQVFXQ_257_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PQVFXQ_96_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PQVFXQ_257_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:11       263,841 reads @  44.3 µs/read;   1.35 M reads/minute\n",
      "Finished in 11.69 s (44 µs/read; 1.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            263,841\n",
      "  Read 1 with adapter:                 221,780 (84.1%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       263,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:   114,646,600 bp\n",
      "  Read 1:    58,072,432 bp\n",
      "  Read 2:    56,574,168 bp\n",
      "Total written (filtered):    113,758,998 bp (99.2%)\n",
      "  Read 1:    57,185,009 bp\n",
      "  Read 2:    56,573,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 221780 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t95\t4122.5\t0\t95\n",
      "4\t221681\t1030.6\t0\t221681\n",
      "5\t1\t257.7\t0\t1\n",
      "56\t1\t0.0\t0\t1\n",
      "145\t1\t0.0\t0\t1\n",
      "208\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t0\t1\n",
      "143\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PSHJIP_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PSHJIP_283_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PSHJIP_122_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PSHJIP_283_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PSHJIP_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PSHJIP_283_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PSHJIP_122_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PSHJIP_283_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:13       299,728 reads @  43.6 µs/read;   1.38 M reads/minute\n",
      "Finished in 13.08 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            299,728\n",
      "  Read 1 with adapter:                  20,071 (6.7%)\n",
      "  Read 2 with adapter:                       6 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       299,728 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,454,672 bp\n",
      "  Read 1:    65,086,623 bp\n",
      "  Read 2:    65,368,049 bp\n",
      "Total written (filtered):    130,374,255 bp (99.9%)\n",
      "  Read 1:    65,006,367 bp\n",
      "  Read 2:    65,367,888 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 20071 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t270\t4683.2\t0\t270\n",
      "4\t19797\t1170.8\t0\t19797\n",
      "33\t1\t0.0\t0\t1\n",
      "41\t1\t0.0\t0\t1\n",
      "79\t1\t0.0\t0\t1\n",
      "105\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 6 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t292.7\t0\t1\n",
      "6\t3\t73.2\t0\t3\n",
      "61\t1\t0.0\t0\t1\n",
      "77\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PTVRXH_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PTVRXH_255_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PTVRXH_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PTVRXH_255_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PTVRXH_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PTVRXH_255_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PTVRXH_94_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PTVRXH_255_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        33,463 reads @  51.1 µs/read;   1.17 M reads/minute\n",
      "Finished in 1.72 s (51 µs/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,463\n",
      "  Read 1 with adapter:                  32,801 (98.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,463 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,522,807 bp\n",
      "  Read 1:     7,197,356 bp\n",
      "  Read 2:     7,325,451 bp\n",
      "Total written (filtered):     14,391,615 bp (99.1%)\n",
      "  Read 1:     7,066,164 bp\n",
      "  Read 2:     7,325,451 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 32801 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t522.9\t0\t12\n",
      "4\t32789\t130.7\t0\t32789\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PYOTBF_158_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PYOTBF_319_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PYOTBF_158_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PYOTBF_319_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PYOTBF_158_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/PYOTBF_319_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PYOTBF_158_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/PYOTBF_319_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        51,149 reads @  48.8 µs/read;   1.23 M reads/minute\n",
      "Finished in 2.50 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,149\n",
      "  Read 1 with adapter:                  37,930 (74.2%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        51,149 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,275,455 bp\n",
      "  Read 1:    11,229,662 bp\n",
      "  Read 2:    11,045,793 bp\n",
      "Total written (filtered):     22,123,772 bp (99.3%)\n",
      "  Read 1:    11,078,098 bp\n",
      "  Read 2:    11,045,674 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 37930 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t799.2\t0\t156\n",
      "4\t37774\t199.8\t0\t37774\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Q4TOSG_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Q4TOSG_169_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Q4TOSG_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Q4TOSG_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Q4TOSG_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Q4TOSG_169_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Q4TOSG_8_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Q4TOSG_169_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        60,813 reads @  46.5 µs/read;   1.29 M reads/minute\n",
      "Finished in 2.84 s (47 µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,813\n",
      "  Read 1 with adapter:                  28,187 (46.4%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        60,813 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,407,296 bp\n",
      "  Read 1:    13,441,178 bp\n",
      "  Read 2:    12,966,118 bp\n",
      "Total written (filtered):     26,294,412 bp (99.6%)\n",
      "  Read 1:    13,328,454 bp\n",
      "  Read 2:    12,965,958 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 28187 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t74\t950.2\t0\t74\n",
      "4\t28111\t237.6\t0\t28111\n",
      "29\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "160\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QJIMXT_139_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QJIMXT_300_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QJIMXT_139_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QJIMXT_300_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QJIMXT_139_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QJIMXT_300_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QJIMXT_139_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QJIMXT_300_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:15       349,514 reads @  45.7 µs/read;   1.31 M reads/minute\n",
      "Finished in 15.99 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            349,514\n",
      "  Read 1 with adapter:                  34,955 (10.0%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       349,514 (100.0%)\n",
      "\n",
      "Total basepairs processed:   151,784,875 bp\n",
      "  Read 1:    76,570,000 bp\n",
      "  Read 2:    75,214,875 bp\n",
      "Total written (filtered):    151,644,357 bp (99.9%)\n",
      "  Read 1:    76,429,622 bp\n",
      "  Read 2:    75,214,735 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 34955 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t5461.2\t0\t80\n",
      "4\t34869\t1365.3\t0\t34869\n",
      "23\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "110\t1\t0.0\t0\t1\n",
      "124\t1\t0.0\t0\t1\n",
      "162\t1\t0.0\t0\t1\n",
      "210\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t85.3\t0\t1\n",
      "20\t1\t0.0\t0\t1\n",
      "114\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QWYAAY_117_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QWYAAY_278_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QWYAAY_117_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QWYAAY_278_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QWYAAY_117_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/QWYAAY_278_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QWYAAY_117_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/QWYAAY_278_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        33,532 reads @  45.5 µs/read;   1.32 M reads/minute\n",
      "Finished in 1.53 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,532\n",
      "  Read 1 with adapter:                  15,862 (47.3%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        33,532 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,596,850 bp\n",
      "  Read 1:     7,264,356 bp\n",
      "  Read 2:     7,332,494 bp\n",
      "Total written (filtered):     14,533,007 bp (99.6%)\n",
      "  Read 1:     7,200,893 bp\n",
      "  Read 2:     7,332,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15862 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t124\t523.9\t0\t124\n",
      "4\t15737\t131.0\t0\t15737\n",
      "143\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "74\t3\t0.0\t0\t3\n",
      "79\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RE2G74_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RE2G74_219_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RE2G74_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RE2G74_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RE2G74_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RE2G74_219_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RE2G74_58_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RE2G74_219_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        16,500 reads @  40.7 µs/read;   1.47 M reads/minute\n",
      "Finished in 0.68 s (41 µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,500\n",
      "  Read 1 with adapter:                  16,297 (98.8%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        16,500 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,195,989 bp\n",
      "  Read 1:     3,655,225 bp\n",
      "  Read 2:     3,540,764 bp\n",
      "Total written (filtered):      7,130,676 bp (99.1%)\n",
      "  Read 1:     3,590,035 bp\n",
      "  Read 2:     3,540,641 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16297 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t257.8\t0\t8\n",
      "4\t16287\t64.5\t0\t16287\n",
      "9\t2\t0.1\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "123\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RERN47_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RERN47_172_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RERN47_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RERN47_172_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RERN47_11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RERN47_172_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RERN47_11_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RERN47_172_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        43,136 reads @  42.4 µs/read;   1.41 M reads/minute\n",
      "Finished in 1.84 s (43 µs/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,136\n",
      "  Read 1 with adapter:                  18,045 (41.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        43,136 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,784,570 bp\n",
      "  Read 1:     9,514,955 bp\n",
      "  Read 2:     9,269,615 bp\n",
      "Total written (filtered):     18,712,206 bp (99.6%)\n",
      "  Read 1:     9,442,591 bp\n",
      "  Read 2:     9,269,615 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 18045 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t85\t674.0\t0\t85\n",
      "4\t17958\t168.5\t0\t17958\n",
      "98\t1\t0.0\t0\t1\n",
      "179\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RR9BC4_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RR9BC4_174_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RR9BC4_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RR9BC4_174_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RR9BC4_13_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RR9BC4_174_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RR9BC4_13_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RR9BC4_174_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        32,402 reads @  45.3 µs/read;   1.33 M reads/minute\n",
      "Finished in 1.47 s (45 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,402\n",
      "  Read 1 with adapter:                  26,063 (80.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,402 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,091,980 bp\n",
      "  Read 1:     7,070,870 bp\n",
      "  Read 2:     7,021,110 bp\n",
      "Total written (filtered):     13,987,719 bp (99.3%)\n",
      "  Read 1:     6,966,609 bp\n",
      "  Read 2:     7,021,110 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 26063 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t32\t506.3\t0\t32\n",
      "4\t26030\t126.6\t0\t26030\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RRA9NL_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RRA9NL_291_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RRA9NL_130_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RRA9NL_291_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RRA9NL_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RRA9NL_291_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RRA9NL_130_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RRA9NL_291_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        44,559 reads @  45.5 µs/read;   1.32 M reads/minute\n",
      "Finished in 2.03 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,559\n",
      "  Read 1 with adapter:                  15,182 (34.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,559 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,358,458 bp\n",
      "  Read 1:     9,719,951 bp\n",
      "  Read 2:     9,638,507 bp\n",
      "Total written (filtered):     19,297,807 bp (99.7%)\n",
      "  Read 1:     9,659,300 bp\n",
      "  Read 2:     9,638,507 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 15182 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t77\t696.2\t0\t77\n",
      "4\t15105\t174.1\t0\t15105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RU951M_127_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RU951M_288_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RU951M_127_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RU951M_288_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RU951M_127_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RU951M_288_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RU951M_127_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RU951M_288_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:12       299,811 reads @  41.0 µs/read;   1.46 M reads/minute\n",
      "Finished in 12.31 s (41 µs/read; 1.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            299,811\n",
      "  Read 1 with adapter:                 287,688 (96.0%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       299,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:   130,199,832 bp\n",
      "  Read 1:    66,263,041 bp\n",
      "  Read 2:    63,936,791 bp\n",
      "Total written (filtered):    129,048,810 bp (99.1%)\n",
      "  Read 1:    65,112,199 bp\n",
      "  Read 2:    63,936,611 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 287688 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t273\t4684.5\t0\t273\n",
      "4\t287412\t1171.1\t0\t287412\n",
      "79\t1\t0.0\t0\t1\n",
      "103\t1\t0.0\t0\t1\n",
      "193\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "180\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RY2J8K_107_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RY2J8K_268_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RY2J8K_107_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RY2J8K_268_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RY2J8K_107_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RY2J8K_268_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RY2J8K_107_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RY2J8K_268_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         7,443 reads @  45.7 µs/read;   1.31 M reads/minute\n",
      "Finished in 0.35 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              7,443\n",
      "  Read 1 with adapter:                   4,665 (62.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         7,443 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,249,343 bp\n",
      "  Read 1:     1,629,217 bp\n",
      "  Read 2:     1,620,126 bp\n",
      "Total written (filtered):      3,230,554 bp (99.4%)\n",
      "  Read 1:     1,610,428 bp\n",
      "  Read 2:     1,620,126 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 4665 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t116.3\t0\t6\n",
      "4\t4658\t29.1\t0\t4658\n",
      "139\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RYYSOI_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RYYSOI_217_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RYYSOI_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RYYSOI_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RYYSOI_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RYYSOI_217_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RYYSOI_56_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RYYSOI_217_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        22,275 reads @  42.6 µs/read;   1.41 M reads/minute\n",
      "Finished in 0.95 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,275\n",
      "  Read 1 with adapter:                   8,553 (38.4%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,691,275 bp\n",
      "  Read 1:     4,802,639 bp\n",
      "  Read 2:     4,888,636 bp\n",
      "Total written (filtered):      9,657,064 bp (99.6%)\n",
      "  Read 1:     4,768,434 bp\n",
      "  Read 2:     4,888,630 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 8553 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t7\t348.0\t0\t7\n",
      "4\t8546\t87.0\t0\t8546\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t5.4\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RZEXX3_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RZEXX3_225_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RZEXX3_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RZEXX3_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RZEXX3_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/RZEXX3_225_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RZEXX3_64_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/RZEXX3_225_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        41,621 reads @  49.5 µs/read;   1.21 M reads/minute\n",
      "Finished in 2.07 s (50 µs/read; 1.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,621\n",
      "  Read 1 with adapter:                  16,487 (39.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        41,621 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,108,694 bp\n",
      "  Read 1:     8,977,623 bp\n",
      "  Read 2:     9,131,071 bp\n",
      "Total written (filtered):     18,042,625 bp (99.6%)\n",
      "  Read 1:     8,911,554 bp\n",
      "  Read 2:     9,131,071 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16487 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t80\t650.3\t0\t80\n",
      "4\t16405\t162.6\t0\t16405\n",
      "66\t1\t0.0\t0\t1\n",
      "143\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/S2JX0O_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/S2JX0O_293_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/S2JX0O_132_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/S2JX0O_293_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/S2JX0O_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/S2JX0O_293_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/S2JX0O_132_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/S2JX0O_293_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:04        82,935 reads @  48.3 µs/read;   1.24 M reads/minute\n",
      "Finished in 4.01 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,935\n",
      "  Read 1 with adapter:                  59,764 (72.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        82,935 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,036,245 bp\n",
      "  Read 1:    18,331,391 bp\n",
      "  Read 2:    17,704,854 bp\n",
      "Total written (filtered):     35,797,008 bp (99.3%)\n",
      "  Read 1:    18,092,154 bp\n",
      "  Read 2:    17,704,854 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 59764 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t109\t1295.9\t0\t109\n",
      "4\t59652\t324.0\t0\t59652\n",
      "23\t1\t0.0\t0\t1\n",
      "124\t1\t0.0\t0\t1\n",
      "155\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SETQFZ_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SETQFZ_186_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SETQFZ_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SETQFZ_186_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SETQFZ_25_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SETQFZ_186_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SETQFZ_25_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SETQFZ_186_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        74,180 reads @  50.1 µs/read;   1.20 M reads/minute\n",
      "Finished in 3.73 s (50 µs/read; 1.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,180\n",
      "  Read 1 with adapter:                  35,383 (47.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        74,180 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,225,980 bp\n",
      "  Read 1:    16,469,866 bp\n",
      "  Read 2:    15,756,114 bp\n",
      "Total written (filtered):     32,084,375 bp (99.6%)\n",
      "  Read 1:    16,328,261 bp\n",
      "  Read 2:    15,756,114 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 35383 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t152\t1159.1\t0\t152\n",
      "4\t35228\t289.8\t0\t35228\n",
      "31\t2\t0.0\t0\t2\n",
      "175\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SFI9L6_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SFI9L6_179_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SFI9L6_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SFI9L6_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SFI9L6_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SFI9L6_179_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SFI9L6_18_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SFI9L6_179_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        80,995 reads @  45.8 µs/read;   1.31 M reads/minute\n",
      "Finished in 3.71 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             80,995\n",
      "  Read 1 with adapter:                  45,700 (56.4%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        80,995 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,182,537 bp\n",
      "  Read 1:    17,432,123 bp\n",
      "  Read 2:    17,750,414 bp\n",
      "Total written (filtered):     34,999,860 bp (99.5%)\n",
      "  Read 1:    17,249,477 bp\n",
      "  Read 2:    17,750,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 45700 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t156\t1265.5\t0\t156\n",
      "4\t45543\t316.4\t0\t45543\n",
      "6\t1\t19.8\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SLV228_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SLV228_289_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SLV228_128_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SLV228_289_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SLV228_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/SLV228_289_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SLV228_128_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/SLV228_289_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        32,862 reads @  47.5 µs/read;   1.26 M reads/minute\n",
      "Finished in 1.57 s (48 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,862\n",
      "  Read 1 with adapter:                  20,891 (63.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,862 (100.0%)\n",
      "\n",
      "Total basepairs processed:    14,275,360 bp\n",
      "  Read 1:     7,232,202 bp\n",
      "  Read 2:     7,043,158 bp\n",
      "Total written (filtered):     14,191,776 bp (99.4%)\n",
      "  Read 1:     7,148,618 bp\n",
      "  Read 2:     7,043,158 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 20891 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t72\t513.5\t0\t72\n",
      "4\t20818\t128.4\t0\t20818\n",
      "96\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/T41N7M_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/T41N7M_254_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/T41N7M_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/T41N7M_254_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/T41N7M_93_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/T41N7M_254_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/T41N7M_93_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/T41N7M_254_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:13       308,000 reads @  44.2 µs/read;   1.36 M reads/minute\n",
      "Finished in 13.61 s (44 µs/read; 1.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            308,000\n",
      "  Read 1 with adapter:                 263,541 (85.6%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       308,000 (100.0%)\n",
      "\n",
      "Total basepairs processed:   133,775,806 bp\n",
      "  Read 1:    67,468,000 bp\n",
      "  Read 2:    66,307,806 bp\n",
      "Total written (filtered):    132,721,781 bp (99.2%)\n",
      "  Read 1:    66,414,116 bp\n",
      "  Read 2:    66,307,665 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 263541 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t542\t4812.5\t0\t542\n",
      "4\t262996\t1203.1\t0\t262996\n",
      "5\t1\t300.8\t0\t1\n",
      "79\t1\t0.0\t0\t1\n",
      "190\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t1\t75.2\t0\t1\n",
      "135\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TEHZN5_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TEHZN5_248_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TEHZN5_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TEHZN5_248_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TEHZN5_87_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TEHZN5_248_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TEHZN5_87_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TEHZN5_248_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        49,749 reads @  46.7 µs/read;   1.28 M reads/minute\n",
      "Finished in 2.33 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,749\n",
      "  Read 1 with adapter:                  26,752 (53.8%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        49,749 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,653,545 bp\n",
      "  Read 1:    10,821,188 bp\n",
      "  Read 2:    10,832,357 bp\n",
      "Total written (filtered):     21,546,233 bp (99.5%)\n",
      "  Read 1:    10,714,059 bp\n",
      "  Read 2:    10,832,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 26752 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t48\t777.3\t0\t48\n",
      "4\t26701\t194.3\t0\t26701\n",
      "45\t2\t0.0\t0\t2\n",
      "91\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t777.3\t0\t1\n",
      "180\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TTNSEZ_125_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TTNSEZ_286_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TTNSEZ_125_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TTNSEZ_286_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TTNSEZ_125_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/TTNSEZ_286_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TTNSEZ_125_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/TTNSEZ_286_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        39,262 reads @  46.1 µs/read;   1.30 M reads/minute\n",
      "Finished in 1.81 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,262\n",
      "  Read 1 with adapter:                  35,059 (89.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        39,262 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,046,867 bp\n",
      "  Read 1:     8,485,826 bp\n",
      "  Read 2:     8,561,041 bp\n",
      "Total written (filtered):     16,906,637 bp (99.2%)\n",
      "  Read 1:     8,345,596 bp\n",
      "  Read 2:     8,561,041 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 35059 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t613.5\t0\t6\n",
      "4\t35053\t153.4\t0\t35053\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/U5Y4NC_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/U5Y4NC_229_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/U5Y4NC_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/U5Y4NC_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/U5Y4NC_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/U5Y4NC_229_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/U5Y4NC_68_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/U5Y4NC_229_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        18,050 reads @  46.7 µs/read;   1.28 M reads/minute\n",
      "Finished in 0.85 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,050\n",
      "  Read 1 with adapter:                   9,898 (54.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        18,050 (100.0%)\n",
      "\n",
      "Total basepairs processed:     7,847,829 bp\n",
      "  Read 1:     3,937,819 bp\n",
      "  Read 2:     3,910,010 bp\n",
      "Total written (filtered):      7,808,319 bp (99.5%)\n",
      "  Read 1:     3,898,309 bp\n",
      "  Read 2:     3,910,010 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 9898 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t82\t282.0\t0\t82\n",
      "4\t9816\t70.5\t0\t9816\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UB1Z91_149_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UB1Z91_310_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UB1Z91_149_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UB1Z91_310_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UB1Z91_149_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UB1Z91_310_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UB1Z91_149_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UB1Z91_310_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        59,104 reads @  44.9 µs/read;   1.34 M reads/minute\n",
      "Finished in 2.66 s (45 µs/read; 1.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,104\n",
      "  Read 1 with adapter:                  13,782 (23.3%)\n",
      "  Read 2 with adapter:                       5 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        59,104 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,700,946 bp\n",
      "  Read 1:    12,779,838 bp\n",
      "  Read 2:    12,921,108 bp\n",
      "Total written (filtered):     25,645,805 bp (99.8%)\n",
      "  Read 1:    12,724,753 bp\n",
      "  Read 2:    12,921,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 13782 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t43\t923.5\t0\t43\n",
      "4\t13739\t230.9\t0\t13739\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t4\t14.4\t0\t4\n",
      "32\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UM6EDM_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UM6EDM_182_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UM6EDM_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UM6EDM_182_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UM6EDM_21_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UM6EDM_182_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UM6EDM_21_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UM6EDM_182_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         1,616 reads @  47.7 µs/read;   1.26 M reads/minute\n",
      "Finished in 0.08 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,616\n",
      "  Read 1 with adapter:                     649 (40.2%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         1,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:       702,995 bp\n",
      "  Read 1:       358,874 bp\n",
      "  Read 2:       344,121 bp\n",
      "Total written (filtered):        700,212 bp (99.6%)\n",
      "  Read 1:       356,091 bp\n",
      "  Read 2:       344,121 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 649 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t3\t25.2\t0\t3\n",
      "4\t644\t6.3\t0\t644\n",
      "99\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UQ1VYJ_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UQ1VYJ_175_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UQ1VYJ_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UQ1VYJ_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UQ1VYJ_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UQ1VYJ_175_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UQ1VYJ_14_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UQ1VYJ_175_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:19       434,648 reads @  43.9 µs/read;   1.37 M reads/minute\n",
      "Finished in 19.09 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            434,648\n",
      "  Read 1 with adapter:                 369,238 (85.0%)\n",
      "  Read 2 with adapter:                       2 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       434,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:   188,896,202 bp\n",
      "  Read 1:    96,510,701 bp\n",
      "  Read 2:    92,385,501 bp\n",
      "Total written (filtered):    187,418,911 bp (99.2%)\n",
      "  Read 1:    95,033,470 bp\n",
      "  Read 2:    92,385,441 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 369238 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t207\t6791.4\t0\t207\n",
      "4\t369023\t1697.8\t0\t369023\n",
      "5\t3\t424.5\t0\t3\n",
      "22\t1\t0.0\t0\t1\n",
      "28\t1\t0.0\t0\t1\n",
      "77\t1\t0.0\t0\t1\n",
      "179\t1\t0.0\t0\t1\n",
      "197\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 2 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/USRRQV_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/USRRQV_192_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/USRRQV_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/USRRQV_192_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/USRRQV_31_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/USRRQV_192_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/USRRQV_31_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/USRRQV_192_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:12       277,185 reads @  45.8 µs/read;   1.31 M reads/minute\n",
      "Finished in 12.69 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            277,185\n",
      "  Read 1 with adapter:                 152,989 (55.2%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       277,185 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,566,982 bp\n",
      "  Read 1:    59,966,279 bp\n",
      "  Read 2:    60,600,703 bp\n",
      "Total written (filtered):    119,954,868 bp (99.5%)\n",
      "  Read 1:    59,354,254 bp\n",
      "  Read 2:    60,600,614 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 152989 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t432\t4331.0\t0\t432\n",
      "4\t152552\t1082.8\t0\t152552\n",
      "5\t1\t270.7\t0\t1\n",
      "106\t1\t0.0\t0\t1\n",
      "109\t1\t0.0\t0\t1\n",
      "110\t1\t0.0\t0\t1\n",
      "191\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t1\t4331.0\t0\t1\n",
      "6\t1\t67.7\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UT1NIU_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UT1NIU_185_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UT1NIU_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UT1NIU_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UT1NIU_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/UT1NIU_185_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UT1NIU_24_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/UT1NIU_185_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        70,708 reads @  51.2 µs/read;   1.17 M reads/minute\n",
      "Finished in 3.63 s (51 µs/read; 1.17 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,708\n",
      "  Read 1 with adapter:                  27,967 (39.6%)\n",
      "  Read 2 with adapter:                       3 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        70,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,708,107 bp\n",
      "  Read 1:    15,213,797 bp\n",
      "  Read 2:    15,494,310 bp\n",
      "Total written (filtered):     30,596,103 bp (99.6%)\n",
      "  Read 1:    15,101,987 bp\n",
      "  Read 2:    15,494,116 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 27967 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t208\t1104.8\t0\t208\n",
      "4\t27757\t276.2\t0\t27757\n",
      "59\t1\t0.0\t0\t1\n",
      "99\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "6\t2\t17.3\t0\t2\n",
      "182\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VGQSRI_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VGQSRI_287_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VGQSRI_126_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VGQSRI_287_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VGQSRI_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VGQSRI_287_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VGQSRI_126_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VGQSRI_287_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        35,656 reads @  46.2 µs/read;   1.30 M reads/minute\n",
      "Finished in 1.65 s (46 µs/read; 1.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,656\n",
      "  Read 1 with adapter:                  34,872 (97.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        35,656 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,474,682 bp\n",
      "  Read 1:     7,669,179 bp\n",
      "  Read 2:     7,805,503 bp\n",
      "Total written (filtered):     15,335,207 bp (99.1%)\n",
      "  Read 1:     7,529,704 bp\n",
      "  Read 2:     7,805,503 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 34872 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t13\t557.1\t0\t13\n",
      "4\t34859\t139.3\t0\t34859\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VLMX7N_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VLMX7N_211_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VLMX7N_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VLMX7N_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VLMX7N_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/VLMX7N_211_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VLMX7N_50_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/VLMX7N_211_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:16       389,607 reads @  41.8 µs/read;   1.44 M reads/minute\n",
      "Finished in 16.29 s (42 µs/read; 1.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            389,607\n",
      "  Read 1 with adapter:                 333,431 (85.6%)\n",
      "  Read 2 with adapter:                       4 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):       389,607 (100.0%)\n",
      "\n",
      "Total basepairs processed:   169,337,250 bp\n",
      "  Read 1:    86,125,049 bp\n",
      "  Read 2:    83,212,201 bp\n",
      "Total written (filtered):    168,001,651 bp (99.2%)\n",
      "  Read 1:    84,789,473 bp\n",
      "  Read 2:    83,212,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 333431 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t219\t6087.6\t0\t219\n",
      "4\t333193\t1521.9\t0\t333193\n",
      "5\t1\t380.5\t0\t1\n",
      "24\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "68\t1\t0.0\t0\t1\n",
      "78\t1\t0.0\t0\t1\n",
      "104\t1\t0.0\t0\t1\n",
      "124\t9\t0.0\t0\t9\n",
      "174\t1\t0.0\t0\t1\n",
      "175\t2\t0.0\t0\t2\n",
      "199\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "5\t1\t380.5\t0\t1\n",
      "6\t3\t95.1\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/W5414C_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/W5414C_242_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/W5414C_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/W5414C_242_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/W5414C_81_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/W5414C_242_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/W5414C_81_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/W5414C_242_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        44,207 reads @  48.0 µs/read;   1.25 M reads/minute\n",
      "Finished in 2.13 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,207\n",
      "  Read 1 with adapter:                  41,452 (93.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,208,295 bp\n",
      "  Read 1:     9,555,044 bp\n",
      "  Read 2:     9,653,251 bp\n",
      "Total written (filtered):     19,042,549 bp (99.1%)\n",
      "  Read 1:     9,389,298 bp\n",
      "  Read 2:     9,653,251 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 41452 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t62\t690.7\t0\t62\n",
      "4\t41390\t172.7\t0\t41390\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WJA8C1_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WJA8C1_239_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WJA8C1_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WJA8C1_239_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WJA8C1_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WJA8C1_239_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WJA8C1_78_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WJA8C1_239_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        24,615 reads @  48.0 µs/read;   1.25 M reads/minute\n",
      "Finished in 1.19 s (48 µs/read; 1.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             24,615\n",
      "  Read 1 with adapter:                  24,276 (98.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        24,615 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,729,894 bp\n",
      "  Read 1:     5,393,608 bp\n",
      "  Read 2:     5,336,286 bp\n",
      "Total written (filtered):     10,632,799 bp (99.1%)\n",
      "  Read 1:     5,296,513 bp\n",
      "  Read 2:     5,336,286 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 24276 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t9\t384.6\t0\t9\n",
      "4\t24267\t96.2\t0\t24267\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WP0TB1_145_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WP0TB1_306_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WP0TB1_145_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WP0TB1_306_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WP0TB1_145_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WP0TB1_306_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WP0TB1_145_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WP0TB1_306_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        36,794 reads @  47.3 µs/read;   1.27 M reads/minute\n",
      "Finished in 1.75 s (47 µs/read; 1.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,794\n",
      "  Read 1 with adapter:                  24,420 (66.4%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        36,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,994,952 bp\n",
      "  Read 1:     8,097,800 bp\n",
      "  Read 2:     7,897,152 bp\n",
      "Total written (filtered):     15,897,254 bp (99.4%)\n",
      "  Read 1:     8,000,102 bp\n",
      "  Read 2:     7,897,152 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 24420 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t71\t574.9\t0\t71\n",
      "4\t24347\t143.7\t0\t24347\n",
      "30\t1\t0.0\t0\t1\n",
      "67\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WQBQQ2_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WQBQQ2_195_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WQBQQ2_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WQBQQ2_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WQBQQ2_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WQBQQ2_195_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WQBQQ2_34_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WQBQQ2_195_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        32,178 reads @  46.6 µs/read;   1.29 M reads/minute\n",
      "Finished in 1.50 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,178\n",
      "  Read 1 with adapter:                  16,149 (50.2%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        32,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    13,972,973 bp\n",
      "  Read 1:     6,955,388 bp\n",
      "  Read 2:     7,017,585 bp\n",
      "Total written (filtered):     13,908,423 bp (99.5%)\n",
      "  Read 1:     6,890,838 bp\n",
      "  Read 2:     7,017,585 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 16149 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t46\t502.8\t0\t46\n",
      "4\t16103\t125.7\t0\t16103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WWN7Y0_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WWN7Y0_222_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WWN7Y0_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WWN7Y0_222_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WWN7Y0_61_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WWN7Y0_222_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WWN7Y0_61_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WWN7Y0_222_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        13,572 reads @  41.7 µs/read;   1.44 M reads/minute\n",
      "Finished in 0.57 s (42 µs/read; 1.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,572\n",
      "  Read 1 with adapter:                  12,492 (92.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        13,572 (100.0%)\n",
      "\n",
      "Total basepairs processed:     5,908,087 bp\n",
      "  Read 1:     3,006,599 bp\n",
      "  Read 2:     2,901,488 bp\n",
      "Total written (filtered):      5,858,125 bp (99.2%)\n",
      "  Read 1:     2,956,637 bp\n",
      "  Read 2:     2,901,488 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 12492 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t6\t212.1\t0\t6\n",
      "4\t12486\t53.0\t0\t12486\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WZVL32_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WZVL32_177_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WZVL32_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WZVL32_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WZVL32_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/WZVL32_177_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WZVL32_16_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/WZVL32_177_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        44,285 reads @  46.1 µs/read;   1.30 M reads/minute\n",
      "Finished in 2.05 s (46 µs/read; 1.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,285\n",
      "  Read 1 with adapter:                  38,375 (86.7%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        44,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,234,303 bp\n",
      "  Read 1:     9,573,795 bp\n",
      "  Read 2:     9,660,508 bp\n",
      "Total written (filtered):     19,080,812 bp (99.2%)\n",
      "  Read 1:     9,420,336 bp\n",
      "  Read 2:     9,660,476 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 38375 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t94\t692.0\t0\t94\n",
      "4\t38280\t173.0\t0\t38280\n",
      "57\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "32\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XCYZJI_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XCYZJI_206_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XCYZJI_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XCYZJI_206_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XCYZJI_45_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XCYZJI_206_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XCYZJI_45_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XCYZJI_206_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        53,080 reads @  43.1 µs/read;   1.39 M reads/minute\n",
      "Finished in 2.29 s (43 µs/read; 1.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,080\n",
      "  Read 1 with adapter:                   4,228 (8.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,054,974 bp\n",
      "  Read 1:    11,524,083 bp\n",
      "  Read 2:    11,530,891 bp\n",
      "Total written (filtered):     23,037,880 bp (99.9%)\n",
      "  Read 1:    11,506,989 bp\n",
      "  Read 2:    11,530,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 4228 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t56\t829.4\t0\t56\n",
      "4\t4170\t207.3\t0\t4170\n",
      "64\t1\t0.0\t0\t1\n",
      "182\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XDEIYI_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XDEIYI_205_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XDEIYI_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XDEIYI_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XDEIYI_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XDEIYI_205_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XDEIYI_44_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XDEIYI_205_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         6,465 reads @  44.0 µs/read;   1.37 M reads/minute\n",
      "Finished in 0.29 s (45 µs/read; 1.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,465\n",
      "  Read 1 with adapter:                   3,073 (47.5%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         6,465 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,821,341 bp\n",
      "  Read 1:     1,438,652 bp\n",
      "  Read 2:     1,382,689 bp\n",
      "Total written (filtered):      2,809,032 bp (99.6%)\n",
      "  Read 1:     1,426,374 bp\n",
      "  Read 2:     1,382,658 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 3073 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t14\t101.0\t0\t14\n",
      "4\t3059\t25.3\t0\t3059\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XO59R8_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XO59R8_166_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XO59R8_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XO59R8_166_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XO59R8_5_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XO59R8_166_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XO59R8_5_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XO59R8_166_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        42,694 reads @  48.7 µs/read;   1.23 M reads/minute\n",
      "Finished in 2.08 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,694\n",
      "  Read 1 with adapter:                  21,120 (49.5%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        42,694 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,609,836 bp\n",
      "  Read 1:     9,499,436 bp\n",
      "  Read 2:     9,110,400 bp\n",
      "Total written (filtered):     18,525,258 bp (99.5%)\n",
      "  Read 1:     9,414,886 bp\n",
      "  Read 2:     9,110,372 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 21120 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t115\t667.1\t0\t115\n",
      "4\t21002\t166.8\t0\t21002\n",
      "41\t1\t0.0\t0\t1\n",
      "78\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "28\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVCUIZ_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVCUIZ_252_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVCUIZ_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVCUIZ_252_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVCUIZ_91_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVCUIZ_252_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVCUIZ_91_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVCUIZ_252_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:03        75,645 reads @  49.7 µs/read;   1.21 M reads/minute\n",
      "Finished in 3.77 s (50 µs/read; 1.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,645\n",
      "  Read 1 with adapter:                  42,749 (56.5%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        75,645 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,885,278 bp\n",
      "  Read 1:    16,646,213 bp\n",
      "  Read 2:    16,239,065 bp\n",
      "Total written (filtered):     32,714,362 bp (99.5%)\n",
      "  Read 1:    16,475,297 bp\n",
      "  Read 2:    16,239,065 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 42749 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t106\t1182.0\t0\t106\n",
      "4\t42642\t295.5\t0\t42642\n",
      "30\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVFIYP_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVFIYP_228_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVFIYP_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVFIYP_228_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVFIYP_67_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/XVFIYP_228_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVFIYP_67_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/XVFIYP_228_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         8,542 reads @  44.6 µs/read;   1.35 M reads/minute\n",
      "Finished in 0.39 s (45 µs/read; 1.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,542\n",
      "  Read 1 with adapter:                   7,271 (85.1%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,542 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,721,754 bp\n",
      "  Read 1:     1,872,214 bp\n",
      "  Read 2:     1,849,540 bp\n",
      "Total written (filtered):      3,692,660 bp (99.2%)\n",
      "  Read 1:     1,843,120 bp\n",
      "  Read 2:     1,849,540 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 7271 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t133.5\t0\t12\n",
      "4\t7258\t33.4\t0\t7258\n",
      "26\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YL5QG5_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YL5QG5_199_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YL5QG5_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YL5QG5_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YL5QG5_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YL5QG5_199_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YL5QG5_38_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YL5QG5_199_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        22,132 reads @  43.6 µs/read;   1.38 M reads/minute\n",
      "Finished in 0.97 s (44 µs/read; 1.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,132\n",
      "  Read 1 with adapter:                  14,161 (64.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        22,132 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,614,685 bp\n",
      "  Read 1:     4,826,956 bp\n",
      "  Read 2:     4,787,729 bp\n",
      "Total written (filtered):      9,558,062 bp (99.4%)\n",
      "  Read 1:     4,770,333 bp\n",
      "  Read 2:     4,787,729 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 14161 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t21\t345.8\t0\t21\n",
      "4\t14140\t86.5\t0\t14140\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YLLVZM_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YLLVZM_180_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YLLVZM_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YLLVZM_180_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YLLVZM_19_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/YLLVZM_180_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YLLVZM_19_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/YLLVZM_180_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        45,023 reads @  45.6 µs/read;   1.31 M reads/minute\n",
      "Finished in 2.06 s (46 µs/read; 1.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,023\n",
      "  Read 1 with adapter:                  34,021 (75.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        45,023 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,610,053 bp\n",
      "  Read 1:     9,998,652 bp\n",
      "  Read 2:     9,611,401 bp\n",
      "Total written (filtered):     19,474,077 bp (99.3%)\n",
      "  Read 1:     9,862,676 bp\n",
      "  Read 2:     9,611,401 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 34021 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t108\t703.5\t0\t108\n",
      "4\t33913\t175.9\t0\t33913\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z2DXHY_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z2DXHY_189_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z2DXHY_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z2DXHY_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z2DXHY_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z2DXHY_189_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z2DXHY_28_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z2DXHY_189_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        48,860 reads @  48.5 µs/read;   1.24 M reads/minute\n",
      "Finished in 2.38 s (49 µs/read; 1.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,860\n",
      "  Read 1 with adapter:                  47,728 (97.7%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        48,860 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,232,291 bp\n",
      "  Read 1:    10,561,079 bp\n",
      "  Read 2:    10,671,212 bp\n",
      "Total written (filtered):     21,041,181 bp (99.1%)\n",
      "  Read 1:    10,369,969 bp\n",
      "  Read 2:    10,671,212 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 47728 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t12\t763.4\t0\t12\n",
      "4\t47714\t190.9\t0\t47714\n",
      "75\t1\t0.0\t0\t1\n",
      "143\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z95BTZ_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z95BTZ_299_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z95BTZ_138_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z95BTZ_299_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z95BTZ_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z95BTZ_299_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z95BTZ_138_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z95BTZ_299_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        26,178 reads @  42.6 µs/read;   1.41 M reads/minute\n",
      "Finished in 1.12 s (43 µs/read; 1.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,178\n",
      "  Read 1 with adapter:                     447 (1.7%)\n",
      "  Read 2 with adapter:                       1 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        26,178 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,368,415 bp\n",
      "  Read 1:     5,632,997 bp\n",
      "  Read 2:     5,735,418 bp\n",
      "Total written (filtered):     11,366,620 bp (100.0%)\n",
      "  Read 1:     5,631,209 bp\n",
      "  Read 2:     5,735,411 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 447 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t447\t102.3\t0\t447\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-18 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "7\t1\t1.6\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z9874X_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z9874X_218_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z9874X_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z9874X_218_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z9874X_57_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/Z9874X_218_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z9874X_57_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/Z9874X_218_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         8,337 reads @  47.3 µs/read;   1.27 M reads/minute\n",
      "Finished in 0.40 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              8,337\n",
      "  Read 1 with adapter:                   3,943 (47.3%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         8,337 (100.0%)\n",
      "\n",
      "Total basepairs processed:     3,632,862 bp\n",
      "  Read 1:     1,853,321 bp\n",
      "  Read 2:     1,779,541 bp\n",
      "Total written (filtered):      3,617,010 bp (99.6%)\n",
      "  Read 1:     1,837,469 bp\n",
      "  Read 2:     1,779,541 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 3943 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t8\t130.3\t0\t8\n",
      "4\t3934\t32.6\t0\t3934\n",
      "92\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZJYE4E_143_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZJYE4E_304_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZJYE4E_143_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZJYE4E_304_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZJYE4E_143_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZJYE4E_304_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZJYE4E_143_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZJYE4E_304_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        53,153 reads @  47.1 µs/read;   1.27 M reads/minute\n",
      "Finished in 2.51 s (47 µs/read; 1.27 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,153\n",
      "  Read 1 with adapter:                  19,450 (36.6%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):        53,153 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,108,041 bp\n",
      "  Read 1:    11,752,219 bp\n",
      "  Read 2:    11,355,822 bp\n",
      "Total written (filtered):     23,030,180 bp (99.7%)\n",
      "  Read 1:    11,674,358 bp\n",
      "  Read 2:    11,355,822 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 19450 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t116\t830.5\t0\t116\n",
      "4\t19333\t207.6\t0\t19333\n",
      "181\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZKJI45_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZKJI45_163_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZKJI45_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZKJI45_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZKJI45_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZKJI45_163_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZKJI45_2_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZKJI45_163_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         6,794 reads @  45.8 µs/read;   1.31 M reads/minute\n",
      "Finished in 0.32 s (47 µs/read; 1.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              6,794\n",
      "  Read 1 with adapter:                   6,536 (96.2%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         6,794 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,976,851 bp\n",
      "  Read 1:     1,492,733 bp\n",
      "  Read 2:     1,484,118 bp\n",
      "Total written (filtered):      2,950,709 bp (99.1%)\n",
      "  Read 1:     1,466,591 bp\n",
      "  Read 2:     1,484,118 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 6536 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t106.2\t0\t2\n",
      "4\t6534\t26.5\t0\t6534\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZQT8ZN_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZQT8ZN_234_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZQT8ZN_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZQT8ZN_234_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.0 with Python 3.8.13\n",
      "Command line parameters: --cores 1 --error-rate 0 --times 1 --overlap 3 --minimum-length 1 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZQT8ZN_73_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-p4d5rt0g/ZQT8ZN_234_L001_R2_001.fastq.gz --front AYTGGGYDTAAAGNG -G CCGTCAATTYHTTTRAGT /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZQT8ZN_73_L001_R1_001.fastq.gz /tmp/qiime2-archive-we88jvpb/394c4773-80e2-46a6-9fba-40e7c8ec3fb9/data/ZQT8ZN_234_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         3,252 reads @  45.9 µs/read;   1.31 M reads/minute\n",
      "Finished in 0.16 s (48 µs/read; 1.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,252\n",
      "  Read 1 with adapter:                   2,434 (74.8%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):         3,252 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,415,226 bp\n",
      "  Read 1:       718,916 bp\n",
      "  Read 2:       696,310 bp\n",
      "Total written (filtered):      1,405,492 bp (99.3%)\n",
      "  Read 1:       709,182 bp\n",
      "  Read 2:       696,310 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: AYTGGGYDTAAAGNG; Type: regular 5'; Length: 15; Trimmed: 2434 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-14 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "3\t2\t50.8\t0\t2\n",
      "4\t2432\t12.7\t0\t2432\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: CCGTCAATTYHTTTRAGT; Type: regular 5'; Length: 18; Trimmed: 0 times\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: Alien_data/primer-trimmed-seqs.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime cutadapt trim-paired \\\n",
    "  --i-demultiplexed-sequences $data_dir/sequences.qza \\\n",
    "  --p-front-f AYTGGGYDTAAAGNG \\\n",
    "  --p-front-r CCGTCAATTYHTTTRAGT \\\n",
    "  --p-error-rate 0 \\\n",
    "  --o-trimmed-sequences $data_dir/primer-trimmed-seqs.qza \\\n",
    "  --verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "d15a8709-0c41-4472-b079-6cbb7c02da00",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/primer-trimmed-seqs.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime demux summarize \\\n",
    "  --i-data $data_dir/primer-trimmed-seqs.qza \\\n",
    "  --o-visualization $data_dir/primer-trimmed-seqs.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "81c2a5b8-b75e-4e68-af9a-6b5ac2844f13",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-c7i8ah1j')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: a3e8ac5e-8ddf-4645-aca7-bc0bd3d885ca>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/primer-trimmed-seqs.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3cefd7c1-b90f-4b9d-bbc4-0748e38065a9",
   "metadata": {},
   "source": [
    "Dada 2 is used to denoise the paired-end sequences. The truncation length is selected based on median value of quality score larger than 30. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "d3f3ff84-9b08-4474-86ef-5ed954f4c554",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Alien_data/dada2_table.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/dada2_rep_set.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: Alien_data/dada2_stats.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime dada2 denoise-paired \\\n",
    "    --i-demultiplexed-seqs $data_dir/primer-trimmed-seqs.qza \\\n",
    "    --p-trunc-len-f 183 \\\n",
    "    --p-trunc-len-r 190 \\\n",
    "    --p-n-threads 4 \\\n",
    "    --o-table $data_dir/dada2_table.qza \\\n",
    "    --o-representative-sequences $data_dir/dada2_rep_set.qza \\\n",
    "    --o-denoising-stats $data_dir/dada2_stats.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "06d8244b-4982-450f-88c0-42cf8ec06ab2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/dada2_stats.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime metadata tabulate \\\n",
    "    --m-input-file $data_dir/dada2_stats.qza \\\n",
    "    --o-visualization $data_dir/dada2_stats.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "33be3db9-b9b9-43e0-a394-d6faa4a0565c",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-4lh91mz7')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 8f87926c-2fde-4497-bfe3-3c6ea402adfd>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/dada2_stats.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "27780a68-dff2-4f29-bad0-1aad6fd23c92",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/dada2_table.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-table summarize \\\n",
    "  --i-table $data_dir/dada2_table.qza \\\n",
    "  --o-visualization $data_dir/dada2_table.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "d61cc7ed-2220-4d9d-87e4-d45683a3b3cf",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-jenfrf_j')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 2a7f7529-81a8-47d3-a192-75249483c000>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/dada2_table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "9298fdfc-2cb2-4198-9f4f-72d49bcd3eb4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/dada2_rep_set.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-table tabulate-seqs \\\n",
    "  --i-data $data_dir/dada2_rep_set.qza \\\n",
    "  --o-visualization $data_dir/dada2_rep_set.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "5012c088-29f4-4b61-8073-05373e3b3017",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-dfmcf583')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 92e1a1db-1b09-4bf5-869e-d75c9035d036>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/dada2_rep_set.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "61a1831f-31b1-40a3-9e0b-b78c4e781365",
   "metadata": {},
   "outputs": [],
   "source": [
    "Taxonomy classification:"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f0d759e3-07ea-4131-888c-a1457e7ce907",
   "metadata": {},
   "source": [
    "Database curation:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "deeb7b34-aead-41ec-93bf-97f60d75a833",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[RNASequence] to: Alien_data/silva-138.1-ssu-nr99-rna-seqs.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Taxonomy] to: Alien_data/silva-138.1-ssu-nr99-tax.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime rescript get-silva-data \\\n",
    "    --p-version '138.1' \\\n",
    "    --p-target 'SSURef_NR99' \\\n",
    "    --p-include-species-labels \\\n",
    "    --o-silva-sequences $data_dir/silva-138.1-ssu-nr99-rna-seqs.qza \\\n",
    "    --o-silva-taxonomy $data_dir/silva-138.1-ssu-nr99-tax.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "69f9dd03-1b4f-4921-b646-68eaf5f6a408",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138.1-ssu-nr99-rna-seqs-cleaned.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    " ! qiime rescript cull-seqs \\\n",
    "     --i-sequences $data_dir/silva-138.1-ssu-nr99-rna-seqs.qza \\\n",
    "     --p-num-degenerates 5 \\\n",
    "     --p-homopolymer-length 8 \\\n",
    "     --p-n-jobs 3 \\\n",
    "     --o-clean-sequences $data_dir/silva-138.1-ssu-nr99-rna-seqs-cleaned.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "21c582b8-c0f4-4972-b09f-e805061784d5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138-ssu-nr99-seqs-filt.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138-ssu-nr99-seqs-discard.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime rescript filter-seqs-length-by-taxon \\\n",
    "    --i-sequences $data_dir/silva-138.1-ssu-nr99-rna-seqs-cleaned.qza \\\n",
    "    --i-taxonomy $data_dir/silva-138.1-ssu-nr99-tax.qza \\\n",
    "    --p-labels Archaea Bacteria Eukaryota \\\n",
    "    --p-min-lens 900 1200 1400 \\\n",
    "    --o-filtered-seqs $data_dir/silva-138-ssu-nr99-seqs-filt.qza \\\n",
    "    --o-discarded-seqs $data_dir/silva-138-ssu-nr99-seqs-discard.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "597d94a6-a677-458f-b6b6-da98c86fe65e",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138-ssu-nr99-seqs-derep-uniq.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Taxonomy] to: Alien_data/silva-138-ssu-nr99-tax-derep-uniq.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime rescript dereplicate \\\n",
    "    --i-sequences $data_dir/silva-138-ssu-nr99-seqs-filt.qza  \\\n",
    "    --i-taxa $data_dir/silva-138.1-ssu-nr99-tax.qza \\\n",
    "    --p-rank-handles 'silva' \\\n",
    "    --p-mode 'uniq' \\\n",
    "    --p-threads 3 \\\n",
    "    --o-dereplicated-sequences $data_dir/silva-138-ssu-nr99-seqs-derep-uniq.qza \\\n",
    "    --o-dereplicated-taxa $data_dir/silva-138-ssu-nr99-tax-derep-uniq.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f8ca9a48-e039-4922-a641-7846bd8c22b9",
   "metadata": {},
   "source": [
    "PCR extraction: \n",
    "The primers used for our samples are designed for the V4-V5 region:\n",
    "563 F: 5′-AYTGGGYDTAAAGNG-3′\n",
    "926 R: 5′-CCGTCAATTYHTTTRAGT-3′"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "1afbd751-9652-42af-8888-f3f8e3684a59",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138-ssu-nr99-seqs-515f-806r.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-classifier extract-reads \\\n",
    "    --i-sequences $data_dir/silva-138-ssu-nr99-seqs-derep-uniq.qza \\\n",
    "    --p-f-primer AYTGGGYDTAAAGNG \\\n",
    "    --p-r-primer CCGTCAATTYHTTTRAGT \\\n",
    "    --p-n-jobs 3 \\\n",
    "    --p-read-orientation 'forward' \\\n",
    "    --o-reads $data_dir/silva-138-ssu-nr99-seqs-515f-806r.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "5a0eab2e-49bf-440b-af42-e4d0a8d1e3f7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/silva-138-ssu-nr99-seqs-515f-806r-uniq.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Taxonomy] to: Alien_data/silva-138-ssu-nr99-tax-515f-806r-derep-uniq.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime rescript dereplicate \\\n",
    "    --i-sequences $data_dir/silva-138-ssu-nr99-seqs-515f-806r.qza \\\n",
    "    --i-taxa $data_dir/silva-138-ssu-nr99-tax-derep-uniq.qza \\\n",
    "    --p-rank-handles 'silva' \\\n",
    "    --p-mode 'uniq' \\\n",
    "    --p-threads 3 \\\n",
    "    --o-dereplicated-sequences $data_dir/silva-138-ssu-nr99-seqs-515f-806r-uniq.qza \\\n",
    "    --o-dereplicated-taxa  $data_dir/silva-138-ssu-nr99-tax-515f-806r-derep-uniq.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2f15ea38-bcad-41de-a0a3-67b8120e0f04",
   "metadata": {},
   "source": [
    "We tried to use the silva database and train the classifers using naive bayes, but it exceeded the memory capacity to train the classifer and also to use the pre-trained Silva database, so we decided to use the pre-trained Greengene classifier. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "abd99ccb-3865-4748-8be7-d7908656c09f",
   "metadata": {},
   "outputs": [],
   "source": [
    "#! qiime feature-classifier fit-classifier-naive-bayes \\\n",
    "#     --i-reference-reads $data_dir/silva-138-ssu-nr99-seqs-515f-806r-uniq.qza \\\n",
    "#     --i-reference-taxonomy $data_dir/silva-138-ssu-nr99-tax-515f-806r-derep-uniq.qza \\\n",
    "#     --p-classify--chunk-size 1000 \\\n",
    "#     --o-classifier $data_dir/515f-806r-classifier.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "370ff0bd-2685-4f8d-8e04-de4dec89f78c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2022-10-26 19:41:49 URL:https://s3-us-west-2.amazonaws.com/qiime2-data/2022.8/common/gg-13-8-99-nb-classifier.qza [104512483/104512483] -> \"Alien_data/gg-13-8-99-nb-classifier.qza\" [1]\n"
     ]
    }
   ],
   "source": [
    "! wget -nv -O $data_dir/gg-13-8-99-nb-classifier.qza 'https://data.qiime2.org/2022.8/common/gg-13-8-99-nb-classifier.qza'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "d96a9733-86f0-46a0-8897-bafeee689f5b",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Taxonomy] to: Alien_data/taxonomy.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-classifier classify-sklearn \\\n",
    "    --i-classifier $data_dir/gg-13-8-99-nb-classifier.qza \\\n",
    "    --i-reads $data_dir/dada2_rep_set.qza \\\n",
    "    --o-classification $data_dir/taxonomy.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "7a94c334-ed97-447e-9fb7-1b7ede804af4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/taxonomy.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime metadata tabulate \\\n",
    "    --m-input-file $data_dir/taxonomy.qza \\\n",
    "    --o-visualization $data_dir/taxonomy.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "ebabd8f3-d92e-4cfc-8efc-f8eb7ba93478",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-pka9xhq8')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: dd568eab-270d-4892-9f13-ae4bb3dc3822>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/taxonomy.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0b6e4b04-0990-4ede-a3ba-b61a2424d4f7",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/taxa-bar-plots.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime taxa barplot \\\n",
    "    --i-table $data_dir/dada2_table.qza \\\n",
    "    --i-taxonomy $data_dir/taxonomy.qza \\\n",
    "    --m-metadata-file $data_dir/sample_metadata.tsv \\\n",
    "    --o-visualization $data_dir/taxa-bar-plots.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "d51a1755-6541-456b-b3ee-54deeb3c68f6",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-cja3ph37')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 0f39b3b8-b8f7-4d8f-84e3-f77b18f1d391>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/taxa-bar-plots.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f95fe2fe-6e03-4a53-b3d5-b0fc60489c41",
   "metadata": {},
   "source": [
    "Mitochondira and chloroplast are filtered since they don't belong to the gut microbiota:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "c2818c5d-f0ae-4df2-9250-19e4213cc8ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: Alien_data/table-filtered.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime taxa filter-table \\\n",
    "    --i-table $data_dir/dada2_table.qza \\\n",
    "    --i-taxonomy $data_dir/taxonomy.qza \\\n",
    "    --p-exclude mitochondria,chloroplast \\\n",
    "    --o-filtered-table $data_dir/table-filtered.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6cebc20d-607e-4054-b5b5-84deff4c7410",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[Sequence] to: Alien_data/rep-seqs-filtered.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime taxa filter-seqs \\\n",
    "    --i-sequences $data_dir/dada2_rep_set.qza \\\n",
    "    --i-taxonomy $data_dir/taxonomy.qza \\\n",
    "    --p-exclude mitochondria,chloroplast \\\n",
    "    --o-filtered-sequences $data_dir/rep-seqs-filtered.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "5b7e969d-f37b-4f67-8219-bb63db7322bc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: Alien_data/taxa-bar-plots_filtered.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime taxa barplot \\\n",
    "    --i-table $data_dir/table-filtered.qza \\\n",
    "    --i-taxonomy $data_dir/taxonomy.qza \\\n",
    "    --m-metadata-file $data_dir/sample_metadata.tsv \\\n",
    "    --o-visualization $data_dir/taxa-bar-plots_filtered.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "876fb459-c86b-4cee-a95b-4c846c0db746",
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-ay344oho')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 6bf8bdcd-fb82-4933-b20f-aa69ef72167c>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/taxa-bar-plots_filtered.qzv')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
