{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "150d84cf",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2021-10-01T10:31:55.394646Z",
     "iopub.status.busy": "2021-10-01T10:31:55.393848Z",
     "iopub.status.idle": "2021-10-01T10:31:55.402993Z",
     "shell.execute_reply": "2021-10-01T10:31:55.401758Z",
     "shell.execute_reply.started": "2021-10-01T10:31:55.394573Z"
    }
   },
   "source": [
    "# Functional Prediction\n",
    "\n",
    "You can learn more about PICRUSt2 on its [GitHub wiki](https://github.com/picrust/picrust2/wiki), in [this tutorial](https://github.com/picrust/picrust2/wiki/q2-picrust2-Tutorial) and the [Nature Biotechnology article](https://doi.org/10.1038/s41587-020-0548-6)."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6ca7c2f6",
   "metadata": {},
   "source": [
    "<a id='setup'></a>\n",
    "## 0. Setup"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "0a3bcc7b",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "import qiime2 as q2\n",
    "import requests\n",
    "\n",
    "from qiime2 import Visualization\n",
    "\n",
    "data_dir = '../data/'\n",
    "    \n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "210739be",
   "metadata": {},
   "outputs": [],
   "source": [
    "def fetch_ipath(ids: list, img_output_path: str, verbose: bool = False):\n",
    "    \"\"\"Fetches a enriched pathways map from iPATH3 for given IDs.\"\"\"\n",
    "    url = 'https://pathways.embl.de/mapping.cgi'\n",
    "    \n",
    "    # remove colon from EC names\n",
    "    if ':' in ids[0]:\n",
    "        ids = [x.replace(':', '') for x in ids]\n",
    "    \n",
    "    if verbose:\n",
    "        print(f'Fetching iPATH3 diagram for ids: {ids}')\n",
    "    params = {\n",
    "        'default_opacity': 0.6,\n",
    "        'export_type': 'svg',\n",
    "        'selection': '\\n'.join(ids)\n",
    "    }   \n",
    "    response = requests.get(url=url, params=params)\n",
    "    \n",
    "    with open(img_output_path, 'wb') as img:\n",
    "        img.write(response.content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1af03ced-2f86-4f4c-9d9f-09a9ba0a6059",
   "metadata": {},
   "outputs": [],
   "source": [
    "# path to the picrust2 conda environment - do not change!\n",
    "picrust_env = '/opt/conda/envs/picrust2/bin'"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ed5ba7be",
   "metadata": {},
   "source": [
    "<a id='picrust'></a>\n",
    "## 1. Functional inference\n",
    "\n",
    "Use Picrust 2 to simulate metagenome data from our dataset. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "09f86b29",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "QIIME is caching your current deployment for improved performance. This may take a few moments and should only happen once per deployment.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved FeatureTable[Frequency] to: shared_data//picrust2_results/ko_metagenome.qza\n",
      "Saved FeatureTable[Frequency] to: shared_data//picrust2_results/ec_metagenome.qza\n",
      "Saved FeatureTable[Frequency] to: shared_data//picrust2_results/pathway_abundance.qza\n"
     ]
    }
   ],
   "source": [
    "%%script env picrust_env=\"$picrust_env\" data_dir=\"$data_dir\" bash\n",
    "\n",
    "# append the env location to PATH so that qiime\n",
    "# can find all required executables\n",
    "export PATH=$picrust_env:$PATH\n",
    "\n",
    "$picrust_env/qiime picrust2 full-pipeline \\\n",
    "    --i-seq $data_dir/denoising/dada2_rep_set.qza \\\n",
    "    --i-table $data_dir/denoising/dada2_table.qza \\\n",
    "    --output-dir $data_dir/picrust2_results \\\n",
    "    --p-placement-tool sepp \\\n",
    "    --p-threads 2 \\\n",
    "    --p-hsp-method pic \\\n",
    "    --p-max-nsti 2 "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c0aaecc2-937f-4634-bbd9-463e35776887",
   "metadata": {},
   "source": [
    "## 2. Analyzing the Effect of Abduction\n",
    "\n",
    "Here, we want to find the difference of functional pathways between abducted and not abducted patients."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "31f68586-166e-418f-a20c-526e789eb03b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Composition] to: ../data//picrust2_results/pathway_abundance_abund.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition add-pseudocount \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance.qza \\\n",
    "    --o-composition-table $data_dir/picrust2_results/pathway_abundance_abund.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "f0079376-cc50-427d-b687-87f2b7e2e130",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: ../data//functional_prediction/pa_ancom_abducted.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --m-metadata-column alleged_abduction \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/functional_prediction/pa_ancom_abducted.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "6443c77a-7912-4a03-b322-499eb8b8fe09",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-j4qrifzr')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 33554640-18eb-46ea-ae79-a44a42118166>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/functional_prediction/pa_ancom_abducted.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "22896178-b0d3-45d3-bc18-616995834c4f",
   "metadata": {},
   "source": [
    "### 2.1 Enriched KEGG orthologs visualization\n",
    "\n",
    "We start by finding the most abundant features in each table:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "f8c058a0-b1d7-4889-8114-2805efdd580c",
   "metadata": {},
   "outputs": [],
   "source": [
    "ko = q2.Artifact.load(f'{data_dir}/picrust2_results/ko_metagenome.qza').view(pd.DataFrame)\n",
    "ec = q2.Artifact.load(f'{data_dir}/picrust2_results/ec_metagenome.qza').view(pd.DataFrame)\n",
    "pa = q2.Artifact.load(f'{data_dir}/picrust2_results/pathway_abundance.qza').view(pd.DataFrame)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "be49af0e-4e1d-4671-909a-17ee04f1311e",
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = pd.read_csv(f'{data_dir}/metadata/str_metadata.tsv', sep='\\t', header=0, index_col=0)\n",
    "\n",
    "ko_meta = ko.merge(metadata[['alleged_abduction']], left_index=True, right_index=True)\n",
    "ec_meta = ec.merge(metadata[['alleged_abduction']], left_index=True, right_index=True)\n",
    "pa_meta = pa.merge(metadata[['alleged_abduction']], left_index=True, right_index=True)\n",
    "\n",
    "ko_meta_avg = ko_meta.groupby('alleged_abduction').mean()\n",
    "ec_meta_avg = ec_meta.groupby('alleged_abduction').mean()\n",
    "pa_meta_avg = pa_meta.groupby('alleged_abduction').mean()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "008c55a9-3f1e-410e-8b98-d97ce7361bf1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving 102 most abundant features...\n",
      "Saving 86 most abundant features...\n",
      "Saving 5 most abundant features...\n"
     ]
    }
   ],
   "source": [
    "def find_most_abundant(df: pd.DataFrame, frac):\n",
    "    if 0 < frac < 1:\n",
    "        frac = int(frac * len(df.columns))\n",
    "    print(f'Saving {frac} most abundant features...')\n",
    "    most_abundant = {\n",
    "        smp: df.loc[smp, :].sort_values(ascending=False)[:frac]\n",
    "        for smp in df.index\n",
    "    }\n",
    "    return most_abundant\n",
    "\n",
    "ko_most_abundant = find_most_abundant(ko_meta_avg, 0.01)\n",
    "ec_most_abundant = find_most_abundant(ec_meta_avg, 0.03)\n",
    "pa_most_abundant = find_most_abundant(pa_meta_avg, 5)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "6fe31da3-3d9a-4f4f-ba8d-a10b67e75c55",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10 most abundant KOs in the abducted group are: ['K01990', 'K06147', 'K07024', 'K01992', 'K02003', 'K02004', 'K02030', 'K02015', 'K02529', 'K01223']\n",
      "10 most abundant KOs in the non-abducted group are: ['K01990', 'K01992', 'K06147', 'K02003', 'K07024', 'K02004', 'K03088', 'K02529', 'K16787', 'K02030']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(f'10 most abundant KOs in the abducted group are: {ko_most_abundant[\"abducted\"].index[:10].tolist()}\\n'\n",
    "      f'10 most abundant KOs in the non-abducted group are: {ko_most_abundant[\"non_abducted\"].index[:10].tolist()}\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "f05d5e79-4fce-454d-9fd8-30515710ca4d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10 most abundant ECs in the abducted group are: ['EC:3.6.4.12', 'EC:2.7.7.7', 'EC:2.7.13.3', 'EC:2.7.1.69', 'EC:2.7.7.6', 'EC:3.2.1.86', 'EC:2.1.1.72', 'EC:5.2.1.8', 'EC:3.4.16.4', 'EC:5.4.2.12']\n",
      "10 most abundant ECs in the non-abducted group are: ['EC:2.7.7.7', 'EC:3.6.4.12', 'EC:2.7.13.3', 'EC:2.7.1.69', 'EC:2.7.7.6', 'EC:1.97.1.4', 'EC:2.1.1.72', 'EC:3.4.16.4', 'EC:5.4.2.12', 'EC:5.2.1.8']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(f'10 most abundant ECs in the abducted group are: {ec_most_abundant[\"abducted\"].index[:10].tolist()}\\n'\n",
    "      f'10 most abundant ECs in the non-abducted group are: {ec_most_abundant[\"non_abducted\"].index[:10].tolist()}\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "d388593b-ff0c-4807-92a5-c73b2716ca83",
   "metadata": {},
   "outputs": [],
   "source": [
    "for smp in ko_most_abundant.keys():\n",
    "    fetch_ipath(ko_most_abundant[smp].index.tolist(), f'../plots/functional_prediction/kos_{smp}.svg')\n",
    "    fetch_ipath(ec_most_abundant[smp].index.str.replace(':', '').tolist(), f'../plots/functional_prediction/ecs_{smp}.svg')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b282a759-af71-4280-b716-8de715d8e8fa",
   "metadata": {},
   "source": [
    "## 3. Analyzing the Effect of Disease Type\n",
    "\n",
    "In this section we would like to predict the functionality of microbioata of patient with different diseases across patients alleged by aliens and not alleged by patients.\n",
    "\n",
    "We start by filtering samples that are abducted by patients."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "68bc7b46",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/ko_metagenome_abducted.qza\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/ec_metagenome_abducted.qza\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/pathway_abundance_abducted.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/ko_metagenome.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where \"[alleged_abduction]='abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/ko_metagenome_abducted.qza\n",
    "\n",
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/ec_metagenome.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where  \"[alleged_abduction]='abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/ec_metagenome_abducted.qza\n",
    "\n",
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where  \"[alleged_abduction]='abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/pathway_abundance_abducted.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "93f26b38-ecd4-42ea-ba28-327b7bbbfea3",
   "metadata": {},
   "source": [
    "and, we do the same for non-abducted patients."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "df4fb7b4-d7ee-4f99-ae62-717dc6485efb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/ko_metagenome_not_abducted.qza\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/ec_metagenome_not_abducted.qza\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved FeatureTable[Frequency] to: ../data//picrust2_results/pathway_abundance_not_abducted.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/ko_metagenome.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where \"[alleged_abduction]='non_abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/ko_metagenome_not_abducted.qza\n",
    "\n",
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/ec_metagenome.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where  \"[alleged_abduction]='non_abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/ec_metagenome_not_abducted.qza\n",
    "\n",
    "! qiime feature-table filter-samples \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_metadata.tsv \\\n",
    "    --p-where  \"[alleged_abduction]='non_abducted'\" \\\n",
    "    --o-filtered-table $data_dir/picrust2_results/pathway_abundance_not_abducted.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8bd2e878-8cf1-4962-b0ae-8056c8d6aa32",
   "metadata": {},
   "source": [
    "Now, we can read in all three artifacts using QIIME 2 Python API - we can view them as DataFrames:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "50eda9ea-511b-41e6-b62a-8fc421def9e3",
   "metadata": {},
   "outputs": [],
   "source": [
    "ko_a = q2.Artifact.load(f'{data_dir}/picrust2_results/ko_metagenome_abducted.qza').view(pd.DataFrame)\n",
    "ec_a = q2.Artifact.load(f'{data_dir}/picrust2_results/ec_metagenome_abducted.qza').view(pd.DataFrame)\n",
    "pa_a = q2.Artifact.load(f'{data_dir}/picrust2_results/pathway_abundance_abducted.qza').view(pd.DataFrame)\n",
    "ko_na = q2.Artifact.load(f'{data_dir}/picrust2_results/ko_metagenome_not_abducted.qza').view(pd.DataFrame)\n",
    "ec_na = q2.Artifact.load(f'{data_dir}/picrust2_results/ec_metagenome_not_abducted.qza').view(pd.DataFrame)\n",
    "pa_na = q2.Artifact.load(f'{data_dir}/picrust2_results/pathway_abundance_not_abducted.qza').view(pd.DataFrame)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "870fc2df-cfeb-4f1d-8212-69ae81265b5c",
   "metadata": {},
   "source": [
    "We can briefly examine the contents of each of those tables, and realize that the values differ across the abducted and non-abducted patients."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "cbc94f07-7b4c-43b8-b115-3206b8833545",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19776</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0DOSLC</th>\n",
       "      <td>54182.069959</td>\n",
       "      <td>6.981562</td>\n",
       "      <td>114120.689680</td>\n",
       "      <td>72354.209497</td>\n",
       "      <td>45604.268712</td>\n",
       "      <td>10.652446</td>\n",
       "      <td>79362.866673</td>\n",
       "      <td>59942.895952</td>\n",
       "      <td>33794.194597</td>\n",
       "      <td>5.324617e-06</td>\n",
       "      <td>...</td>\n",
       "      <td>25.698715</td>\n",
       "      <td>37.359132</td>\n",
       "      <td>22.442708</td>\n",
       "      <td>5.201112</td>\n",
       "      <td>19.893207</td>\n",
       "      <td>27133.128495</td>\n",
       "      <td>0.007928</td>\n",
       "      <td>0.0</td>\n",
       "      <td>37.851223</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0WG67D</th>\n",
       "      <td>36949.682025</td>\n",
       "      <td>6.063987</td>\n",
       "      <td>83110.648508</td>\n",
       "      <td>56205.161333</td>\n",
       "      <td>33241.112284</td>\n",
       "      <td>9.230694</td>\n",
       "      <td>44920.981973</td>\n",
       "      <td>38218.559997</td>\n",
       "      <td>28267.465068</td>\n",
       "      <td>4.644675e-20</td>\n",
       "      <td>...</td>\n",
       "      <td>15.415347</td>\n",
       "      <td>11.101046</td>\n",
       "      <td>11.187774</td>\n",
       "      <td>2.697171</td>\n",
       "      <td>10.796936</td>\n",
       "      <td>24703.230120</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>17.805109</td>\n",
       "      <td>0.003554</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 9847 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              K00001    K00002         K00003        K00004        K00005  \\\n",
       "0DOSLC  54182.069959  6.981562  114120.689680  72354.209497  45604.268712   \n",
       "0WG67D  36949.682025  6.063987   83110.648508  56205.161333  33241.112284   \n",
       "\n",
       "           K00007        K00008        K00009        K00010        K00011  \\\n",
       "0DOSLC  10.652446  79362.866673  59942.895952  33794.194597  5.324617e-06   \n",
       "0WG67D   9.230694  44920.981973  38218.559997  28267.465068  4.644675e-20   \n",
       "\n",
       "        ...     K19776     K19777     K19778    K19779     K19780  \\\n",
       "0DOSLC  ...  25.698715  37.359132  22.442708  5.201112  19.893207   \n",
       "0WG67D  ...  15.415347  11.101046  11.187774  2.697171  10.796936   \n",
       "\n",
       "              K19784    K19785  K19788     K19789    K19791  \n",
       "0DOSLC  27133.128495  0.007928     0.0  37.851223  0.000000  \n",
       "0WG67D  24703.230120  0.000000     0.0  17.805109  0.003554  \n",
       "\n",
       "[2 rows x 9847 columns]"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_a.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "989fc66a-0d4d-448a-8e5d-dd453c5e0cda",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19787</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0KB68F</th>\n",
       "      <td>1361.400386</td>\n",
       "      <td>22.47605</td>\n",
       "      <td>6638.807061</td>\n",
       "      <td>1283.583009</td>\n",
       "      <td>2819.624601</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>12651.473549</td>\n",
       "      <td>4992.046300</td>\n",
       "      <td>5519.033478</td>\n",
       "      <td>3.783984e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>0.024526</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>609.994359</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.010882</td>\n",
       "      <td>0.00029</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0TR6O2</th>\n",
       "      <td>337.235395</td>\n",
       "      <td>99.51724</td>\n",
       "      <td>1772.973885</td>\n",
       "      <td>63.402322</td>\n",
       "      <td>1388.255311</td>\n",
       "      <td>0.000032</td>\n",
       "      <td>1582.917987</td>\n",
       "      <td>383.045118</td>\n",
       "      <td>847.539563</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>51.894474</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.000032</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 10251 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             K00001    K00002       K00003       K00004       K00005  \\\n",
       "0KB68F  1361.400386  22.47605  6638.807061  1283.583009  2819.624601   \n",
       "0TR6O2   337.235395  99.51724  1772.973885    63.402322  1388.255311   \n",
       "\n",
       "          K00007        K00008       K00009       K00010        K00011  ...  \\\n",
       "0KB68F  0.000000  12651.473549  4992.046300  5519.033478  3.783984e-21  ...   \n",
       "0TR6O2  0.000032   1582.917987   383.045118   847.539563  0.000000e+00  ...   \n",
       "\n",
       "          K19777  K19778  K19779  K19780      K19784  K19785  K19787  K19788  \\\n",
       "0KB68F  0.024526     0.0     0.0     0.0  609.994359     0.0     0.0     0.0   \n",
       "0TR6O2  0.000000     0.0     0.0     0.0   51.894474     0.0     0.0     0.0   \n",
       "\n",
       "          K19789   K19791  \n",
       "0KB68F  0.010882  0.00029  \n",
       "0TR6O2  0.000032  0.00000  \n",
       "\n",
       "[2 rows x 10251 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_na.head(2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4a329bbe",
   "metadata": {},
   "source": [
    "These tables contain information about different levels of the functional profiles:\n",
    "\n",
    "1. `ko` table: columns represent KEGG orthologs, as indicated by their names (e.g., **K**19777)\n",
    "2. `ec` table: columns represent enzymes, as indicated by the Enzyme Commission numbers (e.g., **EC**:1.1.1.108)\n",
    "3. `pa` table: columns represent entire pathways using the MetaCyc classification (e.g., ANAGLYCOLYSIS-PWY)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "db35c21d",
   "metadata": {},
   "source": [
    "<a id='ipath'></a>\n",
    "### 3.1 Enriched KEGG orthologs visualization for Disease Types\n",
    "\n",
    "We start by merging our feature table with the treatment column (`disease`) from the metadata:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "240402d0-149d-44e0-a7e6-4b2400fdaf77",
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = pd.read_csv(f'{data_dir}/metadata/str_metadata.tsv', sep='\\t', header=0, index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "2a0fd440",
   "metadata": {},
   "outputs": [],
   "source": [
    "ko_a_meta = ko_a.merge(metadata[['disease']], left_index=True, right_index=True)\n",
    "ec_a_meta = ec_a.merge(metadata[['disease']], left_index=True, right_index=True)\n",
    "pa_a_meta = pa_a.merge(metadata[['disease']], left_index=True, right_index=True)\n",
    "ko_na_meta = ko_na.merge(metadata[['disease']], left_index=True, right_index=True)\n",
    "ec_na_meta = ec_na.merge(metadata[['disease']], left_index=True, right_index=True)\n",
    "pa_na_meta = pa_na.merge(metadata[['disease']], left_index=True, right_index=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0a50441",
   "metadata": {},
   "source": [
    "Next, we will calculate an average abundance of each KO, EC and pathway in each group (abducted vs. non abducetd):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "a56f5bfc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collapse samples per sample_type - calculate average abundance\n",
    "\n",
    "ko_a_meta_avg = ko_a_meta.groupby('disease').mean()\n",
    "ec_a_meta_avg = ec_a_meta.groupby('disease').mean()\n",
    "pa_a_meta_avg = pa_a_meta.groupby('disease').mean()\n",
    "ko_na_meta_avg = ko_na_meta.groupby('disease').mean()\n",
    "ec_na_meta_avg = ec_na_meta.groupby('disease').mean()\n",
    "pa_na_meta_avg = pa_na_meta.groupby('disease').mean()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fce20166-27c7-4973-951e-4e0ffdf0aed8",
   "metadata": {},
   "source": [
    "We can list the averages of KEGG orthologs for different diseases across alien abducted and not abducted patients. We observe that the values are different across patients within each disease group and across the patients with same disease with different alien abduction status."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "36830ebe",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19776</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>disease</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Hodgkin's Disease</th>\n",
       "      <td>34953.468700</td>\n",
       "      <td>2.552380</td>\n",
       "      <td>41150.899491</td>\n",
       "      <td>4825.747676</td>\n",
       "      <td>6031.352334</td>\n",
       "      <td>2.987571</td>\n",
       "      <td>2319.676886</td>\n",
       "      <td>1318.989544</td>\n",
       "      <td>6554.637510</td>\n",
       "      <td>1.033848e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>516.795292</td>\n",
       "      <td>60.329399</td>\n",
       "      <td>242.171153</td>\n",
       "      <td>14.429723</td>\n",
       "      <td>57.305988</td>\n",
       "      <td>1022.062706</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>1147.702727</td>\n",
       "      <td>0.422670</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Leukemia</th>\n",
       "      <td>12857.970171</td>\n",
       "      <td>87.614986</td>\n",
       "      <td>36454.100688</td>\n",
       "      <td>25317.228261</td>\n",
       "      <td>17271.679642</td>\n",
       "      <td>12.264023</td>\n",
       "      <td>26162.447626</td>\n",
       "      <td>23579.801853</td>\n",
       "      <td>13815.782371</td>\n",
       "      <td>7.105095e-04</td>\n",
       "      <td>...</td>\n",
       "      <td>1459.920138</td>\n",
       "      <td>671.360458</td>\n",
       "      <td>696.323734</td>\n",
       "      <td>146.345722</td>\n",
       "      <td>601.635315</td>\n",
       "      <td>12723.246532</td>\n",
       "      <td>0.010643</td>\n",
       "      <td>3.690182e-59</td>\n",
       "      <td>1491.308203</td>\n",
       "      <td>0.000295</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Myelodysplastic Syndromes</th>\n",
       "      <td>1994.688516</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>6361.559141</td>\n",
       "      <td>0.374649</td>\n",
       "      <td>3321.432803</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>19096.681963</td>\n",
       "      <td>25235.315634</td>\n",
       "      <td>8.007399</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>3297.361743</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>40.445930</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Non-Hodgkin's Lymphoma</th>\n",
       "      <td>16468.199466</td>\n",
       "      <td>76.265046</td>\n",
       "      <td>41743.571134</td>\n",
       "      <td>4283.070794</td>\n",
       "      <td>15978.635289</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>53992.016472</td>\n",
       "      <td>63361.706906</td>\n",
       "      <td>1237.352134</td>\n",
       "      <td>1.927133e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>2.694251</td>\n",
       "      <td>2.572491</td>\n",
       "      <td>3.047133</td>\n",
       "      <td>0.639329</td>\n",
       "      <td>2.559274</td>\n",
       "      <td>7232.517849</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>147.295410</td>\n",
       "      <td>0.006617</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows × 9847 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 K00001     K00002        K00003  \\\n",
       "disease                                                            \n",
       "Hodgkin's Disease          34953.468700   2.552380  41150.899491   \n",
       "Leukemia                   12857.970171  87.614986  36454.100688   \n",
       "Myelodysplastic Syndromes   1994.688516   0.000000   6361.559141   \n",
       "Non-Hodgkin's Lymphoma     16468.199466  76.265046  41743.571134   \n",
       "\n",
       "                                 K00004        K00005     K00007  \\\n",
       "disease                                                            \n",
       "Hodgkin's Disease           4825.747676   6031.352334   2.987571   \n",
       "Leukemia                   25317.228261  17271.679642  12.264023   \n",
       "Myelodysplastic Syndromes      0.374649   3321.432803   0.000000   \n",
       "Non-Hodgkin's Lymphoma      4283.070794  15978.635289   0.000000   \n",
       "\n",
       "                                 K00008        K00009        K00010  \\\n",
       "disease                                                               \n",
       "Hodgkin's Disease           2319.676886   1318.989544   6554.637510   \n",
       "Leukemia                   26162.447626  23579.801853  13815.782371   \n",
       "Myelodysplastic Syndromes  19096.681963  25235.315634      8.007399   \n",
       "Non-Hodgkin's Lymphoma     53992.016472  63361.706906   1237.352134   \n",
       "\n",
       "                                 K00011  ...       K19776      K19777  \\\n",
       "disease                                  ...                            \n",
       "Hodgkin's Disease          1.033848e-21  ...   516.795292   60.329399   \n",
       "Leukemia                   7.105095e-04  ...  1459.920138  671.360458   \n",
       "Myelodysplastic Syndromes  0.000000e+00  ...     0.000000    0.000000   \n",
       "Non-Hodgkin's Lymphoma     1.927133e-21  ...     2.694251    2.572491   \n",
       "\n",
       "                               K19778      K19779      K19780        K19784  \\\n",
       "disease                                                                       \n",
       "Hodgkin's Disease          242.171153   14.429723   57.305988   1022.062706   \n",
       "Leukemia                   696.323734  146.345722  601.635315  12723.246532   \n",
       "Myelodysplastic Syndromes    0.000000    0.000000    0.000000   3297.361743   \n",
       "Non-Hodgkin's Lymphoma       3.047133    0.639329    2.559274   7232.517849   \n",
       "\n",
       "                             K19785        K19788       K19789    K19791  \n",
       "disease                                                                   \n",
       "Hodgkin's Disease          0.000000  0.000000e+00  1147.702727  0.422670  \n",
       "Leukemia                   0.010643  3.690182e-59  1491.308203  0.000295  \n",
       "Myelodysplastic Syndromes  0.000000  0.000000e+00    40.445930  0.000000  \n",
       "Non-Hodgkin's Lymphoma     0.000000  0.000000e+00   147.295410  0.006617  \n",
       "\n",
       "[4 rows x 9847 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_a_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "a5bc224f-ab36-4eb3-b93c-9092fb09dd0b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19787</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>disease</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Hodgkin's Disease</th>\n",
       "      <td>19028.235594</td>\n",
       "      <td>1065.362017</td>\n",
       "      <td>34387.092140</td>\n",
       "      <td>5062.437658</td>\n",
       "      <td>14231.673873</td>\n",
       "      <td>0.909434</td>\n",
       "      <td>41327.780462</td>\n",
       "      <td>8815.128883</td>\n",
       "      <td>16317.379247</td>\n",
       "      <td>0.002319</td>\n",
       "      <td>...</td>\n",
       "      <td>357.612024</td>\n",
       "      <td>374.245338</td>\n",
       "      <td>82.476389</td>\n",
       "      <td>357.136535</td>\n",
       "      <td>1636.803313</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>764.903179</td>\n",
       "      <td>0.004102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Leukemia</th>\n",
       "      <td>17227.619162</td>\n",
       "      <td>423.810503</td>\n",
       "      <td>16639.077543</td>\n",
       "      <td>2499.003063</td>\n",
       "      <td>6982.468350</td>\n",
       "      <td>21.041076</td>\n",
       "      <td>18481.732069</td>\n",
       "      <td>11486.349201</td>\n",
       "      <td>9974.081115</td>\n",
       "      <td>0.003699</td>\n",
       "      <td>...</td>\n",
       "      <td>38.012421</td>\n",
       "      <td>371.852232</td>\n",
       "      <td>8.854783</td>\n",
       "      <td>35.672199</td>\n",
       "      <td>5126.323039</td>\n",
       "      <td>0.000112</td>\n",
       "      <td>0.000287</td>\n",
       "      <td>1.960334e-65</td>\n",
       "      <td>190.707274</td>\n",
       "      <td>0.002403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Myelodysplastic Syndromes</th>\n",
       "      <td>13056.800498</td>\n",
       "      <td>1193.270974</td>\n",
       "      <td>14331.828948</td>\n",
       "      <td>360.672058</td>\n",
       "      <td>22843.442305</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>28629.955610</td>\n",
       "      <td>3001.358000</td>\n",
       "      <td>8689.148528</td>\n",
       "      <td>0.101260</td>\n",
       "      <td>...</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.180566</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>220.871235</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>1450.629391</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Non-Hodgkin's Lymphoma</th>\n",
       "      <td>11439.378480</td>\n",
       "      <td>578.621286</td>\n",
       "      <td>26721.551141</td>\n",
       "      <td>3549.133715</td>\n",
       "      <td>21085.984097</td>\n",
       "      <td>1.041146</td>\n",
       "      <td>40508.693863</td>\n",
       "      <td>26224.881159</td>\n",
       "      <td>12625.656417</td>\n",
       "      <td>0.048428</td>\n",
       "      <td>...</td>\n",
       "      <td>352.133688</td>\n",
       "      <td>354.083430</td>\n",
       "      <td>87.898479</td>\n",
       "      <td>351.837638</td>\n",
       "      <td>4058.661470</td>\n",
       "      <td>0.000130</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>8.615288e-62</td>\n",
       "      <td>505.411611</td>\n",
       "      <td>0.001922</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows × 10251 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 K00001       K00002        K00003  \\\n",
       "disease                                                              \n",
       "Hodgkin's Disease          19028.235594  1065.362017  34387.092140   \n",
       "Leukemia                   17227.619162   423.810503  16639.077543   \n",
       "Myelodysplastic Syndromes  13056.800498  1193.270974  14331.828948   \n",
       "Non-Hodgkin's Lymphoma     11439.378480   578.621286  26721.551141   \n",
       "\n",
       "                                K00004        K00005     K00007        K00008  \\\n",
       "disease                                                                         \n",
       "Hodgkin's Disease          5062.437658  14231.673873   0.909434  41327.780462   \n",
       "Leukemia                   2499.003063   6982.468350  21.041076  18481.732069   \n",
       "Myelodysplastic Syndromes   360.672058  22843.442305   0.000000  28629.955610   \n",
       "Non-Hodgkin's Lymphoma     3549.133715  21085.984097   1.041146  40508.693863   \n",
       "\n",
       "                                 K00009        K00010    K00011  ...  \\\n",
       "disease                                                          ...   \n",
       "Hodgkin's Disease           8815.128883  16317.379247  0.002319  ...   \n",
       "Leukemia                   11486.349201   9974.081115  0.003699  ...   \n",
       "Myelodysplastic Syndromes   3001.358000   8689.148528  0.101260  ...   \n",
       "Non-Hodgkin's Lymphoma     26224.881159  12625.656417  0.048428  ...   \n",
       "\n",
       "                               K19777      K19778     K19779      K19780  \\\n",
       "disease                                                                    \n",
       "Hodgkin's Disease          357.612024  374.245338  82.476389  357.136535   \n",
       "Leukemia                    38.012421  371.852232   8.854783   35.672199   \n",
       "Myelodysplastic Syndromes    0.000000    0.180566   0.000000    0.000000   \n",
       "Non-Hodgkin's Lymphoma     352.133688  354.083430  87.898479  351.837638   \n",
       "\n",
       "                                K19784    K19785    K19787        K19788  \\\n",
       "disease                                                                    \n",
       "Hodgkin's Disease          1636.803313  0.000000  0.000000  0.000000e+00   \n",
       "Leukemia                   5126.323039  0.000112  0.000287  1.960334e-65   \n",
       "Myelodysplastic Syndromes   220.871235  0.000000  0.000000  0.000000e+00   \n",
       "Non-Hodgkin's Lymphoma     4058.661470  0.000130  0.000000  8.615288e-62   \n",
       "\n",
       "                                K19789    K19791  \n",
       "disease                                           \n",
       "Hodgkin's Disease           764.903179  0.004102  \n",
       "Leukemia                    190.707274  0.002403  \n",
       "Myelodysplastic Syndromes  1450.629391  0.000000  \n",
       "Non-Hodgkin's Lymphoma      505.411611  0.001922  \n",
       "\n",
       "[4 rows x 10251 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_na_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "c1a7fea5-ed1e-42f7-af80-89f135bd2d5e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saving 98 most abundant features...\n",
      "Saving 83 most abundant features...\n",
      "Saving 102 most abundant features...\n",
      "Saving 85 most abundant features...\n"
     ]
    }
   ],
   "source": [
    "ko_a_most_abundant = find_most_abundant(ko_a_meta_avg, 0.01)\n",
    "ec_a_most_abundant = find_most_abundant(ec_a_meta_avg, 0.03)\n",
    "ko_na_most_abundant = find_most_abundant(ko_na_meta_avg, 0.01)\n",
    "ec_na_most_abundant = find_most_abundant(ec_na_meta_avg, 0.03)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "050bd3f4-be78-435e-a449-ec51d88f7fb5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{\"Hodgkin's Disease\": K07024    358482.184181\n",
       " K01990    343190.368966\n",
       " K02030    331013.996530\n",
       " K06147    268862.436560\n",
       " K01992    259433.682089\n",
       "               ...      \n",
       " K07483     59614.291643\n",
       " K00973     59170.748358\n",
       " K07707     59161.106030\n",
       " K07260     58634.222123\n",
       " K02770     56391.474971\n",
       " Name: Hodgkin's Disease, Length: 98, dtype: float64,\n",
       " 'Leukemia': K01990    294841.929329\n",
       " K06147    266550.480177\n",
       " K01992    257617.909624\n",
       " K07024    241744.607213\n",
       " K02003    180959.641465\n",
       "               ...      \n",
       " K02495     56078.668617\n",
       " K01448     55649.191422\n",
       " K00655     55641.665805\n",
       " K03655     55427.531542\n",
       " K08884     55413.393737\n",
       " Name: Leukemia, Length: 98, dtype: float64,\n",
       " 'Myelodysplastic Syndromes': K06147    52037.914787\n",
       " K01990    51967.581732\n",
       " K01992    50096.364734\n",
       " K03088    47892.255571\n",
       " K02003    46484.951237\n",
       "               ...     \n",
       " K02015    15853.662795\n",
       " K01091    15804.047777\n",
       " K03704    15552.184685\n",
       " K02775    15418.701385\n",
       " K02783    15362.848506\n",
       " Name: Myelodysplastic Syndromes, Length: 98, dtype: float64,\n",
       " \"Non-Hodgkin's Lymphoma\": K01990    361101.640026\n",
       " K07024    341805.868453\n",
       " K06147    336714.405512\n",
       " K01992    299245.110386\n",
       " K02003    274031.352964\n",
       "               ...      \n",
       " K02013     77038.029407\n",
       " K01462     76389.739891\n",
       " K07496     76217.553373\n",
       " K15583     75872.401710\n",
       " K03530     74947.651298\n",
       " Name: Non-Hodgkin's Lymphoma, Length: 98, dtype: float64}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_a_most_abundant"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "a341ce5e-f3d6-43ba-9683-6bb08c19fa0f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10 most abundant KOs in the abducted group are: ['K01990', 'K06147', 'K01992', 'K07024', 'K02003', 'K02004', 'K02015', 'K02030', 'K02016', 'K02529']\n",
      "10 most abundant KOs in the non-abducted group are: ['K01990', 'K06147', 'K01992', 'K07024', 'K02003', 'K02004', 'K02015', 'K02030', 'K02016', 'K02529']\n",
      "10 most abundant KOs in the non-abducted group are: ['K01990', 'K06147', 'K01992', 'K07024', 'K02003', 'K02004', 'K02015', 'K02030', 'K02016', 'K02529']\n",
      "10 most abundant KOs in the non-abducted group are: ['K01990', 'K06147', 'K01992', 'K07024', 'K02003', 'K02004', 'K02015', 'K02030', 'K02016', 'K02529']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(f'10 most abundant KOs in the abducted group are: {ko_a_most_abundant[\"Leukemia\"].index[:10].tolist()}\\n'\n",
    "      f'10 most abundant KOs in the non-abducted group are: {ko_a_most_abundant[\"Leukemia\"].index[:10].tolist()}\\n'\n",
    "      f'10 most abundant KOs in the non-abducted group are: {ko_a_most_abundant[\"Leukemia\"].index[:10].tolist()}\\n'\n",
    "      f'10 most abundant KOs in the non-abducted group are: {ko_a_most_abundant[\"Leukemia\"].index[:10].tolist()}\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "910ab7e5-cf00-4dbb-9834-dc4ced689fe4",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c072f4a2-0547-402c-af6f-895e126023e0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "74b5c8c5-8f7f-4d62-a8fd-594865c73957",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0e56475a-4ce7-4418-a3db-c3c7a71f4eeb",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "702e0414-f9df-42fa-952e-4b201d49735e",
   "metadata": {},
   "source": [
    "### 3.2 Enriched pathways\n",
    "\n",
    "We know visualize the pathway abundances of different groups of patients across disease and alien abduction status."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "b3287037-7f00-4b44-a2f7-ec54de8b14e7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Composition] to: ../data//picrust2_results/pathway_abundance_abducted_abund.qza\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved FeatureTable[Composition] to: ../data//picrust2_results/pathway_abundance_not_abducted_abund.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition add-pseudocount \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_abducted.qza \\\n",
    "    --o-composition-table $data_dir/picrust2_results/pathway_abundance_abducted_abund.qza\n",
    "\n",
    "! qiime composition add-pseudocount \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_not_abducted.qza \\\n",
    "    --o-composition-table $data_dir/picrust2_results/pathway_abundance_not_abducted_abund.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "6c26938f-6953-4de6-8571-0bcf8ea82dc9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: ../data//functional_prediction/pa_ancom_abducted_disease.qzv\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved Visualization to: ../data//functional_prediction/pa_ancom_not_abducted_disease.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column disease \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/functional_prediction/pa_ancom_abducted_disease.qzv\n",
    "\n",
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_not_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column disease \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/functional_prediction/pa_ancom_not_abducted_disease.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "38088f28-dff6-44f6-bbe6-80d95ef9bb1a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-h1e7mkw4')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 249b3f18-348a-46f1-8ecc-88c519f33aea>"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/functional_prediction/pa_ancom_abducted_disease.qzv')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "0247b545-ff4e-4873-b72e-dd9460a0c35b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-0kxzx41d')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: d0f861bd-2e4e-4356-8a3e-e6434a607fc7>"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/functional_prediction/pa_ancom_not_abducted_disease.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a3596787-2ddc-47b4-a5d3-8c0fdea7d570",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "70e1e318-d682-46af-8f1d-30dff77e5028",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "89b9f6ec-921a-438b-9f13-5e9df5dc5dfd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1c94dfce-4197-48ed-b204-83c35e5d6e4b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "86393f58-2055-4c78-8f0c-3777a0d5c449",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4818cfef-c2ba-49f6-ad4b-525798f696eb",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f32c2ae9-1b06-4b19-b20f-36d6ff61cbba",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "78a0abf7-0358-482d-ae6c-f145cf8a4d68",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "07c29567-f379-46c8-84c8-9afa69b4895f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dc5bcc41-807c-436a-bb74-f21b6655af58",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "02ed34d2-29b9-4a62-a603-6df2d465523d",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "a883b090-90c3-4845-b50a-b09e22d54573",
   "metadata": {},
   "source": [
    "## 2. Functional Inference on Stool Consistency"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "76c48d19-a0da-48bd-acac-37453cae5598",
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = pd.read_csv(f'{data_dir}/metadata/str_metadata.tsv', sep='\\t', header=0, index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "c558bd1d-bf24-4d24-9bc5-92e62b0a6172",
   "metadata": {},
   "outputs": [],
   "source": [
    "ko_a_meta = ko_a.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n",
    "ec_a_meta = ec_a.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n",
    "pa_a_meta = pa_a.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n",
    "ko_na_meta = ko_na.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n",
    "ec_na_meta = ec_na.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n",
    "pa_na_meta = pa_na.merge(metadata[['stool_consistency']], left_index=True, right_index=True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "21288dec-9fef-41ca-afdd-eec964804edd",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collapse samples per sample_type - calculate average abundance\n",
    "\n",
    "ko_a_meta_avg = ko_a_meta.groupby('stool_consistency').mean()\n",
    "ec_a_meta_avg = ec_a_meta.groupby('stool_consistency').mean()\n",
    "pa_a_meta_avg = pa_a_meta.groupby('stool_consistency').mean()\n",
    "ko_na_meta_avg = ko_na_meta.groupby('stool_consistency').mean()\n",
    "ec_na_meta_avg = ec_na_meta.groupby('stool_consistency').mean()\n",
    "pa_na_meta_avg = pa_na_meta.groupby('stool_consistency').mean()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "524036e0-9dbd-4a03-a973-eaf3989922a2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19787</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stool_consistency</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>formed</th>\n",
       "      <td>6037.306790</td>\n",
       "      <td>773.656380</td>\n",
       "      <td>20189.709305</td>\n",
       "      <td>2051.262596</td>\n",
       "      <td>11894.709954</td>\n",
       "      <td>0.817587</td>\n",
       "      <td>29769.856206</td>\n",
       "      <td>16085.548045</td>\n",
       "      <td>12866.066883</td>\n",
       "      <td>0.040517</td>\n",
       "      <td>...</td>\n",
       "      <td>343.760265</td>\n",
       "      <td>346.835130</td>\n",
       "      <td>84.681018</td>\n",
       "      <td>343.471666</td>\n",
       "      <td>2282.388267</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>513.854295</td>\n",
       "      <td>0.000347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>liquid</th>\n",
       "      <td>18708.284957</td>\n",
       "      <td>529.118973</td>\n",
       "      <td>28660.095238</td>\n",
       "      <td>2850.176466</td>\n",
       "      <td>21045.077804</td>\n",
       "      <td>17.110777</td>\n",
       "      <td>43873.721815</td>\n",
       "      <td>24820.053589</td>\n",
       "      <td>13028.297977</td>\n",
       "      <td>0.020591</td>\n",
       "      <td>...</td>\n",
       "      <td>15.907100</td>\n",
       "      <td>561.045843</td>\n",
       "      <td>3.651652</td>\n",
       "      <td>15.008893</td>\n",
       "      <td>2734.697251</td>\n",
       "      <td>3.854027e-04</td>\n",
       "      <td>0.000478</td>\n",
       "      <td>3.267224e-65</td>\n",
       "      <td>341.382649</td>\n",
       "      <td>0.003919</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>semi-formed</th>\n",
       "      <td>27573.876340</td>\n",
       "      <td>190.813369</td>\n",
       "      <td>19447.748558</td>\n",
       "      <td>5223.539693</td>\n",
       "      <td>10406.258055</td>\n",
       "      <td>19.993414</td>\n",
       "      <td>15489.318023</td>\n",
       "      <td>10327.139551</td>\n",
       "      <td>8483.660044</td>\n",
       "      <td>0.000058</td>\n",
       "      <td>...</td>\n",
       "      <td>70.334807</td>\n",
       "      <td>85.938083</td>\n",
       "      <td>16.743543</td>\n",
       "      <td>66.983448</td>\n",
       "      <td>9193.116584</td>\n",
       "      <td>7.763663e-132</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>1.435881e-61</td>\n",
       "      <td>273.870191</td>\n",
       "      <td>0.003988</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows × 10251 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         K00001      K00002        K00003       K00004  \\\n",
       "stool_consistency                                                        \n",
       "formed              6037.306790  773.656380  20189.709305  2051.262596   \n",
       "liquid             18708.284957  529.118973  28660.095238  2850.176466   \n",
       "semi-formed        27573.876340  190.813369  19447.748558  5223.539693   \n",
       "\n",
       "                         K00005     K00007        K00008        K00009  \\\n",
       "stool_consistency                                                        \n",
       "formed             11894.709954   0.817587  29769.856206  16085.548045   \n",
       "liquid             21045.077804  17.110777  43873.721815  24820.053589   \n",
       "semi-formed        10406.258055  19.993414  15489.318023  10327.139551   \n",
       "\n",
       "                         K00010    K00011  ...      K19777      K19778  \\\n",
       "stool_consistency                          ...                           \n",
       "formed             12866.066883  0.040517  ...  343.760265  346.835130   \n",
       "liquid             13028.297977  0.020591  ...   15.907100  561.045843   \n",
       "semi-formed         8483.660044  0.000058  ...   70.334807   85.938083   \n",
       "\n",
       "                      K19779      K19780       K19784         K19785  \\\n",
       "stool_consistency                                                      \n",
       "formed             84.681018  343.471666  2282.388267   0.000000e+00   \n",
       "liquid              3.651652   15.008893  2734.697251   3.854027e-04   \n",
       "semi-formed        16.743543   66.983448  9193.116584  7.763663e-132   \n",
       "\n",
       "                     K19787        K19788      K19789    K19791  \n",
       "stool_consistency                                                \n",
       "formed             0.000000  0.000000e+00  513.854295  0.000347  \n",
       "liquid             0.000478  3.267224e-65  341.382649  0.003919  \n",
       "semi-formed        0.000000  1.435881e-61  273.870191  0.003988  \n",
       "\n",
       "[3 rows x 10251 columns]"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_a_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "51ea00b2-4570-458e-aa08-58848913c063",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19776</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>stool_consistency</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>formed</th>\n",
       "      <td>10147.006150</td>\n",
       "      <td>38.277427</td>\n",
       "      <td>28631.098128</td>\n",
       "      <td>15020.481029</td>\n",
       "      <td>18270.325950</td>\n",
       "      <td>24.834485</td>\n",
       "      <td>22120.136212</td>\n",
       "      <td>22564.353026</td>\n",
       "      <td>14162.836028</td>\n",
       "      <td>1.333052e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>3313.347149</td>\n",
       "      <td>1577.393486</td>\n",
       "      <td>1586.251307</td>\n",
       "      <td>343.858760</td>\n",
       "      <td>1414.225607</td>\n",
       "      <td>9928.389741</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>3350.177255</td>\n",
       "      <td>0.000102</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>liquid</th>\n",
       "      <td>17736.375188</td>\n",
       "      <td>93.757042</td>\n",
       "      <td>50932.307112</td>\n",
       "      <td>27855.875670</td>\n",
       "      <td>18077.871448</td>\n",
       "      <td>1.568945</td>\n",
       "      <td>49075.026567</td>\n",
       "      <td>48127.915567</td>\n",
       "      <td>9740.345325</td>\n",
       "      <td>1.499965e-03</td>\n",
       "      <td>...</td>\n",
       "      <td>130.917774</td>\n",
       "      <td>10.554224</td>\n",
       "      <td>50.823766</td>\n",
       "      <td>2.131272</td>\n",
       "      <td>8.428592</td>\n",
       "      <td>15485.064326</td>\n",
       "      <td>0.022468</td>\n",
       "      <td>7.790385e-59</td>\n",
       "      <td>243.304554</td>\n",
       "      <td>0.003744</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>semi-formed</th>\n",
       "      <td>13643.017988</td>\n",
       "      <td>99.836616</td>\n",
       "      <td>29046.519817</td>\n",
       "      <td>15155.787109</td>\n",
       "      <td>12060.491574</td>\n",
       "      <td>2.578626</td>\n",
       "      <td>18869.624145</td>\n",
       "      <td>19745.890068</td>\n",
       "      <td>8253.719387</td>\n",
       "      <td>6.192900e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>64.856889</td>\n",
       "      <td>12.767183</td>\n",
       "      <td>37.792601</td>\n",
       "      <td>3.127061</td>\n",
       "      <td>12.390128</td>\n",
       "      <td>7047.888993</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>140.924726</td>\n",
       "      <td>0.047428</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows × 9847 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         K00001     K00002        K00003        K00004  \\\n",
       "stool_consistency                                                        \n",
       "formed             10147.006150  38.277427  28631.098128  15020.481029   \n",
       "liquid             17736.375188  93.757042  50932.307112  27855.875670   \n",
       "semi-formed        13643.017988  99.836616  29046.519817  15155.787109   \n",
       "\n",
       "                         K00005     K00007        K00008        K00009  \\\n",
       "stool_consistency                                                        \n",
       "formed             18270.325950  24.834485  22120.136212  22564.353026   \n",
       "liquid             18077.871448   1.568945  49075.026567  48127.915567   \n",
       "semi-formed        12060.491574   2.578626  18869.624145  19745.890068   \n",
       "\n",
       "                         K00010        K00011  ...       K19776       K19777  \\\n",
       "stool_consistency                              ...                             \n",
       "formed             14162.836028  1.333052e-21  ...  3313.347149  1577.393486   \n",
       "liquid              9740.345325  1.499965e-03  ...   130.917774    10.554224   \n",
       "semi-formed         8253.719387  6.192900e-21  ...    64.856889    12.767183   \n",
       "\n",
       "                        K19778      K19779       K19780        K19784  \\\n",
       "stool_consistency                                                       \n",
       "formed             1586.251307  343.858760  1414.225607   9928.389741   \n",
       "liquid               50.823766    2.131272     8.428592  15485.064326   \n",
       "semi-formed          37.792601    3.127061    12.390128   7047.888993   \n",
       "\n",
       "                     K19785        K19788       K19789    K19791  \n",
       "stool_consistency                                                 \n",
       "formed             0.000000  0.000000e+00  3350.177255  0.000102  \n",
       "liquid             0.022468  7.790385e-59   243.304554  0.003744  \n",
       "semi-formed        0.000000  0.000000e+00   140.924726  0.047428  \n",
       "\n",
       "[3 rows x 9847 columns]"
      ]
     },
     "execution_count": 44,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_na_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "c4e5e7f4-dc0c-48fe-94d6-f357751cc382",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: shared_data//pa_ancom_abducted_stool_consistency.qzv\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved Visualization to: shared_data//pa_ancom_not_abducted_stool_consistency.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column stool_consistency \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/pa_ancom_abducted_stool_consistency.qzv\n",
    "\n",
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_not_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column stool_consistency \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/pa_ancom_not_abducted_stool_consistency.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "82122d55-240e-4a46-8987-e3f80b2d5551",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-fd8tn2ox')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 6d3f18e9-dfc6-4c96-9127-b0b897e5951f>"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/pa_ancom_abducted_stool_consistency.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "77ec4a4a-d51a-4094-be55-97f8e7f53a2f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-at2tt_hr')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: ac8b2dac-d207-419d-b804-f3c05fe09237>"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/pa_ancom_not_abducted_stool_consistency.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "13836c12-71fb-4fa4-8713-1d7a185078e6",
   "metadata": {},
   "source": [
    "## 3. Functional Inference on Categorical categorical Time Relative to Engraftment\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "a6632a19-37b8-4a25-a951-7ae95afe6cb6",
   "metadata": {},
   "outputs": [],
   "source": [
    "ko_a_meta = ko_a.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)\n",
    "ec_a_meta = ec_a.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)\n",
    "pa_a_meta = pa_a.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)\n",
    "ko_na_meta = ko_na.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)\n",
    "ec_na_meta = ec_na.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)\n",
    "pa_na_meta = pa_na.merge(metadata[['categorical_time_relative_to_engraftment']], left_index=True, right_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "2cc4ba22-fdd5-4cbe-8cbd-b3d175f93adb",
   "metadata": {},
   "outputs": [],
   "source": [
    "# collapse samples per sample_type - calculate average abundance\n",
    "\n",
    "ko_a_meta_avg = ko_a_meta.groupby('categorical_time_relative_to_engraftment').mean()\n",
    "ec_a_meta_avg = ec_a_meta.groupby('categorical_time_relative_to_engraftment').mean()\n",
    "pa_a_meta_avg = pa_a_meta.groupby('categorical_time_relative_to_engraftment').mean()\n",
    "ko_na_meta_avg = ko_na_meta.groupby('categorical_time_relative_to_engraftment').mean()\n",
    "ec_na_meta_avg = ec_na_meta.groupby('categorical_time_relative_to_engraftment').mean()\n",
    "pa_na_meta_avg = pa_na_meta.groupby('categorical_time_relative_to_engraftment').mean()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "a47c0bc6-fecf-470f-be05-3ab55f54304c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19787</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>categorical_time_relative_to_engraftment</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>peri</th>\n",
       "      <td>2114.150651</td>\n",
       "      <td>335.571804</td>\n",
       "      <td>6704.060157</td>\n",
       "      <td>648.476811</td>\n",
       "      <td>3622.080928</td>\n",
       "      <td>1.320137</td>\n",
       "      <td>12160.841250</td>\n",
       "      <td>7810.594355</td>\n",
       "      <td>5321.334654</td>\n",
       "      <td>0.000011</td>\n",
       "      <td>...</td>\n",
       "      <td>31.153772</td>\n",
       "      <td>46.877726</td>\n",
       "      <td>7.337195</td>\n",
       "      <td>29.352181</td>\n",
       "      <td>932.024107</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>81.919162</td>\n",
       "      <td>0.000097</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>post</th>\n",
       "      <td>38891.568070</td>\n",
       "      <td>1018.573990</td>\n",
       "      <td>48330.342573</td>\n",
       "      <td>7486.202129</td>\n",
       "      <td>32076.200421</td>\n",
       "      <td>29.300452</td>\n",
       "      <td>62746.992648</td>\n",
       "      <td>40142.072759</td>\n",
       "      <td>22515.633123</td>\n",
       "      <td>0.077798</td>\n",
       "      <td>...</td>\n",
       "      <td>556.583222</td>\n",
       "      <td>1015.152726</td>\n",
       "      <td>136.602004</td>\n",
       "      <td>554.017200</td>\n",
       "      <td>11353.594292</td>\n",
       "      <td>0.000323</td>\n",
       "      <td>0.0004</td>\n",
       "      <td>1.102229e-61</td>\n",
       "      <td>1114.385819</td>\n",
       "      <td>0.006685</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pre</th>\n",
       "      <td>3349.070585</td>\n",
       "      <td>360.714580</td>\n",
       "      <td>10606.952331</td>\n",
       "      <td>1018.165276</td>\n",
       "      <td>5988.067033</td>\n",
       "      <td>0.595760</td>\n",
       "      <td>14760.258998</td>\n",
       "      <td>5155.612975</td>\n",
       "      <td>6561.072315</td>\n",
       "      <td>0.001120</td>\n",
       "      <td>...</td>\n",
       "      <td>14.079083</td>\n",
       "      <td>17.604554</td>\n",
       "      <td>3.483824</td>\n",
       "      <td>13.948624</td>\n",
       "      <td>614.715979</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>78.476669</td>\n",
       "      <td>0.000082</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows × 10251 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                K00001       K00002  \\\n",
       "categorical_time_relative_to_engraftment                              \n",
       "peri                                       2114.150651   335.571804   \n",
       "post                                      38891.568070  1018.573990   \n",
       "pre                                        3349.070585   360.714580   \n",
       "\n",
       "                                                K00003       K00004  \\\n",
       "categorical_time_relative_to_engraftment                              \n",
       "peri                                       6704.060157   648.476811   \n",
       "post                                      48330.342573  7486.202129   \n",
       "pre                                       10606.952331  1018.165276   \n",
       "\n",
       "                                                K00005     K00007  \\\n",
       "categorical_time_relative_to_engraftment                            \n",
       "peri                                       3622.080928   1.320137   \n",
       "post                                      32076.200421  29.300452   \n",
       "pre                                        5988.067033   0.595760   \n",
       "\n",
       "                                                K00008        K00009  \\\n",
       "categorical_time_relative_to_engraftment                               \n",
       "peri                                      12160.841250   7810.594355   \n",
       "post                                      62746.992648  40142.072759   \n",
       "pre                                       14760.258998   5155.612975   \n",
       "\n",
       "                                                K00010    K00011  ...  \\\n",
       "categorical_time_relative_to_engraftment                          ...   \n",
       "peri                                       5321.334654  0.000011  ...   \n",
       "post                                      22515.633123  0.077798  ...   \n",
       "pre                                        6561.072315  0.001120  ...   \n",
       "\n",
       "                                              K19777       K19778      K19779  \\\n",
       "categorical_time_relative_to_engraftment                                        \n",
       "peri                                       31.153772    46.877726    7.337195   \n",
       "post                                      556.583222  1015.152726  136.602004   \n",
       "pre                                        14.079083    17.604554    3.483824   \n",
       "\n",
       "                                              K19780        K19784    K19785  \\\n",
       "categorical_time_relative_to_engraftment                                       \n",
       "peri                                       29.352181    932.024107  0.000000   \n",
       "post                                      554.017200  11353.594292  0.000323   \n",
       "pre                                        13.948624    614.715979  0.000000   \n",
       "\n",
       "                                          K19787        K19788       K19789  \\\n",
       "categorical_time_relative_to_engraftment                                      \n",
       "peri                                      0.0000  0.000000e+00    81.919162   \n",
       "post                                      0.0004  1.102229e-61  1114.385819   \n",
       "pre                                       0.0000  0.000000e+00    78.476669   \n",
       "\n",
       "                                            K19791  \n",
       "categorical_time_relative_to_engraftment            \n",
       "peri                                      0.000097  \n",
       "post                                      0.006685  \n",
       "pre                                       0.000082  \n",
       "\n",
       "[3 rows x 10251 columns]"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_a_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "bd2e7ad3-9b8f-4c4d-bdab-b54912ca0bac",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>K00001</th>\n",
       "      <th>K00002</th>\n",
       "      <th>K00003</th>\n",
       "      <th>K00004</th>\n",
       "      <th>K00005</th>\n",
       "      <th>K00007</th>\n",
       "      <th>K00008</th>\n",
       "      <th>K00009</th>\n",
       "      <th>K00010</th>\n",
       "      <th>K00011</th>\n",
       "      <th>...</th>\n",
       "      <th>K19776</th>\n",
       "      <th>K19777</th>\n",
       "      <th>K19778</th>\n",
       "      <th>K19779</th>\n",
       "      <th>K19780</th>\n",
       "      <th>K19784</th>\n",
       "      <th>K19785</th>\n",
       "      <th>K19788</th>\n",
       "      <th>K19789</th>\n",
       "      <th>K19791</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>categorical_time_relative_to_engraftment</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>peri</th>\n",
       "      <td>1071.346104</td>\n",
       "      <td>152.465120</td>\n",
       "      <td>8949.881683</td>\n",
       "      <td>2133.061097</td>\n",
       "      <td>4159.275767</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>20253.321044</td>\n",
       "      <td>9911.320896</td>\n",
       "      <td>7723.821127</td>\n",
       "      <td>1.032150e-21</td>\n",
       "      <td>...</td>\n",
       "      <td>2.666667</td>\n",
       "      <td>2.673357</td>\n",
       "      <td>2.666667</td>\n",
       "      <td>0.665968</td>\n",
       "      <td>2.665910</td>\n",
       "      <td>1445.448955</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>2.673435</td>\n",
       "      <td>0.000079</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>post</th>\n",
       "      <td>20671.457512</td>\n",
       "      <td>71.474948</td>\n",
       "      <td>50965.499933</td>\n",
       "      <td>28947.815703</td>\n",
       "      <td>21931.421817</td>\n",
       "      <td>12.972971</td>\n",
       "      <td>37881.680954</td>\n",
       "      <td>41307.699574</td>\n",
       "      <td>12875.523434</td>\n",
       "      <td>7.940989e-04</td>\n",
       "      <td>...</td>\n",
       "      <td>1661.409336</td>\n",
       "      <td>754.177675</td>\n",
       "      <td>792.338813</td>\n",
       "      <td>164.482187</td>\n",
       "      <td>676.069153</td>\n",
       "      <td>15965.293801</td>\n",
       "      <td>0.011895</td>\n",
       "      <td>4.124322e-59</td>\n",
       "      <td>1776.659257</td>\n",
       "      <td>0.027125</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>pre</th>\n",
       "      <td>1493.628627</td>\n",
       "      <td>62.682144</td>\n",
       "      <td>9265.847245</td>\n",
       "      <td>1459.460499</td>\n",
       "      <td>5349.312766</td>\n",
       "      <td>2.577253</td>\n",
       "      <td>13952.401125</td>\n",
       "      <td>9902.369913</td>\n",
       "      <td>5532.318145</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>...</td>\n",
       "      <td>2.798408</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>1.632991</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>1079.559387</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>8.042315</td>\n",
       "      <td>0.000000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows × 9847 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                K00001      K00002  \\\n",
       "categorical_time_relative_to_engraftment                             \n",
       "peri                                       1071.346104  152.465120   \n",
       "post                                      20671.457512   71.474948   \n",
       "pre                                        1493.628627   62.682144   \n",
       "\n",
       "                                                K00003        K00004  \\\n",
       "categorical_time_relative_to_engraftment                               \n",
       "peri                                       8949.881683   2133.061097   \n",
       "post                                      50965.499933  28947.815703   \n",
       "pre                                        9265.847245   1459.460499   \n",
       "\n",
       "                                                K00005     K00007  \\\n",
       "categorical_time_relative_to_engraftment                            \n",
       "peri                                       4159.275767   0.000000   \n",
       "post                                      21931.421817  12.972971   \n",
       "pre                                        5349.312766   2.577253   \n",
       "\n",
       "                                                K00008        K00009  \\\n",
       "categorical_time_relative_to_engraftment                               \n",
       "peri                                      20253.321044   9911.320896   \n",
       "post                                      37881.680954  41307.699574   \n",
       "pre                                       13952.401125   9902.369913   \n",
       "\n",
       "                                                K00010        K00011  ...  \\\n",
       "categorical_time_relative_to_engraftment                              ...   \n",
       "peri                                       7723.821127  1.032150e-21  ...   \n",
       "post                                      12875.523434  7.940989e-04  ...   \n",
       "pre                                        5532.318145  0.000000e+00  ...   \n",
       "\n",
       "                                               K19776      K19777      K19778  \\\n",
       "categorical_time_relative_to_engraftment                                        \n",
       "peri                                         2.666667    2.673357    2.666667   \n",
       "post                                      1661.409336  754.177675  792.338813   \n",
       "pre                                          2.798408    0.000000    1.632991   \n",
       "\n",
       "                                              K19779      K19780  \\\n",
       "categorical_time_relative_to_engraftment                           \n",
       "peri                                        0.665968    2.665910   \n",
       "post                                      164.482187  676.069153   \n",
       "pre                                         0.000000    0.000000   \n",
       "\n",
       "                                                K19784    K19785  \\\n",
       "categorical_time_relative_to_engraftment                           \n",
       "peri                                       1445.448955  0.000000   \n",
       "post                                      15965.293801  0.011895   \n",
       "pre                                        1079.559387  0.000000   \n",
       "\n",
       "                                                K19788       K19789    K19791  \n",
       "categorical_time_relative_to_engraftment                                       \n",
       "peri                                      0.000000e+00     2.673435  0.000079  \n",
       "post                                      4.124322e-59  1776.659257  0.027125  \n",
       "pre                                       0.000000e+00     8.042315  0.000000  \n",
       "\n",
       "[3 rows x 9847 columns]"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ko_na_meta_avg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "cd45a77d-8cf3-4034-ad45-44142bb72f5e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: shared_data//pa_ancom_abducted_categorical_time.qzv\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved Visualization to: shared_data//pa_ancom_not_abducted_categorical_time.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column categorical_time_relative_to_engraftment \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/pa_ancom_abducted_categorical_time.qzv\n",
    "\n",
    "! qiime composition ancom \\\n",
    "    --i-table $data_dir/picrust2_results/pathway_abundance_not_abducted_abund.qza \\\n",
    "    --m-metadata-file $data_dir/metadata/str_nan_metadata.tsv \\\n",
    "    --m-metadata-column categorical_time_relative_to_engraftment \\\n",
    "    --p-transform-function log \\\n",
    "    --o-visualization $data_dir/pa_ancom_not_abducted_categorical_time.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "20427b26-0a05-486e-b7fa-46b179219c53",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-yo5xfb2l')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 3988bff9-d266-4f6e-99cb-646a0b9b3674>"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/pa_ancom_abducted_categorical_time.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "9744eac1-dc3d-4efe-9e95-3fecd2305722",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2-archive-1o0q0v8u')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 28882149-f4f0-48bb-8c61-27706c826af9>"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Visualization.load(f'{data_dir}/pa_ancom_not_abducted_categorical_time.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "81e62cf4-6a1c-4481-b568-95c4427ccffa",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
